<!DOCTYPE html>
<!-- This site was created with Wowchemy. https://www.wowchemy.com -->
<!-- Last Published: May 22, 2024 --><html lang="en-us" >


<head>
  <meta charset="utf-8" />
  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  
  
  
    <meta name="generator" content="Wowchemy 5.7.0 for Hugo" />
  

  
  












  
  










  







  
  
  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
  

  
  
  
    
      
      <link rel="preload" as="style" href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Mono&family=Roboto:wght@400;700&family=Karma&family=Mina&display=swap">
      <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Mono&family=Roboto:wght@400;700&family=Karma&family=Mina&display=swap" media="print" onload="this.media='all'">
    
  

  
  

  
  
    
    <script src="/js/mathjax-config.js"></script>
  

  

  <link rel="stylesheet" href="/css/vendor-bundle.min.047268c6dd09ad74ba54a0ba71837064.css" media="print" onload="this.media='all'">

  
  
  
    
    
      <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.2/css/academicons.min.css" integrity="sha512-KlJCpRsLf+KKu2VQa5vmRuClRFjxc5lXO03ixZt82HZUk41+1I0bD8KBSA0fY290ayMfWYI9udIqeOWSu1/uZg==" crossorigin="anonymous" media="print" onload="this.media='all'">
    

    
    
    
    
      
      
    
    
    

    
    
    
      <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/leaflet@1.7.1/dist/leaflet.min.css" integrity="" crossorigin="anonymous" media="print" onload="this.media='all'">
    

    

    
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
        <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml.js" integrity="" crossorigin="anonymous" async></script>
      
    
      
      

      
      

      
    
      
      

      
      

      
    
  

  
  
  
  
  
  
  <link rel="stylesheet" href="/css/wowchemy.cfa383012e109fa07dd80bafcdf476df.css" />

  
  
  

  
  
  
  
  
  
  
    
    
    <link rel="stylesheet" href="/css/libs/chroma/github-light.min.css" title="hl-light" media="print" onload="this.media='all'" >
    <link rel="stylesheet" href="/css/libs/chroma/dracula.min.css" title="hl-dark" media="print" onload="this.media='all'" disabled>
  

  
  






<script async src="https://www.googletagmanager.com/gtag/js?id=G-3CBVR6GQ07"></script>
<script>
  window.dataLayer = window.dataLayer || [];

  function gtag() {
      dataLayer.push(arguments);
  }

  function trackOutboundLink(url, target) {
    gtag('event', 'click', {
         'event_category': 'outbound',
         'event_label': url,
         'transport_type': 'beacon',
         'event_callback': function () {
           if (target !== '_blank') {
             document.location = url;
           }
         }
    });
    console.debug("Outbound link clicked: " + url);
  }

  function onClickCallback(event) {
    if ((event.target.tagName !== 'A') || (event.target.host === window.location.host)) {
      return;
    }
    trackOutboundLink(event.target, event.target.getAttribute('target'));  
  }

  gtag('js', new Date());
  gtag('config', 'G-3CBVR6GQ07', {});
  gtag('set', {'cookie_flags': 'SameSite=None;Secure'});

  
  document.addEventListener('click', onClickCallback, false);
</script>























  
  
  






  <meta name="author" content="Leonardo Collado-Torres" />





  

<meta name="description" content="__Background__: Spatial transcriptomics is a next-generation sequencing technology that combines the strengths of transcriptome-wide RNA-sequencing with histological imaging to generate spatial maps of gene expression in intact tissue sections. The 10x Genomics Visium and Visium-Immunofluorescence (Visium-IF) platforms are widely available commercial technologies for quantifying spatially-resolved gene expression. These technologies directly couple gene expression with high resolution histological or immunofluorescence images that contain rich morphological information about the tissue section. However, extracting and integrating image features with gene expression data remains challenging. __Results__: Using MATLAB, we developed VistoSeg, which is a pipeline to process, analyze, and interactively visualize the high-resolution images from the 10x Genomics Visium and Visium-IF platforms. The output from VistoSeg can then be integrated with the spatial-molecular information in downstream analyses using common programming languages, such as R or Python. __Conclusion__: VistoSeg provides user-friendly tools for integrating image-derived metrics from histological and immunofluorescent images with spatially-resolved gene expression data. This integrated approach can advance our understanding of the transcriptional landscape within tissue architecture. VistoSeg is freely available at http://research.libd.org/VistoSeg/. __Impact Statement__: Technologies for measuring gene activity levels, referred to as gene expression, have been evolving over decades and are the core of the transcriptomics subfield within genomics. The first report describing individual cell gene expression is from 2009 and as a method it became commercially available in 2014. While single cell transcriptomics increased our resolution beyond homogenate tissue, the advent of spatial transcriptomics technologies and commercial availability of spatial gene expression platforms, such as Visium, has facilitated studying gene expression in anatomical context. Visium measures local gene expression within the histological organization of single 6.5 mm2 cryosection of tissue. Spatially-resolved transcriptomics provides a new challenge: integrating spatial gene expression with high resolution tissue images (brightfield histology or fluorescent antibody staining). VistoSeg image processing software is compatible with both Visium and Visium-IF from 10x Genomics, which are spatially-resolved transcriptomics assays employing histological and immunofluorescent images, respectively. From these images, the number of cells, identity of cell types, and other image-derived markers can be obtained for thousands of 2,375 Âµm2 spots, where genome-wide gene expression is also measured. VistoSeg provides tools that enable processing these images in the context of gene expression maps to integrate these two high dimensional data types, and thus help unlock the new frontier in transcriptomics." />



<link rel="alternate" hreflang="en-us" href="https://lcolladotor.github.io/publication/preprint_vistoseg/" />
<link rel="canonical" href="https://lcolladotor.github.io/publication/preprint_vistoseg/" />



  <link rel="manifest" href="/manifest.webmanifest" />



<link rel="icon" type="image/png" href="/media/icon_hu2b32c4ab415f12f472f73c7b0301b0d1_19400_32x32_fill_lanczos_center_3.png" />
<link rel="apple-touch-icon" type="image/png" href="/media/icon_hu2b32c4ab415f12f472f73c7b0301b0d1_19400_180x180_fill_lanczos_center_3.png" />

<meta name="theme-color" content="#FF6771" />










  






<meta property="twitter:card" content="summary_large_image" />

  <meta property="twitter:site" content="@lcolladotor" />
  <meta property="twitter:creator" content="@lcolladotor" />
<meta property="twitter:image" content="https://lcolladotor.github.io/publication/preprint_vistoseg/featured.png" />
<meta property="og:site_name" content="L. Collado-Torres" />
<meta property="og:url" content="https://lcolladotor.github.io/publication/preprint_vistoseg/" />
<meta property="og:title" content="VistoSeg: processing utilities for high-resolution Visium/Visium-IF images for spatial transcriptomics data | L. Collado-Torres" />
<meta property="og:description" content="__Background__: Spatial transcriptomics is a next-generation sequencing technology that combines the strengths of transcriptome-wide RNA-sequencing with histological imaging to generate spatial maps of gene expression in intact tissue sections. The 10x Genomics Visium and Visium-Immunofluorescence (Visium-IF) platforms are widely available commercial technologies for quantifying spatially-resolved gene expression. These technologies directly couple gene expression with high resolution histological or immunofluorescence images that contain rich morphological information about the tissue section. However, extracting and integrating image features with gene expression data remains challenging. __Results__: Using MATLAB, we developed VistoSeg, which is a pipeline to process, analyze, and interactively visualize the high-resolution images from the 10x Genomics Visium and Visium-IF platforms. The output from VistoSeg can then be integrated with the spatial-molecular information in downstream analyses using common programming languages, such as R or Python. __Conclusion__: VistoSeg provides user-friendly tools for integrating image-derived metrics from histological and immunofluorescent images with spatially-resolved gene expression data. This integrated approach can advance our understanding of the transcriptional landscape within tissue architecture. VistoSeg is freely available at http://research.libd.org/VistoSeg/. __Impact Statement__: Technologies for measuring gene activity levels, referred to as gene expression, have been evolving over decades and are the core of the transcriptomics subfield within genomics. The first report describing individual cell gene expression is from 2009 and as a method it became commercially available in 2014. While single cell transcriptomics increased our resolution beyond homogenate tissue, the advent of spatial transcriptomics technologies and commercial availability of spatial gene expression platforms, such as Visium, has facilitated studying gene expression in anatomical context. Visium measures local gene expression within the histological organization of single 6.5 mm2 cryosection of tissue. Spatially-resolved transcriptomics provides a new challenge: integrating spatial gene expression with high resolution tissue images (brightfield histology or fluorescent antibody staining). VistoSeg image processing software is compatible with both Visium and Visium-IF from 10x Genomics, which are spatially-resolved transcriptomics assays employing histological and immunofluorescent images, respectively. From these images, the number of cells, identity of cell types, and other image-derived markers can be obtained for thousands of 2,375 Âµm2 spots, where genome-wide gene expression is also measured. VistoSeg provides tools that enable processing these images in the context of gene expression maps to integrate these two high dimensional data types, and thus help unlock the new frontier in transcriptomics." /><meta property="og:image" content="https://lcolladotor.github.io/publication/preprint_vistoseg/featured.png" /><meta property="og:locale" content="en-us" />

  
    <meta
      property="article:published_time"
      content="2022-05-13T00:00:00&#43;00:00"
    />
  
  
    <meta property="article:modified_time" content="2023-05-09T04:26:54-04:00">
  






    









<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "Article",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://lcolladotor.github.io/publication/preprint_vistoseg/"
  },
  "headline": "VistoSeg: processing utilities for high-resolution Visium/Visium-IF images for spatial transcriptomics data",
  
  "image": [
    "https://lcolladotor.github.io/publication/preprint_vistoseg/featured.png"
  ],
  
  "datePublished": "2022-05-13T00:00:00Z",
  "dateModified": "2023-05-09T04:26:54-04:00",
  
  "author": {
    "@type": "Person",
    "name": "Madhavi Tippani"
  },
  
  "publisher": {
    "@type": "Organization",
    "name": "L. Collado-Torres",
    "logo": {
      "@type": "ImageObject",
      "url": "https://lcolladotor.github.io/media/icon_hu2b32c4ab415f12f472f73c7b0301b0d1_19400_192x192_fill_lanczos_center_3.png"
    }
  },
  "description": "__Background__: Spatial transcriptomics is a next-generation sequencing technology that combines the strengths of transcriptome-wide RNA-sequencing with histological imaging to generate spatial maps of gene expression in intact tissue sections. The 10x Genomics Visium and Visium-Immunofluorescence (Visium-IF) platforms are widely available commercial technologies for quantifying spatially-resolved gene expression. These technologies directly couple gene expression with high resolution histological or immunofluorescence images that contain rich morphological information about the tissue section. However, extracting and integrating image features with gene expression data remains challenging. __Results__: Using MATLAB, we developed VistoSeg, which is a pipeline to process, analyze, and interactively visualize the high-resolution images from the 10x Genomics Visium and Visium-IF platforms. The output from VistoSeg can then be integrated with the spatial-molecular information in downstream analyses using common programming languages, such as R or Python. __Conclusion__: VistoSeg provides user-friendly tools for integrating image-derived metrics from histological and immunofluorescent images with spatially-resolved gene expression data. This integrated approach can advance our understanding of the transcriptional landscape within tissue architecture. VistoSeg is freely available at http://research.libd.org/VistoSeg/. __Impact Statement__: Technologies for measuring gene activity levels, referred to as gene expression, have been evolving over decades and are the core of the transcriptomics subfield within genomics. The first report describing individual cell gene expression is from 2009 and as a method it became commercially available in 2014. While single cell transcriptomics increased our resolution beyond homogenate tissue, the advent of spatial transcriptomics technologies and commercial availability of spatial gene expression platforms, such as Visium, has facilitated studying gene expression in anatomical context. Visium measures local gene expression within the histological organization of single 6.5 mm2 cryosection of tissue. Spatially-resolved transcriptomics provides a new challenge: integrating spatial gene expression with high resolution tissue images (brightfield histology or fluorescent antibody staining). VistoSeg image processing software is compatible with both Visium and Visium-IF from 10x Genomics, which are spatially-resolved transcriptomics assays employing histological and immunofluorescent images, respectively. From these images, the number of cells, identity of cell types, and other image-derived markers can be obtained for thousands of 2,375 Âµm2 spots, where genome-wide gene expression is also measured. VistoSeg provides tools that enable processing these images in the context of gene expression maps to integrate these two high dimensional data types, and thus help unlock the new frontier in transcriptomics."
}
</script>

  

  




  
  
  

  
  

  


  
  <title>VistoSeg: processing utilities for high-resolution Visium/Visium-IF images for spatial transcriptomics data | L. Collado-Torres</title>

  
  
  
  











</head>


<body id="top" data-spy="scroll" data-offset="70" data-target="#TableOfContents" class="page-wrapper   " data-wc-page-id="4aff11163268b825a412a2dfd3341764" >

  
  
  
  
  
  
  
  
  
  <script src="/js/wowchemy-init.min.028535d572102c468e4bcd43201c3601.js"></script>

  


<aside class="search-modal" id="search">
  <div class="container">
    <section class="search-header">

      <div class="row no-gutters justify-content-between mb-3">
        <div class="col-6">
          <h1>Search</h1>
        </div>
        <div class="col-6 col-search-close">
          <a class="js-search" href="#" aria-label="Close"><i class="fas fa-times-circle text-muted" aria-hidden="true"></i></a>
        </div>
      </div>

      <div id="search-box">
        
        <input name="q" id="search-query" placeholder="Search..." autocapitalize="off"
        autocomplete="off" autocorrect="off" spellcheck="false" type="search" class="form-control"
        aria-label="Search...">
        
      </div>

      
      

      

    </section>
    <section class="section-search-results">

      <div id="search-hits">
        
      </div>

    </section>
  </div>
</aside>



  <div class="page-header header--fixed">
  
  
  
  
  












<header>
  <nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id="navbar-main">
    <div class="container-xl">

      
      <div class="d-none d-lg-inline-flex">
        <a class="navbar-brand" href="/">L. Collado-Torres</a>
      </div>
      

      
      <button type="button" class="navbar-toggler" data-toggle="collapse"
              data-target="#navbar-content" aria-controls="navbar-content" aria-expanded="false" aria-label="Toggle navigation">
      <span><i class="fas fa-bars"></i></span>
      </button>
      

      
      <div class="navbar-brand-mobile-wrapper d-inline-flex d-lg-none">
        <a class="navbar-brand" href="/">L. Collado-Torres</a>
      </div>
      

      
      
      <div class="navbar-collapse main-menu-item collapse justify-content-start" id="navbar-content">

        
        <ul class="navbar-nav d-md-inline-flex">
          

          

          
          
          
            
          

          

          
          
          
          

          
            
              
              
            
            
              
              
              
                
              
              
            
          

          <li class="nav-item">
            <a class="nav-link " href="/#about"><span>Home</span></a>
          </li>

          
          

          

          
          
          
            
          

          

          
          
          
          

          
            
              
              
            
            
              
              
              
                
              
              
            
          

          <li class="nav-item">
            <a class="nav-link " href="/#team"><span>Team</span></a>
          </li>

          
          

          

          
          
          
            
          

          

          
          
          
          

          
            
              
              
            
            
              
              
              
                
              
              
            
          

          <li class="nav-item">
            <a class="nav-link " href="/#projects"><span>Projects</span></a>
          </li>

          
          

          

          
          
          
            
          

          

          
          
          
          

          
            
              
              
            
            
              
              
              
                
              
              
            
          

          <li class="nav-item">
            <a class="nav-link " href="/#talks_favorite"><span>Talks</span></a>
          </li>

          
          

          

          
          
          
            
          

          

          
          
          
          

          
            
              
              
            
            
              
              
              
                
              
              
            
          

          <li class="nav-item">
            <a class="nav-link " href="/#publications"><span>Publications</span></a>
          </li>

          
          

          

          
          
          
            
          

          

          
          
          
          

          
            
              
              
            
            
              
              
              
                
              
              
            
          

          <li class="nav-item">
            <a class="nav-link " href="/#blog"><span>Blog</span></a>
          </li>

          
          

          

          
          
          
            
          

          

          
          
          
          

          
            
              
              
            
            
              
              
              
                
              
              
            
          

          <li class="nav-item">
            <a class="nav-link " href="/#teaching"><span>Teaching</span></a>
          </li>

          
          

          

          
          
          

          

          
          
          
          

          
            
              
              
            
            
              
              
              
                
              
              
            
          

          <li class="nav-item">
            <a class="nav-link " href="/#cv"><span>CV</span></a>
          </li>

          
          

          

          
          
          
            
          

          

          
          
          
          

          
            
              
              
            
            
              
              
              
                
              
              
            
          

          <li class="nav-item">
            <a class="nav-link " href="/#contact"><span>Contact</span></a>
          </li>

          
          

        

          
        </ul>
      </div>

      <ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2">

        
        
          
        

        
        
        
        <li class="nav-item">
          <a class="nav-link js-search" href="#" aria-label="Search"><i class="fas fa-search" aria-hidden="true"></i></a>
        </li>
        

        
        
        
        <li class="nav-item dropdown theme-dropdown">
          <a href="#" class="nav-link" data-toggle="dropdown" aria-haspopup="true" aria-label="Display preferences">
            <i class="fas fa-moon" aria-hidden="true"></i>
          </a>
          <div class="dropdown-menu">
            <a href="#" class="dropdown-item js-set-theme-light">
              <span>Light</span>
            </a>
            <a href="#" class="dropdown-item js-set-theme-dark">
              <span>Dark</span>
            </a>
            <a href="#" class="dropdown-item js-set-theme-auto">
              <span>Automatic</span>
            </a>
          </div>
        </li>
        

        
        

      </ul>

    </div>
  </nav>
</header>


  </div>

  <div class="page-body">
    
    
    

    








<div class="pub">

  






















  
  


<div class="article-container pt-3">
  <h1>VistoSeg: processing utilities for high-resolution Visium/Visium-IF images for spatial transcriptomics data</h1>

  

  


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/madhavi-tippani/">Madhavi Tippani</a></span>, <span >
      <a href="/authors/heena-r.-divecha/">Heena R. Divecha</a></span>, <span >
      <a href="/authors/joseph-l.-catallini-ii/">Joseph L. Catallini II</a></span>, <span >
      <a href="/authors/sang-ho-kwon/">Sang Ho Kwon</a></span>, <span >
      <a href="/authors/lukas-m.-weber/">Lukas M. Weber</a></span>, <span >
      <a href="/authors/abby-spangler/">Abby Spangler</a></span>, <span >
      <a href="/authors/andrew-e.-jaffe/">Andrew E. Jaffe</a></span>, <span >
      <a href="/authors/stephanie-c.-hicks/">Stephanie C. Hicks</a></span>, <span >
      <a href="/authors/keri-martinowich/">Keri Martinowich</a></span>, <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >
      <a href="/authors/stephanie-c.-page-dagger/">Stephanie C. Page &amp;dagger;</a></span>, <span >
      <a href="/authors/kristen-r.-maynard-dagger/">Kristen R. Maynard &amp;dagger;</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    May, 2022
  </span>
  

  

  

  
  
  
  

  
  

</div>

  




<div class="btn-links mb-3">
  
  








  





<a href="#" class="btn btn-outline-primary btn-page-header js-cite-modal"
        data-filename="/publication/preprint_vistoseg/cite.bib">
  Cite
</a>





  
  <a class="btn btn-outline-primary btn-page-header" href="/project/spatial/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header" href="https://doi.org/10.1101/2021.08.04.452489" target="_blank" rel="noopener">
  DOI
</a>


  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header" href="https://www.biorxiv.org/content/10.1101/2021.08.04.452489v2" target="_blank" rel="noopener">
    Preprint</a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header" href="https://github.com/LieberInstitute/VistoSeg" target="_blank" rel="noopener">
    Code</a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header" href="http://research.libd.org/VistoSeg/" target="_blank" rel="noopener">
    Documentation</a>


</div>


</div>


<div class="article-header article-container featured-image-wrapper mt-4 mb-4" style="max-width: 720px; max-height: 387px;">
  <div style="position: relative">
    <img src="/publication/preprint_vistoseg/featured_huc934cca8b6133a3bc1a854249f7ce153_1656983_720x2500_fit_q75_h2_lanczos_3.webp" width="720" height="387" alt="" class="featured-image">
    <span class="article-header-caption">Image credit: <a href="https://doi.org/10.1101/2021.08.04.452489" target="_blank" rel="noopener"><strong>bioRxiv</strong></a></span>
  </div>
</div>



  <div class="article-container">

    
    <h3>Abstract</h3>
    <p class="pub-abstract"><strong>Background</strong>: Spatial transcriptomics is a next-generation sequencing technology that combines the strengths of transcriptome-wide RNA-sequencing with histological imaging to generate spatial maps of gene expression in intact tissue sections. The 10x Genomics Visium and Visium-Immunofluorescence (Visium-IF) platforms are widely available commercial technologies for quantifying spatially-resolved gene expression. These technologies directly couple gene expression with high resolution histological or immunofluorescence images that contain rich morphological information about the tissue section. However, extracting and integrating image features with gene expression data remains challenging. <strong>Results</strong>: Using MATLAB, we developed VistoSeg, which is a pipeline to process, analyze, and interactively visualize the high-resolution images from the 10x Genomics Visium and Visium-IF platforms. The output from VistoSeg can then be integrated with the spatial-molecular information in downstream analyses using common programming languages, such as R or Python. <strong>Conclusion</strong>: VistoSeg provides user-friendly tools for integrating image-derived metrics from histological and immunofluorescent images with spatially-resolved gene expression data. This integrated approach can advance our understanding of the transcriptional landscape within tissue architecture. VistoSeg is freely available at <a href="http://research.libd.org/VistoSeg/" target="_blank" rel="noopener">http://research.libd.org/VistoSeg/</a>. <strong>Impact Statement</strong>: Technologies for measuring gene activity levels, referred to as gene expression, have been evolving over decades and are the core of the transcriptomics subfield within genomics. The first report describing individual cell gene expression is from 2009 and as a method it became commercially available in 2014. While single cell transcriptomics increased our resolution beyond homogenate tissue, the advent of spatial transcriptomics technologies and commercial availability of spatial gene expression platforms, such as Visium, has facilitated studying gene expression in anatomical context. Visium measures local gene expression within the histological organization of single 6.5 mm2 cryosection of tissue. Spatially-resolved transcriptomics provides a new challenge: integrating spatial gene expression with high resolution tissue images (brightfield histology or fluorescent antibody staining). VistoSeg image processing software is compatible with both Visium and Visium-IF from 10x Genomics, which are spatially-resolved transcriptomics assays employing histological and immunofluorescent images, respectively. From these images, the number of cells, identity of cell types, and other image-derived markers can be obtained for thousands of 2,375 Âµm2 spots, where genome-wide gene expression is also measured. VistoSeg provides tools that enable processing these images in the context of gene expression maps to integrate these two high dimensional data types, and thus help unlock the new frontier in transcriptomics.</p>
    

    
    
    <div class="row">
      <div class="col-md-1"></div>
      <div class="col-md-10">
        <div class="row">
          <div class="col-12 col-md-3 pub-row-heading">Type</div>
          <div class="col-12 col-md-9">
            <a href="/publication/#3">
              Preprint
            </a>
          </div>
        </div>
      </div>
      <div class="col-md-1"></div>
    </div>
    <div class="d-md-none space-below"></div>
    

    
    <div class="row">
      <div class="col-md-1"></div>
      <div class="col-md-10">
        <div class="row">
          <div class="col-12 col-md-3 pub-row-heading">Publication</div>
          <div class="col-12 col-md-9"><em>bioRxiv</em></div>
        </div>
      </div>
      <div class="col-md-1"></div>
    </div>
    <div class="d-md-none space-below"></div>
    

    <div class="space-below"></div>

    <div class="article-style"><blockquote class="twitter-tweet"><p lang="en" dir="ltr">Excited to share my 1st, first-author paper: <a href="https://t.co/x7SInbaVcy">https://t.co/x7SInbaVcy</a>. Learned &amp; accomplished so much through this process - can now call myself a âbookdownâ, âLaTeXâ, â<a href="https://twitter.com/overleaf?ref_src=twsrc%5Etfw">@Overleaf</a>â, â<a href="https://twitter.com/biorxiv?ref_src=twsrc%5Etfw">@bioRxiv</a>, â<a href="https://twitter.com/git?ref_src=twsrc%5Etfw">@Git</a>â expert. Thanks to my mentors <a href="https://twitter.com/lcolladotor?ref_src=twsrc%5Etfw">@lcolladotor</a> , <a href="https://twitter.com/martinowk?ref_src=twsrc%5Etfw">@martinowk</a> , <a href="https://twitter.com/CerceoPage?ref_src=twsrc%5Etfw">@CerceoPage</a> , <a href="https://twitter.com/kr_maynard?ref_src=twsrc%5Etfw">@kr_maynard</a> <a href="https://t.co/rDcyllvylz">pic.twitter.com/rDcyllvylz</a></p>&mdash; MadhaviTippani (@MadhaviTippani) <a href="https://twitter.com/MadhaviTippani/status/1423734212729921536?ref_src=twsrc%5Etfw">August 6, 2021</a></blockquote>
<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>


</div>

    





<div class="article-tags">
  
  <a class="badge badge-light" href="/tag/spatial/">spatial</a>
  
  <a class="badge badge-light" href="/tag/vistoseg/">VistoSeg</a>
  
  <a class="badge badge-light" href="/tag/spatialdlpfc/">spatialDLPFC</a>
  
</div>



<div class="share-box">
  <ul class="share">
    
      
      
      
        
      
      
      
      
      
      
      
      <li>
        <a href="https://twitter.com/intent/tweet?url=https%3A%2F%2Flcolladotor.github.io%2Fpublication%2Fpreprint_vistoseg%2F&amp;text=VistoSeg%3A&#43;processing&#43;utilities&#43;for&#43;high-resolution&#43;Visium%2FVisium-IF&#43;images&#43;for&#43;spatial&#43;transcriptomics&#43;data" target="_blank" rel="noopener" class="share-btn-twitter" aria-label="twitter">
          <i class="fab fa-twitter"></i>
        </a>
      </li>
    
      
      
      
        
      
      
      
      
      
      
      
      <li>
        <a href="https://www.facebook.com/sharer.php?u=https%3A%2F%2Flcolladotor.github.io%2Fpublication%2Fpreprint_vistoseg%2F&amp;t=VistoSeg%3A&#43;processing&#43;utilities&#43;for&#43;high-resolution&#43;Visium%2FVisium-IF&#43;images&#43;for&#43;spatial&#43;transcriptomics&#43;data" target="_blank" rel="noopener" class="share-btn-facebook" aria-label="facebook">
          <i class="fab fa-facebook"></i>
        </a>
      </li>
    
      
      
      
        
      
      
      
      
      
      
      
        
      
      <li>
        <a href="mailto:?subject=VistoSeg%3A%20processing%20utilities%20for%20high-resolution%20Visium%2FVisium-IF%20images%20for%20spatial%20transcriptomics%20data&amp;body=https%3A%2F%2Flcolladotor.github.io%2Fpublication%2Fpreprint_vistoseg%2F" target="_blank" rel="noopener" class="share-btn-email" aria-label="envelope">
          <i class="fas fa-envelope"></i>
        </a>
      </li>
    
      
      
      
        
      
      
      
      
      
      
      
      <li>
        <a href="https://www.linkedin.com/shareArticle?url=https%3A%2F%2Flcolladotor.github.io%2Fpublication%2Fpreprint_vistoseg%2F&amp;title=VistoSeg%3A&#43;processing&#43;utilities&#43;for&#43;high-resolution&#43;Visium%2FVisium-IF&#43;images&#43;for&#43;spatial&#43;transcriptomics&#43;data" target="_blank" rel="noopener" class="share-btn-linkedin" aria-label="linkedin-in">
          <i class="fab fa-linkedin-in"></i>
        </a>
      </li>
    
      
      
      
        
      
      
      
      
      
      
      
      <li>
        <a href="whatsapp://send?text=VistoSeg%3A&#43;processing&#43;utilities&#43;for&#43;high-resolution&#43;Visium%2FVisium-IF&#43;images&#43;for&#43;spatial&#43;transcriptomics&#43;data%20https%3A%2F%2Flcolladotor.github.io%2Fpublication%2Fpreprint_vistoseg%2F" target="_blank" rel="noopener" class="share-btn-whatsapp" aria-label="whatsapp">
          <i class="fab fa-whatsapp"></i>
        </a>
      </li>
    
      
      
      
        
      
      
      
      
      
      
      
      <li>
        <a href="https://service.weibo.com/share/share.php?url=https%3A%2F%2Flcolladotor.github.io%2Fpublication%2Fpreprint_vistoseg%2F&amp;title=VistoSeg%3A&#43;processing&#43;utilities&#43;for&#43;high-resolution&#43;Visium%2FVisium-IF&#43;images&#43;for&#43;spatial&#43;transcriptomics&#43;data" target="_blank" rel="noopener" class="share-btn-weibo" aria-label="weibo">
          <i class="fab fa-weibo"></i>
        </a>
      </li>
    
  </ul>
</div>











  
  
    




  
    




  
    




  
    




  
    




  
    




  
    




  
    




  
    




  
    



  
  
  
    
  
  
  
  <div class="media author-card content-widget-hr">
    
      
      <a href="https://lcolladotor.github.io/"><img class="avatar mr-3 avatar-circle" src="/authors/admin/avatar_hu61670ae1305c5c407abf5eedc95c7971_127149_270x270_fill_q75_lanczos_center.JPG" alt="Leonardo Collado-Torres"></a>
    

    <div class="media-body">
      <h5 class="card-title"><a href="https://lcolladotor.github.io/">Leonardo Collado-Torres</a></h5>
      <h6 class="card-subtitle">Investigator @ LIBD, Assistant Professor, Department of Biostatistics @ JHBSPH</h6>
      <p class="card-text"><a href="https://twitter.com/search?q=%23rstats&amp;src=hashtag_click" target="_blank" rel="noopener">#rstats</a> <a href="https://twitter.com/Bioconductor" target="_blank" rel="noopener">@Bioconductor</a>/ð§  genomics <a href="https://twitter.com/LieberInstitute" target="_blank" rel="noopener">@LieberInstitute</a>/<a href="https://twitter.com/lcgunam" target="_blank" rel="noopener">@lcgunam</a> <a href="https://twitter.com/jhubiostat" target="_blank" rel="noopener">@jhubiostat</a> <a href="https://twitter.com/jtleek" target="_blank" rel="noopener">@jtleek</a> <a href="https://twitter.com/andrewejaffe" target="_blank" rel="noopener">@andrewejaffe</a> alumni/<a href="https://twitter.com/LIBDrstats" target="_blank" rel="noopener">@LIBDrstats</a> <a href="https://twitter.com/CDSBMexico" target="_blank" rel="noopener">@CDSBMexico</a> co-founder</p>
      <ul class="network-icon" aria-hidden="true">
  
    
    
    
      
    
    
    
    
    
    <li>
      <a href="mailto:lcolladotor@gmail.com" >
        <i class="fas fa-envelope"></i>
      </a>
    </li>
  
    
    
    
      
    
    
    
    
    
      
    
    <li>
      <a href="http://lcolladotor.github.io" target="_blank" rel="noopener">
        <i class="fas fa-home"></i>
      </a>
    </li>
  
    
    
    
      
    
    
    
    
    
      
    
    <li>
      <a href="https://twitter.com/lcolladotor" target="_blank" rel="noopener">
        <i class="fab fa-twitter"></i>
      </a>
    </li>
  
    
    
    
      
    
    
    
    
    
      
    
    <li>
      <a href="https://www.youtube.com/@lcolladotor/playlists" target="_blank" rel="noopener">
        <i class="fab fa-youtube"></i>
      </a>
    </li>
  
    
    
    
    
    
    
    
      
    
    <li>
      <a href="https://scholar.google.com/citations?user=h57-MykAAAAJ" target="_blank" rel="noopener">
        <i class="ai ai-google-scholar"></i>
      </a>
    </li>
  
    
    
    
      
    
    
    
    
    
      
    
    <li>
      <a href="https://github.com/lcolladotor" target="_blank" rel="noopener">
        <i class="fab fa-github"></i>
      </a>
    </li>
  
    
    
    
    
    
    
    
      
    
    <li>
      <a href="http://orcid.org/0000-0003-2140-308X" target="_blank" rel="noopener">
        <i class="ai ai-orcid"></i>
      </a>
    </li>
  
    
    
    
    
    
    
    
      
    
    <li>
      <a href="https://www.ncbi.nlm.nih.gov/myncbi/leonardo.collado%20torres.1/bibliography/public/" target="_blank" rel="noopener">
        <i class="ai ai-pubmed"></i>
      </a>
    </li>
  
    
    
    
    
    
    
    
      
    
    <li>
      <a href="https://impactstory.org/u/0000-0003-2140-308X" target="_blank" rel="noopener">
        <i class="ai ai-impactstory"></i>
      </a>
    </li>
  
    
    
    
      
    
    
    
    
    
      
    
    <li>
      <a href="https://www.linkedin.com/in/lcollado/" target="_blank" rel="noopener">
        <i class="fab fa-linkedin"></i>
      </a>
    </li>
  
    
    
    
    
    
    
    
      
    
    <li>
      <a href="https://publons.com/a/1262671" target="_blank" rel="noopener">
        <i class="ai ai-publons"></i>
      </a>
    </li>
  
    
    
    
    
    
    
    
      
    
    <li>
      <a href="http://lcolladotor.github.io/cv/en_CV_lcollado.pdf" target="_blank" rel="noopener">
        <i class="ai ai-cv"></i>
      </a>
    </li>
  
</ul>

    </div>
  </div>


  
    




  
    




  







  
  
  

  

  










  </div>
</div>
  </div>

  <div class="page-footer">
    
    
    <div class="container">
      <footer class="site-footer">

  












  
  
  
  
  













  
  
  

  
  
    
  
  
    
  

  

  
  <p class="powered-by copyright-license-text">
    All thoughts and opinions here are my own (Leonardo Collado Torres). The icon was designed by <a href="https://www.linkedin.com/in/mauricio-guzman-6529b551/" target="_blank" rel="noopener">Mauricio GuzmÃ¡n</a> and is inspired by <a href="https://en.wikipedia.org/wiki/Huichol" target="_blank" rel="noopener">Huichol culture</a>; it represents my community building interests. Â© 2011-2024 LCT. This work is licensed under <a href="https://creativecommons.org/licenses/by-nc-sa/4.0" rel="noopener noreferrer" target="_blank">CC BY NC SA 4.0</a>
  </p>
  

  <p class="powered-by footer-license-icons">
    <a href="https://creativecommons.org/licenses/by-nc-sa/4.0" rel="noopener noreferrer" target="_blank" aria-label="Creative Commons">
      <i class="fab fa-creative-commons fa-2x" aria-hidden="true"></i>
      <i class="fab fa-creative-commons-by fa-2x" aria-hidden="true"></i>
      
        <i class="fab fa-creative-commons-nc fa-2x" aria-hidden="true"></i>
      
      
        <i class="fab fa-creative-commons-sa fa-2x" aria-hidden="true"></i>
      
    </a>
  </p>




  
  <p>
      
      
      <script type="text/javascript" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=ra-4ef23e545d41b87c"></script>
      
      

      <script type="text/javascript">
      var addthis_config = {
      	data_ga_property: 'UA-45299226-1',
      	data_ga_social : true,
      	data_track_addressbar: true,
        data_track_addressbar_paths: ['/201*/*/*/*/*']
      };
      var addthis_share = {
      	
      	url_transforms : {
      		shorten: {
      			twitter: 'bitly'
      		}
      	}, 
      	shorteners : {
      		bitly : {}
      	}
      }
      </script>
      
      <center>	 
      <div id="clustrmaps-widget"></div><script type='text/javascript' id='clustrmaps' src='//cdn.clustrmaps.com/map_v2.js?cl=ffffff&w=300&t=m&d=Zo-deincaJGsqhqjIWsujtdoNOMJsnnZLefKmxDKVv8&co=124b73&cmo=dd53ff&cmn=4ec724'></script>
      </center>
  </p>
  

  <p class="powered-by">
    
    
    
      
      
      
      
      
      
      Published with <a href="https://wowchemy.com/?utm_campaign=poweredby" target="_blank" rel="noopener">Wowchemy</a> â the free, <a href="https://github.com/wowchemy/wowchemy-hugo-themes" target="_blank" rel="noopener">open source</a> website builder that empowers creators.
    
  </p>
</footer>

    </div>
    
  </div>

  


<script src="/js/vendor-bundle.min.938a3a7554cd9f6602290411f64d2617.js"></script>




  

  
  

  






  <script src="https://cdn.jsdelivr.net/npm/leaflet@1.7.1/dist/leaflet.min.js" integrity="" crossorigin="anonymous"></script>








  
  <script id="search-hit-fuse-template" type="text/x-template">
    <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
    </div>
  </script>
  
    <script src="https://cdn.jsdelivr.net/gh/krisk/Fuse@v3.2.1/dist/fuse.min.js" integrity="sha512-o38bmzBGX+hD3JHWUFCDA09btWaqrNmoJ3RXLlrysA7PP01Kgs4UlE4MhelE1v5dJR3+cxlR4qQlotsW7jKsnw==" crossorigin="anonymous"></script>
    <script src="https://cdn.jsdelivr.net/gh/julmot/mark.js@8.11.1/dist/jquery.mark.min.js" integrity="sha512-mhbv5DqBMgrWL+32MmsDOt/OAvqr/cHimk6B8y/bx/xS88MVkYGPiVv2ixKVrkywF2qHplNRUvFsAHUdxZ3Krg==" crossorigin="anonymous"></script>
  









  <script id="dsq-count-scr" src="https://fellgernonbit.disqus.com/count.js" async></script>




  
  
  
  
  
  
  







<script id="page-data" type="application/json">{"use_headroom":true}</script>



  <script src="/js/wowchemy-headroom.db4755770454eb63685f8de785c0a172.js" type="module"></script>









  
  


<script src="/en/js/wowchemy.min.2945731fa88332cb0ec33cf47d29879f.js"></script>



  <script src="/js/wowchemy-map.a26e9d2f7238ba5b868384f1c5bc6477.js" type="module"></script>




  
<div id="modal" class="modal fade" role="dialog">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <h5 class="modal-title">Cite</h5>
        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
      </div>
      <div class="modal-body">
        
        <pre><code></code></pre>
      </div>
      <div class="modal-footer">
        <a class="btn btn-outline-primary my-1 js-copy-cite" href="#" target="_blank">
          <i class="fas fa-copy"></i> Copy
        </a>
        <a class="btn btn-outline-primary my-1 js-download-cite" href="#" target="_blank">
          <i class="fas fa-download"></i> Download
        </a>
        <div id="modal-error"></div>
      </div>
    </div>
  </div>
</div>


  <script src="/js/wowchemy-publication.af9327db0521d4a01354bfc8b77a4324.js" type="module"></script>


















</body>
</html>
