<!DOCTYPE html>
<!-- This site was created with Wowchemy. https://www.wowchemy.com -->
<!-- Last Published: April 11, 2024 --><html lang="en-us" >


<head>
  <meta charset="utf-8" />
  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  
  
  
    <meta name="generator" content="Wowchemy 5.7.0 for Hugo" />
  

  
  












  
  










  







  
  
  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
  

  
  
  
    
      
      <link rel="preload" as="style" href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Mono&family=Roboto:wght@400;700&family=Karma&family=Mina&display=swap">
      <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Mono&family=Roboto:wght@400;700&family=Karma&family=Mina&display=swap" media="print" onload="this.media='all'">
    
  

  
  

  
  
    
    <script src="/js/mathjax-config.js"></script>
  

  

  <link rel="stylesheet" href="/css/vendor-bundle.min.047268c6dd09ad74ba54a0ba71837064.css" media="print" onload="this.media='all'">

  
  
  
    
    
      <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.2/css/academicons.min.css" integrity="sha512-KlJCpRsLf+KKu2VQa5vmRuClRFjxc5lXO03ixZt82HZUk41+1I0bD8KBSA0fY290ayMfWYI9udIqeOWSu1/uZg==" crossorigin="anonymous" media="print" onload="this.media='all'">
    

    
    
    
    
      
      
    
    
    

    
    
    
      <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/leaflet@1.7.1/dist/leaflet.min.css" integrity="" crossorigin="anonymous" media="print" onload="this.media='all'">
    

    

    
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
        <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml.js" integrity="" crossorigin="anonymous" async></script>
      
    
      
      

      
      

      
    
      
      

      
      

      
    
  

  
  
  
  
  
  
  <link rel="stylesheet" href="/css/wowchemy.cfa383012e109fa07dd80bafcdf476df.css" />

  
  
  

  
  
  
  
  
  
  
    
    
    <link rel="stylesheet" href="/css/libs/chroma/github-light.min.css" title="hl-light" media="print" onload="this.media='all'" >
    <link rel="stylesheet" href="/css/libs/chroma/dracula.min.css" title="hl-dark" media="print" onload="this.media='all'" disabled>
  

  
  






<script async src="https://www.googletagmanager.com/gtag/js?id=G-3CBVR6GQ07"></script>
<script>
  window.dataLayer = window.dataLayer || [];

  function gtag() {
      dataLayer.push(arguments);
  }

  function trackOutboundLink(url, target) {
    gtag('event', 'click', {
         'event_category': 'outbound',
         'event_label': url,
         'transport_type': 'beacon',
         'event_callback': function () {
           if (target !== '_blank') {
             document.location = url;
           }
         }
    });
    console.debug("Outbound link clicked: " + url);
  }

  function onClickCallback(event) {
    if ((event.target.tagName !== 'A') || (event.target.host === window.location.host)) {
      return;
    }
    trackOutboundLink(event.target, event.target.getAttribute('target'));  
  }

  gtag('js', new Date());
  gtag('config', 'G-3CBVR6GQ07', {});
  gtag('set', {'cookie_flags': 'SameSite=None;Secure'});

  
  document.addEventListener('click', onClickCallback, false);
</script>























  
  
  






  <meta name="author" content="Leonardo Collado-Torres" />





  

<meta name="description" content="R/Bioconductor-powered Team Data Science" />



<link rel="alternate" hreflang="en-us" href="https://lcolladotor.github.io/publication/" />
<link rel="canonical" href="https://lcolladotor.github.io/publication/" />



  <link rel="manifest" href="/manifest.webmanifest" />



<link rel="icon" type="image/png" href="/media/icon_hu2b32c4ab415f12f472f73c7b0301b0d1_19400_32x32_fill_lanczos_center_3.png" />
<link rel="apple-touch-icon" type="image/png" href="/media/icon_hu2b32c4ab415f12f472f73c7b0301b0d1_19400_180x180_fill_lanczos_center_3.png" />

<meta name="theme-color" content="#FF6771" />










  
  






<meta property="twitter:card" content="summary" />

  <meta property="twitter:site" content="@lcolladotor" />
  <meta property="twitter:creator" content="@lcolladotor" />
<meta property="twitter:image" content="https://lcolladotor.github.io/media/icon_hu2b32c4ab415f12f472f73c7b0301b0d1_19400_512x512_fill_lanczos_center_3.png" />
<meta property="og:site_name" content="L. Collado-Torres" />
<meta property="og:url" content="https://lcolladotor.github.io/publication/" />
<meta property="og:title" content="Publications | L. Collado-Torres" />
<meta property="og:description" content="R/Bioconductor-powered Team Data Science" /><meta property="og:image" content="https://lcolladotor.github.io/media/icon_hu2b32c4ab415f12f472f73c7b0301b0d1_19400_512x512_fill_lanczos_center_3.png" /><meta property="og:locale" content="en-us" />

  










  
  
  

  
  
    <link rel="alternate" href="/publication/index.xml" type="application/rss+xml" title="L. Collado-Torres" />
  

  


  
  <title>Publications | L. Collado-Torres</title>

  
  
  
  











</head>


<body id="top" data-spy="scroll" data-offset="70" data-target="#TableOfContents" class="page-wrapper   " data-wc-page-id="3a079e7dad19be978a318345a7749d34" >

  
  
  
  
  
  
  
  
  
  <script src="/js/wowchemy-init.min.028535d572102c468e4bcd43201c3601.js"></script>

  


<aside class="search-modal" id="search">
  <div class="container">
    <section class="search-header">

      <div class="row no-gutters justify-content-between mb-3">
        <div class="col-6">
          <h1>Search</h1>
        </div>
        <div class="col-6 col-search-close">
          <a class="js-search" href="#" aria-label="Close"><i class="fas fa-times-circle text-muted" aria-hidden="true"></i></a>
        </div>
      </div>

      <div id="search-box">
        
        <input name="q" id="search-query" placeholder="Search..." autocapitalize="off"
        autocomplete="off" autocorrect="off" spellcheck="false" type="search" class="form-control"
        aria-label="Search...">
        
      </div>

      
      

      

    </section>
    <section class="section-search-results">

      <div id="search-hits">
        
      </div>

    </section>
  </div>
</aside>



  <div class="page-header header--fixed">
  
  
  
  
  












<header>
  <nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id="navbar-main">
    <div class="container-xl">

      
      <div class="d-none d-lg-inline-flex">
        <a class="navbar-brand" href="/">L. Collado-Torres</a>
      </div>
      

      
      <button type="button" class="navbar-toggler" data-toggle="collapse"
              data-target="#navbar-content" aria-controls="navbar-content" aria-expanded="false" aria-label="Toggle navigation">
      <span><i class="fas fa-bars"></i></span>
      </button>
      

      
      <div class="navbar-brand-mobile-wrapper d-inline-flex d-lg-none">
        <a class="navbar-brand" href="/">L. Collado-Torres</a>
      </div>
      

      
      
      <div class="navbar-collapse main-menu-item collapse justify-content-start" id="navbar-content">

        
        <ul class="navbar-nav d-md-inline-flex">
          

          

          
          
          
            
          

          

          
          
          
          

          
            
              
              
            
            
              
              
              
                
              
              
            
          

          <li class="nav-item">
            <a class="nav-link " href="/#about"><span>Home</span></a>
          </li>

          
          

          

          
          
          
            
          

          

          
          
          
          

          
            
              
              
            
            
              
              
              
                
              
              
            
          

          <li class="nav-item">
            <a class="nav-link " href="/#team"><span>Team</span></a>
          </li>

          
          

          

          
          
          
            
          

          

          
          
          
          

          
            
              
              
            
            
              
              
              
                
              
              
            
          

          <li class="nav-item">
            <a class="nav-link " href="/#projects"><span>Projects</span></a>
          </li>

          
          

          

          
          
          
            
          

          

          
          
          
          

          
            
              
              
            
            
              
              
              
                
              
              
            
          

          <li class="nav-item">
            <a class="nav-link " href="/#talks_favorite"><span>Talks</span></a>
          </li>

          
          

          

          
          
          
            
          

          

          
          
          
          

          
            
              
              
            
            
              
              
              
                
              
              
            
          

          <li class="nav-item">
            <a class="nav-link " href="/#publications"><span>Publications</span></a>
          </li>

          
          

          

          
          
          
            
          

          

          
          
          
          

          
            
              
              
            
            
              
              
              
                
              
              
            
          

          <li class="nav-item">
            <a class="nav-link " href="/#blog"><span>Blog</span></a>
          </li>

          
          

          

          
          
          
            
          

          

          
          
          
          

          
            
              
              
            
            
              
              
              
                
              
              
            
          

          <li class="nav-item">
            <a class="nav-link " href="/#teaching"><span>Teaching</span></a>
          </li>

          
          

          

          
          
          

          

          
          
          
          

          
            
              
              
            
            
              
              
              
                
              
              
            
          

          <li class="nav-item">
            <a class="nav-link " href="/#cv"><span>CV</span></a>
          </li>

          
          

          

          
          
          
            
          

          

          
          
          
          

          
            
              
              
            
            
              
              
              
                
              
              
            
          

          <li class="nav-item">
            <a class="nav-link " href="/#contact"><span>Contact</span></a>
          </li>

          
          

        

          
        </ul>
      </div>

      <ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2">

        
        
          
        

        
        
        
        <li class="nav-item">
          <a class="nav-link js-search" href="#" aria-label="Search"><i class="fas fa-search" aria-hidden="true"></i></a>
        </li>
        

        
        
        
        <li class="nav-item dropdown theme-dropdown">
          <a href="#" class="nav-link" data-toggle="dropdown" aria-haspopup="true" aria-label="Display preferences">
            <i class="fas fa-moon" aria-hidden="true"></i>
          </a>
          <div class="dropdown-menu">
            <a href="#" class="dropdown-item js-set-theme-light">
              <span>Light</span>
            </a>
            <a href="#" class="dropdown-item js-set-theme-dark">
              <span>Dark</span>
            </a>
            <a href="#" class="dropdown-item js-set-theme-auto">
              <span>Automatic</span>
            </a>
          </div>
        </li>
        

        
        

      </ul>

    </div>
  </nav>
</header>


  </div>

  <div class="page-body">
    
    
    

    
















  

  
  
  
    
  
<div class="universal-wrapper pt-3">
  <h1>Publications</h1>

  

  
</div>



<div class="universal-wrapper">
  <div class="row">
    <div class="col-lg-12">

      

      
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      

      <div class="form-row mb-4">
        <div class="col-auto">
          <input type="search" class="filter-search form-control form-control-sm" placeholder="Search..." autocapitalize="off"
          autocomplete="off" autocorrect="off" role="textbox" spellcheck="false">
        </div>
        <div class="col-auto">
          <select class="pub-filters pubtype-select form-control form-control-sm" data-filter-group="pubtype">
            <option value="*">Type</option>
            
            
            <option value=".pubtype-0">
              Uncategorized
            </option>
            
            <option value=".pubtype-2">
              Journal article
            </option>
            
            <option value=".pubtype-3">
              Preprint
            </option>
            
            <option value=".pubtype-6">
              Book section
            </option>
            
          </select>
        </div>
        <div class="col-auto">
          <select class="pub-filters form-control form-control-sm" data-filter-group="year">
            <option value="*">Date</option>
            
            
            
            <option value=".year-2023">
              2023
            </option>
            
            <option value=".year-2022">
              2022
            </option>
            
            <option value=".year-2021">
              2021
            </option>
            
            <option value=".year-2020">
              2020
            </option>
            
            <option value=".year-2019">
              2019
            </option>
            
            <option value=".year-2018">
              2018
            </option>
            
            <option value=".year-2017">
              2017
            </option>
            
            <option value=".year-2016">
              2016
            </option>
            
            <option value=".year-2015">
              2015
            </option>
            
            <option value=".year-2014">
              2014
            </option>
            
            <option value=".year-2013">
              2013
            </option>
            
            <option value=".year-2011">
              2011
            </option>
            
            <option value=".year-2010">
              2010
            </option>
            
            
          </select>
        </div>
      </div>

      <div id="container-publications">
        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-3 year-2023">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/boyi-guo/">Boyi Guo</a></span>, <span >
      <a href="/authors/lahuuki/">Louise A. Huuki-Myers</a></span>, <span >
      <a href="/authors/melissa-grant-peters/">Melissa Grant-Peters</a></span>, <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >
      <a href="/authors/stephanie-c.-hicks/">Stephanie C. Hicks</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    June, 2023
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>bioRxiv</em>
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/preprint_escher/" >
      <div class="img-hover-zoom">
        <img src="/publication/preprint_escher/featured_hu1249d03ff0cb7dd6002542043f189317_104665_808x455_fill_q75_h2_lanczos_smart1.webp" height="455" width="808"
            class="article-banner" alt="escheR: Unified multi-dimensional visualizations with Gestalt principles" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/preprint_escher/" >escheR: Unified multi-dimensional visualizations with Gestalt principles</a>
  </div>

  
  <a href="/publication/preprint_escher/"  class="summary-link">
    <div class="article-style">
      <p>The creation of effective visualizations is a fundamental component of data analysis. In biomedical research, new challenges are emerging to visualize multi-dimensional data in a 2D space, but current data visualization tools have limited capabilities. To address this problem, we leverage Gestalt principles to improve the design and interpretability of multi-dimensional data in 2D data visualizations, layering aesthetics to display multiple variables. The proposed visualization can be applied to spatially-resolved transcriptomics data, but also broadly to data visualized in 2D space, such as embedding visualizations. We provide an open source R package escheR, which is built off of the state-of-the-art ggplot2 visualization framework and can be seamlessly integrated into genomics toolboxes and workflows.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/preprint_escher/cite.bib">
  Cite
</a>





  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/spatial/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1101/22023.03.18.533302" target="_blank" rel="noopener">
  DOI
</a>


  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.biorxiv.org/content/10.1101/22023.03.18.533302v2" target="_blank" rel="noopener">
    Preprint</a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/boyiguo1/Manuscript_escheR" target="_blank" rel="noopener">
    Code</a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="http://bioconductor.org/packages/escheR" target="_blank" rel="noopener">
    Bioconductor</a>


  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2023">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/erik-d.-nelson/">Erik D. Nelson</a></span>, <span >
      <a href="/authors/kristen-r.-maynard/">Kristen R. Maynard</a></span>, <span >
      <a href="/authors/kyndall-r.-nicholas/">Kyndall R. Nicholas</a></span>, <span >
      <a href="/authors/matthew-n.-tran/">Matthew N. Tran</a></span>, <span >
      <a href="/authors/heena-r.-divecha/">Heena R. Divecha</a></span>, <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >
      <a href="/authors/stephanie-c.-hicks-dagger/">Stephanie C. Hicks &amp;dagger;</a></span>, <span >
      <a href="/authors/keri-martinowich-dagger/">Keri Martinowich &amp;dagger;</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    May, 2023
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Hippocampus</em>
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/2023-05_hpc_erik/" >
      <div class="img-hover-zoom">
        <img src="/publication/2023-05_hpc_erik/featured_hu229a0bd74045d5627d4b637bb159ca5a_470264_808x455_fill_q75_h2_lanczos_smart1_3.webp" height="455" width="808"
            class="article-banner" alt="Activity-regulated gene expression across cell types of the mouse hippocampus" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/2023-05_hpc_erik/" >Activity-regulated gene expression across cell types of the mouse hippocampus</a>
  </div>

  
  <a href="/publication/2023-05_hpc_erik/"  class="summary-link">
    <div class="article-style">
      <p>Activity-regulated gene (ARG) expression patterns in the hippocampus (HPC) regulate synaptic plasticity, learning, and memory, and are linked to both risk and treatment responses for many neuropsychiatric disorders. The HPC contains discrete classes of neurons with specialized functions, but cell type-specific activity-regulated transcriptional programs are not well characterized. Here, we used single-nucleus RNA-sequencing (snRNA-seq) in a mouse model of acute electroconvulsive seizures (ECS) to identify cell type-specific molecular signatures associated with induced activity in HPC neurons. We used unsupervised clustering and a priori marker genes to computationally annotate 15,990 high-quality HPC neuronal nuclei from N = 4 mice across all major HPC subregions and neuron types. Activity-induced transcriptomic responses were divergent across neuron populations, with dentate granule cells being particularly responsive to activity. Differential expression analysis identified both upregulated and downregulated cell type-specific gene sets in neurons following ECS. Within these gene sets, we identified enrichment of pathways associated with varying biological processes such as synapse organization, cellular signaling, and transcriptional regulation. Finally, we used matrix factorization to reveal continuous gene expression patterns differentially associated with cell type, ECS, and biological processes. This work provides a rich resource for interrogating activity-regulated transcriptional responses in HPC neurons at single-nuclei resolution in the context of ECS, which can provide biological insight into the roles of defined neuronal subtypes in HPC function.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/2023-05_hpc_erik/cite.bib">
  Cite
</a>













<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/https://doi.org/10.1002/hipo.23548" target="_blank" rel="noopener">
  DOI
</a>


  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.biorxiv.org/content/10.1101/2022.11.23.517593v1" target="_blank" rel="noopener">
    Preprint</a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/Erik-D-Nelson/ARG_HPC_snRNAseq" target="_blank" rel="noopener">
    Code</a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="https://libd.shinyapps.io/2022_HPC_ARG/" target="_blank" rel="noopener">
    iSEE app</a>


  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-3 year-2023">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/sang-ho-kwon/">Sang Ho Kwon</a></span>, <span >
      <a href="/authors/sowmya-parthiban/">Sowmya Parthiban</a></span>, <span >
      <a href="/authors/madhavi-tippani/">Madhavi Tippani</a></span>, <span >
      <a href="/authors/heena-r.-divecha/">Heena R. Divecha</a></span>, <span >
      <a href="/authors/nickeagles/">Nicholas J. Eagles</a></span>, <span >
      <a href="/authors/jashandeep-s.-lobana/">Jashandeep S. Lobana</a></span>, <span >
      <a href="/authors/stephen-r.-williams/">Stephen R. Williams</a></span>, <span >
      <a href="/authors/michelle-mak/">Michelle Mak</a></span>, <span >
      <a href="/authors/rahul-a.-bharadwaj/">Rahul A. Bharadwaj</a></span>, <span >
      <a href="/authors/joel-e.-kleinman/">Joel E. Kleinman</a></span>, <span >
      <a href="/authors/thomas-m.-hyde/">Thomas M. Hyde</a></span>, <span >
      <a href="/authors/stephanie-c.-page/">Stephanie C. Page</a></span>, <span >
      <a href="/authors/stephanie-c.-hicks-dagger/">Stephanie C. Hicks &amp;dagger;</a></span>, <span >
      <a href="/authors/keri-martinowich-dagger/">Keri Martinowich &amp;dagger;</a></span>, <span >
      <a href="/authors/kristen-r.-maynard-dagger/">Kristen R. Maynard &amp;dagger;</a></span>, <span >
      <a href="/authors/leonardo-collado-torres-dagger/">Leonardo Collado-Torres &amp;dagger;</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    April, 2023
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>bioRxiv</em>
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/preprint_visium_spg_ad/" >
      <div class="img-hover-zoom">
        <img src="/publication/preprint_visium_spg_ad/featured_hu30d9e586e357db086be45d934480aedd_4677360_808x455_fill_q75_h2_lanczos_smart1_3.webp" height="455" width="808"
            class="article-banner" alt="Influence of Alzheimer’s disease related neuropathology on local microenvironment gene expression in the human inferior temporal cortex" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/preprint_visium_spg_ad/" >Influence of Alzheimer’s disease related neuropathology on local microenvironment gene expression in the human inferior temporal cortex</a>
  </div>

  
  <a href="/publication/preprint_visium_spg_ad/"  class="summary-link">
    <div class="article-style">
      <p>Neuropathological lesions in the brains of individuals affected with neurodegenerative disorders are hypothesized to trigger molecular and cellular processes that disturb homeostasis of local microenvironments. Here, we applied the 10x Genomics Visium Spatial Proteogenomics (Visium-SPG) platform, which measures spatial gene expression coupled with immunofluorescence protein co-detection, in post-mortem human brain tissue from individuals with late-stage Alzheimer’s disease (AD) to investigate changes in spatial gene expression with respect to amyloid-β (Aβ) and hyperphosphorylated tau (pTau) pathology. We identified Aβ-associated transcriptomic signatures in the human inferior temporal cortex (ITC) during late-stage AD, which we further investigated at cellular resolution with combined immunofluorescence and single molecule fluorescent in situ hybridization (smFISH) co-detection technology. We present a workflow for analysis of Visium-SPG data and demonstrate the power of multi-omic profiling to identify spatially-localized changes in molecular dynamics that are linked to pathology in human brain disease. We provide the scientific community with web-based, interactive resources to access the datasets of the spatially resolved AD-related transcriptomes at <a href="https://research.libd.org/Visium_SPG_AD/" target="_blank" rel="noopener">https://research.libd.org/Visium_SPG_AD/</a>.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/preprint_visium_spg_ad/cite.bib">
  Cite
</a>





  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/spatial/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1101/2023.04.20.537710" target="_blank" rel="noopener">
  DOI
</a>


  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.biorxiv.org/content/10.1101/2023.04.20.537710v1" target="_blank" rel="noopener">
    Preprint</a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/LieberInstitute/Visium_SPG_AD" target="_blank" rel="noopener">
    Code</a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="http://research.libd.org/Visium_SPD_AD/" target="_blank" rel="noopener">
    Documentation</a>


  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-3 year-2023">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/s-garcia-ruiz/">S García-Ruiz</a></span>, <span >
      <a href="/authors/d-zhang/">D Zhang</a></span>, <span >
      <a href="/authors/e-k-gustavsson-g-rocamora-perez/">E K Gustavsson, G Rocamora-Perez</a></span>, <span >
      <a href="/authors/m-grant-peters/">M Grant-Peters</a></span>, <span >
      <a href="/authors/a-fairbrother-browne/">A Fairbrother-Browne</a></span>, <span >
      <a href="/authors/r-h-reynolds/">R H Reynolds</a></span>, <span >
      <a href="/authors/j-w-brenton/">J W Brenton</a></span>, <span >
      <a href="/authors/a-l-gil-martinez/">A L Gil-Martínez</a></span>, <span >
      <a href="/authors/z-chen/">Z Chen</a></span>, <span >
      <a href="/authors/d-c-rio/">D C Rio</a></span>, <span >
      <a href="/authors/j-a-botia/">J A Botia</a></span>, <span >
      <a href="/authors/s-guelfi/">S Guelfi</a></span>, <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >
      <a href="/authors/m-ryten/">M Ryten</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    March, 2023
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>bioRxiv</em>
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/preprint_splicing_noise/" >
      <div class="img-hover-zoom">
        <img src="/publication/preprint_splicing_noise/featured_hu20ad6b0b44ec64c831a2de128982708b_681867_808x455_fill_q75_h2_lanczos_smart1_3.webp" height="455" width="808"
            class="article-banner" alt="Splicing accuracy varies across human introns, tissues and age" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/preprint_splicing_noise/" >Splicing accuracy varies across human introns, tissues and age</a>
  </div>

  
  <a href="/publication/preprint_splicing_noise/"  class="summary-link">
    <div class="article-style">
      <p>Alternative splicing impacts most multi-exonic human genes. Inaccuracies during this process may have an important role in ageing and disease. Here, we investigated mis-splicing using RNA-sequencing data from ~14K control samples and 42 human body sites, focusing on split reads partially mapping to known transcripts in annotation. We show that mis-splicing occurs at different rates across introns and tissues and that these splicing inaccuracies are primarily affected by the abundance of core components of the spliceosome assembly and its regulators. Using publicly available data on short-hairpin RNA-knockdowns of numerous spliceosomal components and related regulators, we found support for the importance of RNA-binding proteins in mis-splicing. We also demonstrated that age is positively correlated with mis-splicing, and it affects genes implicated in neurodegenerative diseases. This in-depth characterisation of mis-splicing can have important implications for our understanding of the role of splicing inaccuracies in human disease and the interpretation of long-read RNA-sequencing data.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/preprint_splicing_noise/cite.bib">
  Cite
</a>





  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/recount3/">
    Project
  </a>
  

  
    
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1101/2023.03.29.534370" target="_blank" rel="noopener">
  DOI
</a>


  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.biorxiv.org/content/10.1101/2023.03.29.534370v1" target="_blank" rel="noopener">
    Preprint</a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/SoniaRuiz/splicing-accuracy-manuscript" target="_blank" rel="noopener">
    Code</a>


  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-3 year-2023">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/kynon-j.m.-benjamin/">Kynon J.M. Benjamin</a></span>, <span >
      <a href="/authors/qiang-chen/">Qiang Chen</a></span>, <span >
      <a href="/authors/nickeagles/">Nicholas J. Eagles</a></span>, <span >
      <a href="/authors/lahuuki/">Louise A. Huuki-Myers</a></span>, <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >
      <a href="/authors/joshstolz/">Joshua M. Stolz</a></span>, <span >
      <a href="/authors/joo-heon-shin/">Joo Heon Shin</a></span>, <span >
      <a href="/authors/apua-c.m.-paquola/">Apuã C.M. Paquola</a></span>, <span >
      <a href="/authors/thomas-m.-hyde/">Thomas M. Hyde</a></span>, <span >
      <a href="/authors/joel-e.-kleinman/">Joel E. Kleinman</a></span>, <span >
      <a href="/authors/andrew-e.-jaffe/">Andrew E. Jaffe</a></span>, <span >
      <a href="/authors/shizhong-han-dagger/">Shizhong Han &amp;dagger;</a></span>, <span >
      <a href="/authors/daniel-r.-weinberger-dagger/">Daniel R. Weinberger  &amp;dagger;</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    March, 2023
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>bioRxiv</em>
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/preprint_aanri/" >
      <div class="img-hover-zoom">
        <img src="/publication/preprint_aanri/featured_huc807d586db907e907a19c3699ceb919e_379554_808x455_fill_q75_h2_lanczos_smart1_3.webp" height="455" width="808"
            class="article-banner" alt="Genetic and environmental contributions to ancestry differences in gene expression in the human brain" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/preprint_aanri/" >Genetic and environmental contributions to ancestry differences in gene expression in the human brain</a>
  </div>

  
  <a href="/publication/preprint_aanri/"  class="summary-link">
    <div class="article-style">
      <p>Ancestral differences in genomic variation are determining factors in gene regulation; however, most gene expression studies have been limited to European ancestry samples or adjusted for ancestry to identify ancestry-independent associations. We instead examined the impact of genetic ancestry on gene expression and DNA methylation (DNAm) in admixed African/Black American neurotypical individuals to untangle effects of genetic and environmental factors. Ancestry-associated differentially expressed genes (DEGs), transcripts, and gene networks, while notably not implicating neurons, are enriched for genes related to immune response and vascular tissue and explain up to 26% of heritability for ischemic stroke, 27% of heritability for Parkinson’s disease, and 30% of heritability for Alzhemier’s disease. Ancestry-associated DEGs also show general enrichment for heritability of diverse immune-related traits but depletion for psychiatric-related traits. The cell-type enrichments and direction of effects vary by brain region. These DEGs are less evolutionarily constrained and are largely explained by genetic variations; roughly 15% are predicted by DNAm variation implicating environmental exposures. We also compared Black and White Americans, confirming most of these ancestry-associated DEGs. Our results highlight how environment and genetic background affect genetic ancestry differences in gene expression in the human brain and affect risk for brain illness.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/preprint_aanri/cite.bib">
  Cite
</a>





  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/brainseq/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1101/2023.03.28.534458" target="_blank" rel="noopener">
  DOI
</a>


  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.biorxiv.org/content/10.1101/2023.03.28.534458v1" target="_blank" rel="noopener">
    Preprint</a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/LieberInstitute/aanri_phase1" target="_blank" rel="noopener">
    Code</a>


  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-3 year-2023">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/lahuuki/">Louise A. Huuki-Myers</a></span>, <span >
      <a href="/authors/abby-spangler/">Abby Spangler</a></span>, <span >
      <a href="/authors/nickeagles/">Nicholas J. Eagles</a></span>, <span >
      <a href="/authors/kelsey-d.-montgomery/">Kelsey D. Montgomery</a></span>, <span >
      <a href="/authors/sang-ho-kwon/">Sang Ho Kwon</a></span>, <span >
      <a href="/authors/boyi-guo/">Boyi Guo</a></span>, <span >
      <a href="/authors/melissa-grant-peters/">Melissa Grant-Peters</a></span>, <span >
      <a href="/authors/heena-r.-divecha/">Heena R. Divecha</a></span>, <span >
      <a href="/authors/madhavi-tippani/">Madhavi Tippani</a></span>, <span >
      <a href="/authors/chaichontat-sriworarat/">Chaichontat Sriworarat</a></span>, <span >
      <a href="/authors/annie-b.-nguyen/">Annie B. Nguyen</a></span>, <span >
      <a href="/authors/prashanthi-ravichandran/">Prashanthi Ravichandran</a></span>, <span >
      <a href="/authors/matthew-n.-tran/">Matthew N. Tran</a></span>, <span >
      <a href="/authors/arta-seyedian/">Arta Seyedian</a></span>, <span >
      <a href="/authors/psychencode-consortium/">PsychENCODE consortium</a></span>, <span >
      <a href="/authors/thomas-m.-hyde/">Thomas M. Hyde</a></span>, <span >
      <a href="/authors/joel-e.-kleinman/">Joel E. Kleinman</a></span>, <span >
      <a href="/authors/alexis-battle/">Alexis Battle</a></span>, <span >
      <a href="/authors/stephanie-c.-page/">Stephanie C. Page</a></span>, <span >
      <a href="/authors/mina-ryten/">Mina Ryten</a></span>, <span >
      <a href="/authors/stephanie-c.-hicks/">Stephanie C. Hicks</a></span>, <span >
      <a href="/authors/keri-martinowich/">Keri Martinowich</a></span>, <span >
      <a href="/authors/leonardo-collado-torres-dagger/">Leonardo Collado-Torres &amp;dagger;</a></span>, <span >
      <a href="/authors/kristen-r.-maynard-dagger/">Kristen R. Maynard &amp;dagger;</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    February, 2023
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>bioRxiv</em>
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/preprint_spatialdlpfc/" >
      <div class="img-hover-zoom">
        <img src="/publication/preprint_spatialdlpfc/featured_hu0f9b42b08470d4791136a729aca1a3c9_6139716_808x455_fill_q75_h2_lanczos_smart1_3.webp" height="455" width="808"
            class="article-banner" alt="Integrated single cell and unsupervised spatial transcriptomic analysis defines molecular anatomy of the human dorsolateral prefrontal cortex" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/preprint_spatialdlpfc/" >Integrated single cell and unsupervised spatial transcriptomic analysis defines molecular anatomy of the human dorsolateral prefrontal cortex</a>
  </div>

  
  <a href="/publication/preprint_spatialdlpfc/"  class="summary-link">
    <div class="article-style">
      <p>The molecular organization of the human neocortex has been historically studied in the context of its histological layers. However, emerging spatial transcriptomic technologies have enabled unbiased identification of transcriptionally-defined spatial domains that move beyond classic cytoarchitecture. Here we used the Visium spatial gene expression platform to generate a data-driven molecular neuroanatomical atlas across the anterior-posterior axis of the human dorsolateral prefrontal cortex (DLPFC). Integration with paired single nucleus RNA-sequencing data revealed distinct cell type compositions and cell-cell interactions across spatial domains. Using PsychENCODE and publicly available data, we map the enrichment of cell types and genes associated with neuropsychiatric disorders to discrete spatial domains. Finally, we provide resources for the scientific community to explore these integrated spatial and single cell datasets at <a href="https://research.libd.org/spatialDLPFC/" target="_blank" rel="noopener">https://research.libd.org/spatialDLPFC/</a>.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/preprint_spatialdlpfc/cite.bib">
  Cite
</a>





  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/spatial/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1101/2023.02.15.528722" target="_blank" rel="noopener">
  DOI
</a>


  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.biorxiv.org/content/10.1101/2023.02.15.528722v1" target="_blank" rel="noopener">
    Preprint</a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/LieberInstitute/spatialDLPFC" target="_blank" rel="noopener">
    Code</a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/LieberInstitute/DLPFC_snRNAseq" target="_blank" rel="noopener">
    snRNA-seq code</a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="http://research.libd.org/spatialDLPFC/" target="_blank" rel="noopener">
    Documentation</a>


  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-3 year-2023">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/chaichontat-sriworarat/">Chaichontat Sriworarat</a></span>, <span >
      <a href="/authors/annie-nguyen/">Annie Nguyen</a></span>, <span >
      <a href="/authors/nickeagles/">Nicholas J. Eagles</a></span>, <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >
      <a href="/authors/keri-martinowich/">Keri Martinowich</a></span>, <span >
      <a href="/authors/kristen-r.-maynard-dagger/">Kristen R. Maynard &amp;dagger;</a></span>, <span >
      <a href="/authors/stephanie-c.-hicks-dagger/">Stephanie C. Hicks &amp;dagger;</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    January, 2023
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>bioRxiv</em>
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/preprint_samuibrowser/" >
      <div class="img-hover-zoom">
        <img src="/publication/preprint_samuibrowser/featured_hue5ee80043b992f0f1e23adccc6ac3150_1252963_808x455_fill_q75_h2_lanczos_smart1_3.webp" height="455" width="808"
            class="article-banner" alt="Performant web-based interactive visualization tool for spatially-resolved transcriptomics experiments" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/preprint_samuibrowser/" >Performant web-based interactive visualization tool for spatially-resolved transcriptomics experiments</a>
  </div>

  
  <a href="/publication/preprint_samuibrowser/"  class="summary-link">
    <div class="article-style">
      <p>High-resolution and multiplexed imaging techniques are giving us an increasingly detailed observation of a biological system. However, sharing, exploring, and customizing the visualization of large multidimensional images can be a challenge. Here, we introduce Samui, a performant and interactive image visualization tool that runs completely in the web browser. Samui is specifically designed for fast image visualization and annotation and enables users to browse through large images and their selected features within seconds of receiving a link. We demonstrate the broad utility of Samui with images generated with two platforms: Vizgen MERFISH and 10x Genomics Visium Spatial Gene Expression. Samui along with example datasets is available at <a href="https://samuibrowser.com" target="_blank" rel="noopener">https://samuibrowser.com</a>.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/preprint_samuibrowser/cite.bib">
  Cite
</a>





  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/spatial/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1101/2023.01.28.525943" target="_blank" rel="noopener">
  DOI
</a>


  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.biorxiv.org/content/10.1101/2023.01.28.525943v1" target="_blank" rel="noopener">
    Preprint</a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="https://samuibrowser.com" target="_blank" rel="noopener">
    Samui Browser</a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/chaichontat/samui" target="_blank" rel="noopener">
    Code</a>


  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2022">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/sonia-garcia-ruiz/">Sonia Garcia Ruiz</a></span>, <span >
      <a href="/authors/emil-k-gustavsson/">Emil K Gustavsson</a></span>, <span >
      <a href="/authors/david-zhang/">David Zhang</a></span>, <span >
      <a href="/authors/regina-h-reynolds/">Regina H Reynolds</a></span>, <span >
      <a href="/authors/zhongbo-chen/">Zhongbo Chen</a></span>, <span >
      <a href="/authors/aine-fairbrother-browne/">Aine Fairbrother-Browne</a></span>, <span >
      <a href="/authors/ana-luisa-gil-martinez/">Ana Luisa Gil-Martínez</a></span>, <span >
      <a href="/authors/juan-a-botia/">Juan A Botia</a></span>, <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >
      <a href="/authors/mina-ryten/">Mina Ryten</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    November, 2022
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Nucleic Acids Research</em>
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/2023-01_introverse/" >
      <div class="img-hover-zoom">
        <img src="/publication/2023-01_introverse/featured_hub36224a8706972ce057cb06a6549eca0_410484_808x455_fill_q75_h2_lanczos_smart1_3.webp" height="455" width="808"
            class="article-banner" alt="IntroVerse: a comprehensive database of introns across human tissues" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/2023-01_introverse/" >IntroVerse: a comprehensive database of introns across human tissues</a>
  </div>

  
  <a href="/publication/2023-01_introverse/"  class="summary-link">
    <div class="article-style">
      <p>Dysregulation of RNA splicing contributes to both rare and complex diseases. RNA-sequencing data from human tissues has shown that this process can be inaccurate, resulting in the presence of novel introns detected at low frequency across samples and within an individual. To enable the full spectrum of intron use to be explored, we have developed IntroVerse, which offers an extensive catalogue on the splicing of 332,571 annotated introns and a linked set of 4,679,474 novel junctions covering 32,669 different genes. This dataset has been generated through the analysis of 17,510 human control RNA samples from 54 tissues provided by the Genotype-Tissue Expression Consortium. IntroVerse has two unique features: (i) it provides a complete catalogue of novel junctions and (ii) each novel junction has been assigned to a specific annotated intron. This unique, hierarchical structure offers multiple uses, including the identification of novel transcripts from known genes and their tissue-specific usage, and the assessment of background splicing noise for introns thought to be mis-spliced in disease states. IntroVerse provides a user-friendly web interface and is freely available at <a href="https://rytenlab.com/browser/app/introverse" target="_blank" rel="noopener">https://rytenlab.com/browser/app/introverse</a>.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/2023-01_introverse/cite.bib">
  Cite
</a>





  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/recount3/">
    Project
  </a>
  

  
    
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1093/nar/gkac1056" target="_blank" rel="noopener">
  DOI
</a>


  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/SoniaRuiz/introverse" target="_blank" rel="noopener">
    Code</a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="https://rytenlab.com/browser/app/introverse" target="_blank" rel="noopener">
    IntroVerse</a>


  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2022">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/kynon-jm-benjamin/">Kynon JM Benjamin</a></span>, <span >
      <a href="/authors/qiang-chen/">Qiang Chen</a></span>, <span >
      <a href="/authors/andrew-e-jaffe/">Andrew E Jaffe</a></span>, <span >
      <a href="/authors/joshstolz/">Joshua M. Stolz</a></span>, <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >
      <a href="/authors/lahuuki/">Louise A. Huuki-Myers</a></span>, <span >
      <a href="/authors/emily-e-burke/">Emily E Burke</a></span>, <span >
      <a href="/authors/ria-arora/">Ria Arora</a></span>, <span >
      <a href="/authors/arthur-s-feltrin/">Arthur S Feltrin</a></span>, <span >
      <a href="/authors/andre-rocha-barbosa/">André Rocha Barbosa</a></span>, <span >
      <a href="/authors/eugenia-radulescu/">Eugenia Radulescu</a></span>, <span >
      <a href="/authors/giulio-pergola/">Giulio Pergola</a></span>, <span >
      <a href="/authors/joo-heon-shin/">Joo Heon Shin</a></span>, <span >
      <a href="/authors/william-s-ulrich/">William S Ulrich</a></span>, <span >
      <a href="/authors/amy-deep-soboslay/">Amy Deep-Soboslay</a></span>, <span >
      <a href="/authors/ran-tao/">Ran Tao</a></span>, <span >
      <a href="/authors/the-brainseq-consortium/">The BrainSeq Consortium</a></span>, <span >
      <a href="/authors/thomas-m-hyde/">Thomas M Hyde</a></span>, <span >
      <a href="/authors/joel-e-kleinman/">Joel E Kleinman</a></span>, <span >
      <a href="/authors/jennifer-a-erwin-dagger/">Jennifer A Erwin &amp;dagger;</a></span>, <span >
      <a href="/authors/daniel-r-weinberger-dagger/">Daniel R Weinberger &amp;dagger;</a></span>, <span >
      <a href="/authors/apua-cm-paquola-dagger/">Apuã CM Paquola &amp;dagger;</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    November, 2022
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Nature Neuroscience</em>
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/2022-11_bsp3/" >
      <div class="img-hover-zoom">
        <img src="/publication/2022-11_bsp3/featured_hu7286b2919250c0322098c2424bfc1244_1113323_808x455_fill_q75_h2_lanczos_smart1_3.webp" height="455" width="808"
            class="article-banner" alt="Analysis of the caudate nucleus transcriptome in individuals with schizophrenia highlights effects of antipsychotics and new risk genes" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/2022-11_bsp3/" >Analysis of the caudate nucleus transcriptome in individuals with schizophrenia highlights effects of antipsychotics and new risk genes</a>
  </div>

  
  <a href="/publication/2022-11_bsp3/"  class="summary-link">
    <div class="article-style">
      <p>Most studies of gene expression in the brains of individuals with schizophrenia have focused on cortical regions, but subcortical nuclei such as the striatum are prominently implicated in the disease, and current antipsychotic drugs target the striatum’s dense dopaminergic innervation. Here, we performed a comprehensive analysis of the genetic and transcriptional landscape of schizophrenia in the postmortem caudate nucleus of the striatum of 443 individuals (245 neurotypical individuals, 154 individuals with schizophrenia and 44 individuals with bipolar disorder), 210 from African and 233 from European ancestries. Integrating expression quantitative trait loci analysis, Mendelian randomization with the latest schizophrenia genome-wide association study, transcriptome-wide association study and differential expression analysis, we identified many genes associated with schizophrenia risk, including potentially the dopamine D2 receptor short isoform. We found that antipsychotic medication has an extensive influence on caudate gene expression. We constructed caudate nucleus gene expression networks that highlight interactions involving schizophrenia risk. These analyses provide a resource for the study of schizophrenia and insights into risk mechanisms and potential therapeutic targets.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.medrxiv.org/content/10.1101/2020.11.18.20230540v2" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1038/s41593-022-01182-7" target="_blank" rel="noopener">
  PDF
</a>



<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/2022-11_bsp3/cite.bib">
  Cite
</a>





  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/brainseq/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1038/s41593-022-01182-7" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-3 year-2022">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/kynon-jm-benjamin-dagger-____/">Kynon JM Benjamin &amp;dagger; __*__</a></span>, <span >
      <a href="/authors/ria-arora-____/">Ria Arora __*__</a></span>, <span >
      <a href="/authors/joshstolz/">Joshua M. Stolz</a></span>, <span >
      <a href="/authors/laura-dignazio/">Laura D’Ignazio</a></span>, <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >
      <a href="/authors/thomas-m-hyde/">Thomas M Hyde</a></span>, <span >
      <a href="/authors/joel-e-kleinman/">Joel E Kleinman</a></span>, <span >
      <a href="/authors/daniel-r-weinberger-dagger/">Daniel R Weinberger &amp;dagger;</a></span>, <span >
      <a href="/authors/apua-cm-paquola-dagger/">Apuã CM Paquola &amp;dagger;</a></span>, <span >
      <a href="/authors/jennifer-a-erwin-dagger/">Jennifer A Erwin &amp;dagger;</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    September, 2022
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>medRxiv</em>
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/preprint_sex_differences_kj/" >
      <div class="img-hover-zoom">
        <img src="/publication/preprint_sex_differences_kj/featured_hu27dc5daae53d8cf6f0aa2b876c576e18_1156078_808x455_fill_q75_h2_lanczos_smart1_3.webp" height="455" width="808"
            class="article-banner" alt="Transcriptional and genetic sex differences for schizophrenia across the dorsolateral prefrontal cortex, hippocampus, and caudate nucleus" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/preprint_sex_differences_kj/" >Transcriptional and genetic sex differences for schizophrenia across the dorsolateral prefrontal cortex, hippocampus, and caudate nucleus</a>
  </div>

  
  <a href="/publication/preprint_sex_differences_kj/"  class="summary-link">
    <div class="article-style">
      <p>Schizophrenia is a complex neuropsychiatric disorder with sexually dimorphic features, including differential symptomatology, drug responsiveness, and male incidence rate. To date, only the prefrontal cortex has been examined in large-scale transcriptome analyses for sex differences in schizophrenia. Here, we examined the BrainSeq Consortium RNA-sequencing and genotypes for the caudate nucleus (n=399), dorsolateral prefrontal cortex (DLPFC; n=377), and hippocampus (n=394) to characterize sex differences in schizophrenia. We identified genomic features (genes, transcripts, exons, and exon-exon junctions) associated with sex, sex-specific expression in schizophrenia, and sex-interacting expression quantitative trait loci (si-eQTL) associated with schizophrenia risk. We found 878 unique genes with sex differences across brain regions, including ANK3, which shows male-biased expression in the caudate nucleus. X-chromosome dosage was significantly decreased in the hippocampus of female and male individuals with schizophrenia. Our sex interaction model revealed 15 novel junctions dysregulated for schizophrenia in a sex-specific manner. Sex-specific schizophrenia analysis identified dozens of expressed, sex-specific features with enrichment in the transcriptional response of cellular stress. Finally, our si-eQTL analysis revealed 974 unique genes, 14 of which are associated with schizophrenia risk. Overall, our results increased the number of annotated sex-biased features, identified sex-specific schizophrenia genes, and provided the first annotation of si-eQTL in the human DLPFC and hippocampus. Altogether, these results point to the importance of sex-informed analysis of sexually dimorphic traits and inform personalized therapeutic strategies in schizophrenia..</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/preprint_sex_differences_kj/cite.bib">
  Cite
</a>





  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/brainseq/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1101/2022.09.30.22280452" target="_blank" rel="noopener">
  DOI
</a>


  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.medrxiv.org/content/10.1101/2022.09.30.22280452v1" target="_blank" rel="noopener">
    Preprint</a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/LieberInstitute/sex_differences_sz" target="_blank" rel="noopener">
    Code</a>


  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2022">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/gabriel-e.-hoffman-dagger/">Gabriel E. Hoffman &amp;dagger;</a></span>, <span >
      <a href="/authors/andrew-e.-jaffe-dagger/">Andrew E. Jaffe &amp;dagger;</a></span>, <span >
      <a href="/authors/michael-j.-gandal-dagger/">Michael J. Gandal &amp;dagger;</a></span>, <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >
      <a href="/authors/solveig-k.-sieberts/">Solveig K. Sieberts</a></span>, <span >
      <a href="/authors/bernie-devlin/">Bernie Devlin</a></span>, <span >
      <a href="/authors/daniel-h.-geschwind/">Daniel H. Geschwind</a></span>, <span >
      <a href="/authors/daniel-r.-weinberger/">Daniel R. Weinberger</a></span>, <span >
      <a href="/authors/panos-roussos/">Panos Roussos</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    September, 2022
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Molecular Psychiatry</em>
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/2022-09_commentary_schizophrenia/" >
      <div class="img-hover-zoom">
        <img src="/publication/2022-09_commentary_schizophrenia/featured_hu5295d88cad4f49dedf7a513107a18e5c_220226_808x455_fill_q75_h2_lanczos_smart1_3.webp" height="455" width="808"
            class="article-banner" alt="Comment on: What genes are differentially expressed in individuals with schizophrenia? A systematic review" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/2022-09_commentary_schizophrenia/" >Comment on: What genes are differentially expressed in individuals with schizophrenia? A systematic review</a>
  </div>

  
  <a href="/publication/2022-09_commentary_schizophrenia/"  class="summary-link">
    <div class="article-style">
      <p>Following the recent review by Merikangas et al. summarizing differential gene expression in schizophrenia, we wanted to provide readers with our perspective on the state of the field in order to highlight current resources and ongoing challenges. We have led data collection and analysis of post mortem brains of control and schizophrenia samples across multiple large-scale cohorts, and we have provided data and results resources for each of these studies (Table 1).</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/2022-09_commentary_schizophrenia/cite.bib">
  Cite
</a>





  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/brainseq/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1038/s41380-022-01781-7" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2022">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/bpardo/">Brenda Pardo</a></span>, <span >
      <a href="/authors/abby-spangler/">Abby Spangler</a></span>, <span >
      <a href="/authors/lukas-m.-weber/">Lukas M. Weber</a></span>, <span >
      <a href="/authors/stephanie-c.-hicks/">Stephanie C. Hicks</a></span>, <span >
      <a href="/authors/andrew-e.-jaffe/">Andrew E. Jaffe</a></span>, <span >
      <a href="/authors/keri-martinowich/">Keri Martinowich</a></span>, <span >
      <a href="/authors/kristen-r.-maynard/">Kristen R. Maynard</a></span>, <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    June, 2022
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>BMC Genomics</em>
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/2022-06_spatiallibd/" >
      <div class="img-hover-zoom">
        <img src="/publication/2022-06_spatiallibd/featured_hud0e2c61eda697b362d9fab2a7479f6ed_132900_808x455_fill_q75_h2_lanczos_smart1.webp" height="455" width="808"
            class="article-banner" alt="spatialLIBD: an R/Bioconductor package to visualize spatially-resolved transcriptomics data" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/2022-06_spatiallibd/" >spatialLIBD: an R/Bioconductor package to visualize spatially-resolved transcriptomics data</a>
  </div>

  
  <a href="/publication/2022-06_spatiallibd/"  class="summary-link">
    <div class="article-style">
      <p><strong>Background</strong>. Spatially-resolved transcriptomics has now enabled the quantification of high-throughput and transcriptome-wide gene expression in intact tissue while also retaining the spatial coordinates. Incorporating the precise spatial mapping of gene activity advances our understanding of intact tissue-specific biological processes. In order to interpret these novel spatial data types, interactive visualization tools are necessary. <strong>Results</strong>. We describe <em>spatialLIBD</em>, an R/Bioconductor package to interactively explore spatially-resolved transcriptomics data generated with the 10x Genomics Visium platform. The package contains functions to interactively access, visualize, and inspect the observed spatial gene expression data and data-driven clusters identified with supervised or unsupervised analyses, either on the user’s computer or through a web application. <strong>Conclusions</strong>. <em>spatialLIBD</em> is available at <a href="https://bioconductor.org/packages/spatialLIBD" target="_blank" rel="noopener">https://bioconductor.org/packages/spatialLIBD</a>. It is fully compatible with SpatialExperiment and the Bioconductor ecosystem. Its functionality facilitates analyzing and interactively exploring spatially-resolved data from the Visium platform.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.biorxiv.org/content/10.1101/2021.04.29.440149v1" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1186/s12864-022-08601-w" target="_blank" rel="noopener">
  PDF
</a>



<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/2022-06_spatiallibd/cite.bib">
  Cite
</a>


  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://bioconductor.org/packages/spatialLIBD" target="_blank" rel="noopener">
  Code
</a>




  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/spatial/">
    Project
  </a>
  





  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="http://research.libd.org/spatialLIBD/" target="_blank" rel="noopener">
  Slides
</a>




  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/LieberInstitute/spatialLIBD" target="_blank" rel="noopener">
  Source Document
</a>


<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1186/s12864-022-08601-w" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-3 year-2022">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/madhavi-tippani/">Madhavi Tippani</a></span>, <span >
      <a href="/authors/heena-r.-divecha/">Heena R. Divecha</a></span>, <span >
      <a href="/authors/joseph-l.-catallini-ii/">Joseph L. Catallini II</a></span>, <span >
      <a href="/authors/sang-ho-kwon/">Sang Ho Kwon</a></span>, <span >
      <a href="/authors/lukas-m.-weber/">Lukas M. Weber</a></span>, <span >
      <a href="/authors/abby-spangler/">Abby Spangler</a></span>, <span >
      <a href="/authors/andrew-e.-jaffe/">Andrew E. Jaffe</a></span>, <span >
      <a href="/authors/stephanie-c.-hicks/">Stephanie C. Hicks</a></span>, <span >
      <a href="/authors/keri-martinowich/">Keri Martinowich</a></span>, <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >
      <a href="/authors/stephanie-c.-page-dagger/">Stephanie C. Page &amp;dagger;</a></span>, <span >
      <a href="/authors/kristen-r.-maynard-dagger/">Kristen R. Maynard &amp;dagger;</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    May, 2022
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>bioRxiv</em>
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/preprint_vistoseg/" >
      <div class="img-hover-zoom">
        <img src="/publication/preprint_vistoseg/featured_huc934cca8b6133a3bc1a854249f7ce153_1656983_808x455_fill_q75_h2_lanczos_smart1_3.webp" height="455" width="808"
            class="article-banner" alt="VistoSeg: processing utilities for high-resolution Visium/Visium-IF images for spatial transcriptomics data" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/preprint_vistoseg/" >VistoSeg: processing utilities for high-resolution Visium/Visium-IF images for spatial transcriptomics data</a>
  </div>

  
  <a href="/publication/preprint_vistoseg/"  class="summary-link">
    <div class="article-style">
      <p><strong>Background</strong>: Spatial transcriptomics is a next-generation sequencing technology that combines the strengths of transcriptome-wide RNA-sequencing with histological imaging to generate spatial maps of gene expression in intact tissue sections. The 10x Genomics Visium and Visium-Immunofluorescence (Visium-IF) platforms are widely available commercial technologies for quantifying spatially-resolved gene expression. These technologies directly couple gene expression with high resolution histological or immunofluorescence images that contain rich morphological information about the tissue section. However, extracting and integrating image features with gene expression data remains challenging. <strong>Results</strong>: Using MATLAB, we developed VistoSeg, which is a pipeline to process, analyze, and interactively visualize the high-resolution images from the 10x Genomics Visium and Visium-IF platforms. The output from VistoSeg can then be integrated with the spatial-molecular information in downstream analyses using common programming languages, such as R or Python. <strong>Conclusion</strong>: VistoSeg provides user-friendly tools for integrating image-derived metrics from histological and immunofluorescent images with spatially-resolved gene expression data. This integrated approach can advance our understanding of the transcriptional landscape within tissue architecture. VistoSeg is freely available at <a href="http://research.libd.org/VistoSeg/" target="_blank" rel="noopener">http://research.libd.org/VistoSeg/</a>. <strong>Impact Statement</strong>: Technologies for measuring gene activity levels, referred to as gene expression, have been evolving over decades and are the core of the transcriptomics subfield within genomics. The first report describing individual cell gene expression is from 2009 and as a method it became commercially available in 2014. While single cell transcriptomics increased our resolution beyond homogenate tissue, the advent of spatial transcriptomics technologies and commercial availability of spatial gene expression platforms, such as Visium, has facilitated studying gene expression in anatomical context. Visium measures local gene expression within the histological organization of single 6.5 mm2 cryosection of tissue. Spatially-resolved transcriptomics provides a new challenge: integrating spatial gene expression with high resolution tissue images (brightfield histology or fluorescent antibody staining). VistoSeg image processing software is compatible with both Visium and Visium-IF from 10x Genomics, which are spatially-resolved transcriptomics assays employing histological and immunofluorescent images, respectively. From these images, the number of cells, identity of cell types, and other image-derived markers can be obtained for thousands of 2,375 µm2 spots, where genome-wide gene expression is also measured. VistoSeg provides tools that enable processing these images in the context of gene expression maps to integrate these two high dimensional data types, and thus help unlock the new frontier in transcriptomics.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/preprint_vistoseg/cite.bib">
  Cite
</a>





  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/spatial/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1101/2021.08.04.452489" target="_blank" rel="noopener">
  DOI
</a>


  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.biorxiv.org/content/10.1101/2021.08.04.452489v2" target="_blank" rel="noopener">
    Preprint</a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/LieberInstitute/VistoSeg" target="_blank" rel="noopener">
    Code</a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="http://research.libd.org/VistoSeg/" target="_blank" rel="noopener">
    Documentation</a>


  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-0 year-2022">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    May, 2022
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/poster2022bog/" >
      <div class="img-hover-zoom">
        <img src="/publication/poster2022bog/featured_huda2e03ad466eb84cc2084d34418d5420_3724015_808x455_fill_q75_h2_lanczos_top_3.webp" height="455" width="808"
            class="article-banner" alt="Spatial Transcriptomics Analysis of Aβ-tau Synergy in the Inferior Temporal Cortex of the Human Brain in Alzheimer’s Disease" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/poster2022bog/" >Spatial Transcriptomics Analysis of Aβ-tau Synergy in the Inferior Temporal Cortex of the Human Brain in Alzheimer’s Disease</a>
  </div>

  
  <a href="/publication/poster2022bog/"  class="summary-link">
    <div class="article-style">
      <p>Poster presented at BoG2022</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  









  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/spatial/">
    Project
  </a>
  





  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://speakerdeck.com/lcolladotor/bog2022" target="_blank" rel="noopener">
  Slides
</a>







  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2022">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/dario-righelli-____/">Dario Righelli __*__</a></span>, <span >
      <a href="/authors/lukas-m.-weber-____/">Lukas M. Weber __*__</a></span>, <span >
      <a href="/authors/helena-l.-crowell-____/">Helena L. Crowell __*__</a></span>, <span >
      <a href="/authors/bpardo/">Brenda Pardo</a></span>, <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >
      <a href="/authors/shila-ghazanfar/">Shila Ghazanfar</a></span>, <span >
      <a href="/authors/aaron-t.-l.-lun/">Aaron T. L. Lun,</a></span>, <span >
      <a href="/authors/stephanie-c.-hicks-dagger/">Stephanie C. Hicks &amp;dagger;</a></span>, <span >
      <a href="/authors/davide-risso-dagger/">Davide Risso &amp;dagger;</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    April, 2022
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Oxford Bioinformatics</em>
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/2022_04_spatialexperiment/" >
      <div class="img-hover-zoom">
        <img src="/publication/2022_04_spatialexperiment/featured_hucf0352a9ffded1e9cfaadf06fed212bd_180419_808x455_fill_q75_h2_lanczos_smart1_3.webp" height="455" width="808"
            class="article-banner" alt="SpatialExperiment: infrastructure for spatially resolved transcriptomics data in R using Bioconductor" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/2022_04_spatialexperiment/" >SpatialExperiment: infrastructure for spatially resolved transcriptomics data in R using Bioconductor</a>
  </div>

  
  <a href="/publication/2022_04_spatialexperiment/"  class="summary-link">
    <div class="article-style">
      <p><strong>Summary</strong> SpatialExperiment is a new data infrastructure for storing and accessing spatially resolved transcriptomics data, implemented within the R/Bioconductor framework, which provides advantages of modularity, interoperability, standardized operations, and comprehensive documentation. Here, we demonstrate the structure and user interface with examples from the 10x Genomics Visium and seqFISH platforms, and provide access to example datasets and visualization tools in the STexampleData, TENxVisiumData, and ggspavis packages. <strong>Availability and Implementation</strong> The SpatialExperiment, STexampleData, TENxVisiumData, and ggspavis packages are available from Bioconductor. The package versions described in this manuscript are available in Bioconductor version 3.15 onwards. <strong>Supplementary information</strong> Supplementary tables and figures are available at Bioinformatics online.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1101/2021.01.27.428431" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1093/bioinformatics/btac299" target="_blank" rel="noopener">
  PDF
</a>



<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/2022_04_spatialexperiment/cite.bib">
  Cite
</a>


  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://bioconductor.org/packages/SpatialExperiment" target="_blank" rel="noopener">
  Code
</a>




  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/spatial/">
    Project
  </a>
  








  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/drighelli/SpatialExperiment" target="_blank" rel="noopener">
  Source Document
</a>


<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1093/bioinformatics/btac299" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-3 year-2022">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/lahuuki/">Louise A. Huuki-Myers</a></span>, <span >
      <a href="/authors/kelsey-d.-montgomery/">Kelsey D. Montgomery</a></span>, <span >
      <a href="/authors/sang-ho-kwon/">Sang Ho Kwon</a></span>, <span >
      <a href="/authors/stephanie-c.-page/">Stephanie C. Page</a></span>, <span >
      <a href="/authors/stephanie-c.-hicks/">Stephanie C. Hicks</a></span>, <span >
      <a href="/authors/kristen-r.-maynard-dagger/">Kristen R. Maynard &amp;dagger;</a></span>, <span >
      <a href="/authors/leonardo-collado-torres-dagger/">Leonardo Collado-Torres &amp;dagger;</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    April, 2022
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/preprint_treg/" >
      <div class="img-hover-zoom">
        <img src="/publication/preprint_treg/featured_hu63f5c5bcd3149c42f83266bf497dccac_203478_808x455_fill_q75_h2_lanczos_smart1.webp" height="455" width="808"
            class="article-banner" alt="Data-driven Identification of Total RNA Expression Genes (TREGs) for Estimation of RNA Abundance in Heterogeneous Cell Types" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/preprint_treg/" >Data-driven Identification of Total RNA Expression Genes (TREGs) for Estimation of RNA Abundance in Heterogeneous Cell Types</a>
  </div>

  
  <a href="/publication/preprint_treg/"  class="summary-link">
    <div class="article-style">
      <p>Next-generation sequencing technologies have facilitated data-driven identification of gene sets with different features including genes with stable expression, cell-type specific expression, or spatially variable expression. Here, we aimed to define and identify a new class of &quot;control&quot; genes called Total RNA Expression Genes (TREGs), which correlate with total RNA abundance in heterogeneous cell types of different sizes and transcriptional activity. We provide a data-driven method to identify TREGs from single cell RNA-sequencing (RNA-seq) data, available as an R/Bioconductor package at <a href="https://bioconductor.org/packages/TREG" target="_blank" rel="noopener">https://bioconductor.org/packages/TREG</a>. We demonstrated the utility of our method in the postmortem human brain using multiplex single molecule fluorescent in situ hybridization (smFISH) and compared candidate TREGs against classic housekeeping genes. We identified <em>AKT3</em> as a top TREG across five brain regions, especially in the dorsolateral prefrontal cortex.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/preprint_treg/cite.bib">
  Cite
</a>





  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/deconvolution/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1101/2022.04.28.489923" target="_blank" rel="noopener">
  DOI
</a>


  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.biorxiv.org/content/10.1101/2022.04.28.489923v1" target="_blank" rel="noopener">
    Preprint</a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/LieberInstitute/TREG" target="_blank" rel="noopener">
    Code</a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="http://research.libd.org/TREG/" target="_blank" rel="noopener">
    Documentation</a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="http://bioconductor.org/packages/TREG/" target="_blank" rel="noopener">
    Bioconductor</a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/LieberInstitute/TREG_paper" target="_blank" rel="noopener">
    Source</a>


  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-3 year-2022">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/nickeagles/">Nicholas J. Eagles</a></span>, <span >
      <a href="/authors/richard-wilton/">Richard Wilton</a></span>, <span >
      <a href="/authors/andrew-e.-jaffe/">Andrew E. Jaffe</a></span>, <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    April, 2022
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/preprint_biocmap/" >
      <div class="img-hover-zoom">
        <img src="/publication/preprint_biocmap/featured_hue75d49778918b1f803ad4ccebf079362_235282_808x455_fill_q75_h2_lanczos_smart1.webp" height="455" width="808"
            class="article-banner" alt="BiocMAP: A Bioconductor-friendly, GPU-Accelerated Pipeline for Bisulfite-Sequencing Data" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/preprint_biocmap/" >BiocMAP: A Bioconductor-friendly, GPU-Accelerated Pipeline for Bisulfite-Sequencing Data</a>
  </div>

  
  <a href="/publication/preprint_biocmap/"  class="summary-link">
    <div class="article-style">
      <p>Background Bisulfite sequencing is a powerful tool for profiling genomic methylation, an epigenetic modification critical in the understanding of cancer, psychiatric disorders, and many other conditions. Raw data generated by whole genome bisulfite sequencing (WGBS) requires several computational steps before it is ready for statistical analysis, and particular care is required to process data in a timely and memory-efficient manner. Alignment to a reference genome is one of the most computationally demanding steps in a WGBS workflow, taking several hours or even days with commonly used WGBS-specific alignment software. This naturally motivates the creation of computational workflows that can utilize GPU-based alignment software to greatly speed up the bottleneck step. In addition, WGBS produces raw data that is large and often unwieldy; a lack of memory-efficient representation of data by existing pipelines renders WGBS impractical or impossible to many researchers. Results We present BiocMAP, a Bioconductor-friendly Methylation Analysis Pipeline consisting of two modules, to address the above concerns. The first module performs computationally-intensive read alignment using Arioc, a GPU-accelerated short-read aligner. The extraction module extracts and merges DNA methylation proportions - the fractions of methylated cytosines across all cells in a sample at a given genomic site. Since GPUs are not always available on the same computing environments where traditional CPU-based analyses are convenient, BiocMAP is split into two modules, with just the alignment module requiring an available GPU. Bioconductor-based output objects in R utilize an on-disk data representation to drastically reduce required main memory and make WGBS projects computationally feasible to more researchers. Conclusions BiocMAP is implemented using Nextflow and available at <a href="http://research.libd.org/BiocMAP/" target="_blank" rel="noopener">http://research.libd.org/BiocMAP/</a>. To enable reproducible analysis across a variety of typical computing environments, BiocMAP can be containerized with Docker or Singularity, and executed locally or with the SLURM or SGE scheduling engines. By providing Bioconductor objects, BiocMAP’s output can be integrated with powerful analytical open source software for analyzing methylation data.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/preprint_biocmap/cite.bib">
  Cite
</a>













<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1101/2022.04.20.488947" target="_blank" rel="noopener">
  DOI
</a>


  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.biorxiv.org/content/10.1101/2022.04.20.488947v1" target="_blank" rel="noopener">
    Preprint</a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/LieberInstitute/BiocMAP" target="_blank" rel="noopener">
    Code</a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="http://research.libd.org/BiocMAP/" target="_blank" rel="noopener">
    Documentation</a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/LieberInstitute/BiocMAP/blob/master/documentation/example_analysis/age_neun_analysis.pdf" target="_blank" rel="noopener">
    Example analysis</a>


  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2022">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/giovanna-punzi/">Giovanna Punzi</a></span>, <span >
      <a href="/authors/gianluca-ursini/">Gianluca Ursini</a></span>, <span >
      <a href="/authors/qiang-chen/">Qiang Chen</a></span>, <span >
      <a href="/authors/eugenia-radulescu/">Eugenia Radulescu</a></span>, <span >
      <a href="/authors/ran-tao/">Ran Tao</a></span>, <span >
      <a href="/authors/lahuuki/">Louise A. Huuki-Myers</a></span>, <span >
      <a href="/authors/pasquale-di-carlo/">Pasquale Di Carlo</a></span>, <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >
      <a href="/authors/joo-heon-shin/">Joo Heon Shin</a></span>, <span >
      <a href="/authors/roberto-catanesi/">Roberto Catanesi</a></span>, <span >
      <a href="/authors/andrew-e.-jaffe/">Andrew E. Jaffe</a></span>, <span >
      <a href="/authors/thomas-m.-hyde/">Thomas M. Hyde</a></span>, <span >
      <a href="/authors/joel-e.-kleinman/">Joel E. Kleinman</a></span>, <span >
      <a href="/authors/trudy-f.c.-mackay/">Trudy F.C. Mackay</a></span>, <span >
      <a href="/authors/daniel-r.-weinberger-dagger/">Daniel R. Weinberger  &amp;dagger;</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    March, 2022
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>American Journal of Psychiatry</em>
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/2022_03_punzi-genetics/" >Genetics and Brain Transcriptomics of Completed Suicide</a>
  </div>

  
  <a href="/publication/2022_03_punzi-genetics/"  class="summary-link">
    <div class="article-style">
      <p>Objective:  The authors sought to study the transcriptomic and genomic features of completed suicide by parsing the method chosen, to capture molecular correlates of the distinctive frame of mind of individuals who die by suicide, while reducing heterogeneity.  Methods:  The authors analyzed gene expression (RNA sequencing) from postmortem dorsolateral prefrontal cortex of patients who died by suicide with violent compared with nonviolent means, nonsuicide patients with the same psychiatric disorders, and a neurotypical group (total N=329). They then examined genomic risk scores (GRSs) for each psychiatric disorder included, and GRSs for cognition (IQ) and for suicide attempt, testing how they predict diagnosis or traits (total N=888).  Results:  Patients who died by suicide by violent means showed a transcriptomic pattern remarkably divergent from each of the other patient groups but less from the neurotypical group; consistently, their genomic profile of risk was relatively low for their diagnosed illness as well as for suicide attempt, and relatively high for IQ: they were more similar to the neurotypical group than to other patients. Differentially expressed genes (DEGs) associated with patients who died by suicide by violent means pointed to purinergic signaling in microglia, showing similarities to a genome-wide association study of Drosophila aggression. Weighted gene coexpression network analysis revealed that these DEGs were coexpressed in a context of mitochondrial metabolic activation unique to suicide by violent means.  Conclusions:  These findings suggest that patients who die by suicide by violent means are in part biologically separable from other patients with the same diagnoses, and their behavioral outcome may be less dependent on genetic risk for conventional psychiatric disorders and be associated with an alteration of purinergic signaling and mitochondrial metabolism.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/2022_03_punzi-genetics/cite.bib">
  Cite
</a>













<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1176/appi.ajp.2021.21030299" target="_blank" rel="noopener">
  DOI
</a>


  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="https://ajp.psychiatryonline.org/doi/10.1176/appi.ajp.2021.21030299" target="_blank" rel="noopener">
    URL</a>


  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2022">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/peter-p.-zandi/">Peter P. Zandi</a></span>, <span >
      <a href="/authors/andrew-e.-jaffe/">Andrew E. Jaffe</a></span>, <span >
      <a href="/authors/fernando-s.-goes/">Fernando S. Goes</a></span>, <span >
      <a href="/authors/emily-e.-burke/">Emily E. Burke</a></span>, <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >
      <a href="/authors/lahuuki/">Louise A. Huuki-Myers</a></span>, <span >
      <a href="/authors/aseyedia/">Arta Seyedian</a></span>, <span >
      <a href="/authors/yian-lin/">Yian Lin</a></span>, <span >
      <a href="/authors/fayaz-seifuddin/">Fayaz Seifuddin</a></span>, <span >
      <a href="/authors/mehdi-pirooznia/">Mehdi Pirooznia</a></span>, <span >
      <a href="/authors/christopher-a.-ross/">Christopher A. Ross</a></span>, <span >
      <a href="/authors/joel-e.-kleinman/">Joel E. Kleinman</a></span>, <span >
      <a href="/authors/daniel-r.-weinberger/">Daniel R. Weinberger</a></span>, <span >
      <a href="/authors/thomas-m.-hyde-dagger/">Thomas M. Hyde  &amp;dagger;</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    March, 2022
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Nature Neuroscience</em>
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/2022_03_zandi-amygdala/" >
      <div class="img-hover-zoom">
        <img src="/publication/2022_03_zandi-amygdala/featured_hu9e3fb06f7a2763749b9b0fdefbd84e4c_156081_808x455_fill_q75_h2_lanczos_smart1_3.webp" height="455" width="808"
            class="article-banner" alt="Amygdala and anterior cingulate transcriptomes from individuals with bipolar disorder reveal downregulated neuroimmune and synaptic pathways" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/2022_03_zandi-amygdala/" >Amygdala and anterior cingulate transcriptomes from individuals with bipolar disorder reveal downregulated neuroimmune and synaptic pathways</a>
  </div>

  
  <a href="/publication/2022_03_zandi-amygdala/"  class="summary-link">
    <div class="article-style">
      <p>Recent genetic studies have identified variants associated with bipolar disorder (BD), but it remains unclear how brain gene expression is altered in BD and how genetic risk for BD may contribute to these alterations. Here, we obtained transcriptomes from subgenual anterior cingulate cortex and amygdala samples from post-mortem brains of individuals with BD and neurotypical controls, including 511 total samples from 295 unique donors. We examined differential gene expression between cases and controls and the transcriptional effects of BD-associated genetic variants. We found two coexpressed modules that were associated with transcriptional changes in BD: one enriched for immune and inflammatory genes and the other with genes related to the postsynaptic membrane. Over 50% of BD genome-wide significant loci contained significant expression quantitative trait loci (QTL) (eQTL), and these data converged on several individual genes, including SCN2A and GRIN2A. Thus, these data implicate specific genes and pathways that may contribute to the pathology of BD.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  





<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/2022_03_zandi-amygdala/cite.bib">
  Cite
</a>





  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/brainseq/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1038/s41593-022-01024-6" target="_blank" rel="noopener">
  DOI
</a>


  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.nature.com/articles/s41593-022-01024-6" target="_blank" rel="noopener">
    URL</a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/LieberInstitute/zandiHyde_bipolar_rnaseq" target="_blank" rel="noopener">
    Code</a>


  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2021">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/christopher-wilks/">Christopher Wilks</a></span>, <span >
      <a href="/authors/shijie-c.-zheng/">Shijie C. Zheng</a></span>, <span >
      <a href="/authors/feng-yong-chen/">Feng Yong Chen</a></span>, <span >
      <a href="/authors/rone-charles/">Rone Charles</a></span>, <span >
      <a href="/authors/brad-solomon/">Brad Solomon</a></span>, <span >
      <a href="/authors/jonathan-p.-ling/">Jonathan P. Ling</a></span>, <span >
      <a href="/authors/eddie-luidy-imada/">Eddie Luidy Imada</a></span>, <span >
      <a href="/authors/david-zhang/">David Zhang</a></span>, <span >
      <a href="/authors/lance-joseph/">Lance Joseph</a></span>, <span >
      <a href="/authors/jeffrey-t.-leek/">Jeffrey T. Leek</a></span>, <span >
      <a href="/authors/andrew-e.-jaffe/">Andrew E. Jaffe</a></span>, <span >
      <a href="/authors/abhinav-nellore/">Abhinav Nellore</a></span>, <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >
      <a href="/authors/kasper-d.-hansen-dagger/">Kasper D. Hansen &amp;dagger;</a></span>, <span >
      <a href="/authors/ben-langmead-dagger/">Ben Langmead &amp;dagger;</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    November, 2021
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Genome Biol
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/2021_11_recount3/" >
      <div class="img-hover-zoom">
        <img src="/publication/2021_11_recount3/featured_hu22452ae3cdaaf511465cf3f616aa40dc_1575466_808x455_fill_q75_h2_lanczos_smart1_3.webp" height="455" width="808"
            class="article-banner" alt="recount3: summaries and queries for large-scale RNA-seq expression and splicing" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/2021_11_recount3/" >recount3: summaries and queries for large-scale RNA-seq expression and splicing</a>
  </div>

  
  <a href="/publication/2021_11_recount3/"  class="summary-link">
    <div class="article-style">
      <p>We present recount3, a resource consisting of over 750,000 publicly available human and mouse RNA sequencing (RNA-seq) samples uniformly processed by our new Monorail analysis pipeline. To facilitate access to the data, we provide the recount3 and snapcount R/Bioconductor packages as well as complementary web resources. Using these tools, data can be downloaded as study-level summaries or queried for specific exon-exon junctions, genes, samples, or other features. Monorail can be used to process local and/or private data, allowing results to be directly compared to any study in recount3. Taken together, our tools help biologists maximize the utility of publicly available RNA-seq data, especially to improve their understanding of newly collected data. recount3 is available from <a href="http://rna.recount.bio" target="_blank" rel="noopener">http://rna.recount.bio</a>.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.biorxiv.org/content/10.1101/2021.05.21.445138v1" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1186/s13059-021-02533-6" target="_blank" rel="noopener">
  PDF
</a>



<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/2021_11_recount3/cite.bib">
  Cite
</a>


  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://bioconductor.org/packages/recount3" target="_blank" rel="noopener">
  Code
</a>


  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="http://rna.recount.bio/" target="_blank" rel="noopener">
  Dataset
</a>



  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/recount3/">
    Project
  </a>
  








  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/LieberInstitute/recount3" target="_blank" rel="noopener">
  Source Document
</a>


<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1186/s13059-021-02533-6" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2021">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/matthew-n.-tran-____/">Matthew N. Tran __*__</a></span>, <span >
      <a href="/authors/kristen-r.-maynard-____/">Kristen R. Maynard __*__</a></span>, <span >
      <a href="/authors/abby-spangler/">Abby Spangler</a></span>, <span >
      <a href="/authors/lahuuki/">Louise A. Huuki-Myers</a></span>, <span >
      <a href="/authors/kelsey-d.-montgomery/">Kelsey D. Montgomery</a></span>, <span >
      <a href="/authors/vijay-sadashivaiah/">Vijay Sadashivaiah</a></span>, <span >
      <a href="/authors/madhavi-tippani/">Madhavi Tippani</a></span>, <span >
      <a href="/authors/brianna-k.-barry/">Brianna K. Barry</a></span>, <span >
      <a href="/authors/dana-b.-hancock/">Dana B. Hancock</a></span>, <span >
      <a href="/authors/stephanie-c.-hicks/">Stephanie C. Hicks</a></span>, <span >
      <a href="/authors/joel-e.-kleinman/">Joel E. Kleinman</a></span>, <span >
      <a href="/authors/thomas-m.-hyde/">Thomas M. Hyde</a></span>, <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >
      <a href="/authors/andrew-e.-jaffe-dagger/">Andrew E. Jaffe &amp;dagger;</a></span>, <span >
      <a href="/authors/keri-martinowich-dagger/">Keri Martinowich &amp;dagger;</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    October, 2021
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Neuron</em>
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/2021_10_matt_snrnaseq/" >
      <div class="img-hover-zoom">
        <img src="/publication/2021_10_matt_snrnaseq/featured_hu31caafb2af503003e4082a0b2076ef35_1370537_808x455_fill_q75_h2_lanczos_smart1_3.webp" height="455" width="808"
            class="article-banner" alt="Single-nucleus transcriptome analysis reveals cell-type-specific molecular signatures across reward circuitry in the human brain" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/2021_10_matt_snrnaseq/" >Single-nucleus transcriptome analysis reveals cell-type-specific molecular signatures across reward circuitry in the human brain</a>
  </div>

  
  <a href="/publication/2021_10_matt_snrnaseq/"  class="summary-link">
    <div class="article-style">
      <p>Single-cell gene expression technologies are powerful tools to study cell types in the human brain, but efforts have largely focused on cortical brain regions. We therefore created a single-nucleus RNA-sequencing resource of 70,615 high-quality nuclei to generate a molecular taxonomy of cell types across five human brain regions that serve as key nodes of the human brain reward circuitry: nucleus accumbens, amygdala, subgenual anterior cingulate cortex, hippocampus, and dorsolateral prefrontal cortex. We first identified novel subpopulations of interneurons and medium spiny neurons (MSNs) in the nucleus accumbens and further characterized robust GABAergic inhibitory cell populations in the amygdala. Joint analyses across the 107 reported cell classes revealed cell-type substructure and unique patterns of transcriptomic dynamics. We identified discrete subpopulations of D1- and D2-expressing MSNs in the nucleus accumbens to which we mapped cell-type-specific enrichment for genetic risk associated with both psychiatric disease and addiction.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.biorxiv.org/content/10.1101/2020.10.07.329839v1" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1016/j.neuron.2021.09.001" target="_blank" rel="noopener">
  PDF
</a>



<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/2021_10_matt_snrnaseq/cite.bib">
  Cite
</a>


  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/LieberInstitute/10xPilot_snRNAseq-human" target="_blank" rel="noopener">
  Code
</a>












<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1016/j.neuron.2021.09.001" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2021">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/kira-a.-perzel-mandell/">Kira A. Perzel Mandell</a></span>, <span >
      <a href="/authors/nickeagles/">Nicholas J. Eagles</a></span>, <span >
      <a href="/authors/richard-wilton/">Richard Wilton</a></span>, <span >
      <a href="/authors/amanda-j.-price/">Amanda J. Price</a></span>, <span >
      <a href="/authors/stephen-a.-semick/">Stephen A. Semick</a></span>, <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >
      <a href="/authors/william-s.-ulrich/">William S. Ulrich</a></span>, <span >
      <a href="/authors/ran-tao/">Ran Tao</a></span>, <span >
      <a href="/authors/shizhong-han/">Shizhong Han</a></span>, <span >
      <a href="/authors/alexander-s.-szalay/">Alexander S. Szalay</a></span>, <span >
      <a href="/authors/thomas-m.-hyde/">Thomas M. Hyde</a></span>, <span >
      <a href="/authors/joel-e.-kleinman/">Joel E. Kleinman</a></span>, <span >
      <a href="/authors/daniel-r.-weinberger-dagger/">Daniel R. Weinberger  &amp;dagger;</a></span>, <span >
      <a href="/authors/andrew-e.-jaffe-dagger/">Andrew E. Jaffe &amp;dagger;</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    September, 2021
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Nat Commun
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/2021_09_kira_wgbs/" >
      <div class="img-hover-zoom">
        <img src="/publication/2021_09_kira_wgbs/featured_hu8781d08f274780fb57a21986f7925011_1526694_808x455_fill_q75_h2_lanczos_smart1_3.webp" height="455" width="808"
            class="article-banner" alt="Genome-wide sequencing-based identification of methylation quantitative trait loci and their role in schizophrenia risk" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/2021_09_kira_wgbs/" >Genome-wide sequencing-based identification of methylation quantitative trait loci and their role in schizophrenia risk</a>
  </div>

  
  <a href="/publication/2021_09_kira_wgbs/"  class="summary-link">
    <div class="article-style">
      <p>DNA methylation (DNAm) is an epigenetic regulator of gene expression and a hallmark of gene-environment interaction. Using whole-genome bisulfite sequencing, we have surveyed DNAm in 344 samples of human postmortem brain tissue from neurotypical subjects and individuals with schizophrenia. We identify genetic influence on local methylation levels throughout the genome, both at CpG sites and CpH sites, with 86% of SNPs and 55% of CpGs being part of methylation quantitative trait loci (meQTLs). These associations can further be clustered into regions that are differentially methylated by a given SNP, highlighting the genes and regions with which these loci are epigenetically associated. These findings can be used to better characterize schizophrenia GWAS-identified variants as epigenetic risk variants. Regions differentially methylated by schizophrenia risk-SNPs explain much of the heritability associated with risk loci, despite covering only a fraction of the genomic space. We provide a comprehensive, single base resolution view of association between genetic variation and genomic methylation, and implicate schizophrenia GWAS-associated variants as influencing the epigenetic plasticity of the brain.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.biorxiv.org/content/10.1101/2020.09.24.311878v1" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1038/s41467-021-25517-3" target="_blank" rel="noopener">
  PDF
</a>



<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/2021_09_kira_wgbs/cite.bib">
  Cite
</a>













<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1038/s41467-021-25517-3" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2021">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/nickeagles/">Nicholas J. Eagles</a></span>, <span >
      <a href="/authors/emily-e.-burke/">Emily E. Burke</a></span>, <span >
      <a href="/authors/jacob-leonard/">Jacob Leonard</a></span>, <span >
      <a href="/authors/brianna-k.-barry/">Brianna K. Barry</a></span>, <span >
      <a href="/authors/joshstolz/">Joshua M. Stolz</a></span>, <span >
      <a href="/authors/lahuuki/">Louise A. Huuki-Myers</a></span>, <span >
      <a href="/authors/badoi-n.-phan/">BaDoi N. Phan</a></span>, <span >
      <a href="/authors/violeta-larios-serrato/">Violeta Larios Serrato</a></span>, <span >
      <a href="/authors/everardo-gutierrez-millan/">Everardo Gutiérrez-Millán</a></span>, <span >
      <a href="/authors/israel-aguilar-ordonez/">Israel Aguilar-Ordoñez</a></span>, <span >
      <a href="/authors/andrew-e.-jaffe/">Andrew E. Jaffe</a></span>, <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    May, 2021
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>BMC Bioinformatics</em>
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/2021_05_speaqeasy/" >
      <div class="img-hover-zoom">
        <img src="/publication/2021_05_speaqeasy/featured_hu3ff9cf35ce6cca5d8f3ae598625549f8_55174_808x455_fill_q75_h2_lanczos_smart1_3.webp" height="455" width="808"
            class="article-banner" alt="SPEAQeasy: a scalable pipeline for expression analysis and quantification for R/Bioconductor-powered RNA-seq analyses" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/2021_05_speaqeasy/" >SPEAQeasy: a scalable pipeline for expression analysis and quantification for R/Bioconductor-powered RNA-seq analyses</a>
  </div>

  
  <a href="/publication/2021_05_speaqeasy/"  class="summary-link">
    <div class="article-style">
      <p><strong>Background</strong>. RNA sequencing (RNA-seq) is a common and widespread biological assay, and an increasing amount of data is generated with it. In practice, there are a large number of individual steps a researcher must perform before raw RNA-seq reads yield directly valuable information, such as differential gene expression data. Existing software tools are typically specialized, only performing one step–such as alignment of reads to a reference genome–of a larger workflow. The demand for a more comprehensive and reproducible workflow has led to the production of a number of publicly available RNA-seq pipelines. However, we have found that most require computational expertise to set up or share among several users, are not actively maintained, or lack features we have found to be important in our own analyses. <strong>Results</strong>. In response to these concerns, we have developed a Scalable Pipeline for Expression Analysis and Quantification (SPEAQeasy), which is easy to install and share, and provides a bridge towards R/Bioconductor downstream analysis solutions. SPEAQeasy is portable across computational frameworks (SGE, SLURM, local, docker integration) and different configuration files are provided (<a href="http://research.libd.org/SPEAQeasy/%29" target="_blank" rel="noopener">http://research.libd.org/SPEAQeasy/)</a>. <strong>Conclusions</strong>. SPEAQeasy is user-friendly and lowers the computational-domain entry barrier for biologists and clinicians to RNA-seq data processing as the main input file is a table with sample names and their corresponding FASTQ files. The goal is to provide a flexible pipeline that is immediately usable by researchers, regardless of their technical background or computing environment.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.biorxiv.org/content/10.1101/2020.12.11.386789v1" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1186/s12859-021-04142-3" target="_blank" rel="noopener">
  PDF
</a>



<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/2021_05_speaqeasy/cite.bib">
  Cite
</a>


  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/LieberInstitute/SPEAQeasy" target="_blank" rel="noopener">
  Code
</a>


  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/LieberInstitute/SPEAQeasy-example" target="_blank" rel="noopener">
  Dataset
</a>



  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/brainseq/">
    Project
  </a>
  





  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="http://research.libd.org/SPEAQeasy/" target="_blank" rel="noopener">
  Slides
</a>





<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1186/s12859-021-04142-3" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-3 year-2021">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/david-zhang/">David Zhang</a></span>, <span >
      <a href="/authors/regina-h.-reynolds/">Regina H. Reynolds</a></span>, <span >
      <a href="/authors/sonia-garcia-ruiz/">Sonia Garcia Ruiz</a></span>, <span >
      <a href="/authors/emil-k-gustavsson/">Emil K Gustavsson</a></span>, <span >
      <a href="/authors/sid-sethi/">Sid Sethi</a></span>, <span >
      <a href="/authors/sara-aguti/">Sara Aguti</a></span>, <span >
      <a href="/authors/ines-a.-barbosa/">Ines A. Barbosa</a></span>, <span >
      <a href="/authors/jack-j.-collier/">Jack J. Collier</a></span>, <span >
      <a href="/authors/henry-houlden/">Henry Houlden</a></span>, <span >
      <a href="/authors/robert-mcfarland/">Robert McFarland</a></span>, <span >
      <a href="/authors/francesco-muntoni/">Francesco Muntoni</a></span>, <span >
      <a href="/authors/monika-olahova/">Monika Oláhová</a></span>, <span >
      <a href="/authors/joanna-poulton/">Joanna Poulton</a></span>, <span >
      <a href="/authors/michael-simpson/">Michael Simpson</a></span>, <span >
      <a href="/authors/robert-d.s.-pitceathly/">Robert D.S. Pitceathly</a></span>, <span >
      <a href="/authors/robert-w.-taylor/">Robert W. Taylor</a></span>, <span >
      <a href="/authors/haiyan-zhou/">Haiyan Zhou</a></span>, <span >
      <a href="/authors/charu-deshpande/">Charu Deshpande</a></span>, <span >
      <a href="/authors/juan-a.-botia/">Juan A. Botia</a></span>, <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >
      <a href="/authors/mina-ryten/">Mina Ryten</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    March, 2021
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>bioRxiv</em>
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/preprint_dasper/" >
      <div class="img-hover-zoom">
        <img src="/publication/preprint_dasper/featured_hu7f2dcb628c7345deb86bd1bc881b38a0_141942_808x455_fill_q75_h2_lanczos_smart1.webp" height="455" width="808"
            class="article-banner" alt="Detection of pathogenic splicing events from RNA-sequencing data using dasper" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/preprint_dasper/" >Detection of pathogenic splicing events from RNA-sequencing data using dasper</a>
  </div>

  
  <a href="/publication/preprint_dasper/"  class="summary-link">
    <div class="article-style">
      <p>Although next-generation sequencing technologies have accelerated the discovery of novel gene-to-disease associations, many patients with suspected Mendelian diseases still leave the clinic without a genetic diagnosis. An estimated one third of these patients will have disorders caused by mutations impacting splicing. RNA-sequencing has been shown to be a promising diagnostic tool, however few methods have been developed to integrate RNA-sequencing data into the diagnostic pipeline. Here, we introduce dasper, an R/Bioconductor package that improves upon existing tools for detecting aberrant splicing by using machine learning to incorporate disruptions in exon-exon junction counts as well as coverage. dasper is designed for diagnostics, providing a rank-based report of how aberrant each splicing event looks, as well as including visualization functionality to facilitate interpretation. We validate dasper using 16 patient-derived fibroblast cell lines harbouring pathogenic variants known to impact splicing. We find that dasper is able to detect pathogenic splicing events with greater accuracy than existing LeafCutterMD or z-score approaches. Furthermore, by only applying a broad OMIM gene filter (without any variant-level filters), dasper is able to detect pathogenic splicing events within the top 10 most aberrant identified for each patient. Since using publicly available control data minimises costs associated with incorporating RNA-sequencing into diagnostic pipelines, we also investigate the use of 504 GTEx fibroblast samples as controls. We find that dasper leverages publicly available data effectively, ranking pathogenic splicing events in the top 25. Thus, we believe dasper can increase diagnostic yield for a pathogenic splicing variants and enable the efficient implementation of RNA-sequencing for diagnostics in clinical laboratories.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.biorxiv.org/content/10.1101/2021.03.29.437534v1" target="_blank" rel="noopener">
  Preprint
</a>




  





<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/preprint_dasper/cite.bib">
  Cite
</a>


  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://bioconductor.org/packages/dasper" target="_blank" rel="noopener">
  Code
</a>




  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/recount2/">
    Project
  </a>
  





  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="http://dzhang32.github.io/dasper/" target="_blank" rel="noopener">
  Slides
</a>




  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/dzhang32/dasper" target="_blank" rel="noopener">
  Source Document
</a>


<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1101/2021.03.29.437534" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2021">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/nicholas-e.-clifton/">Nicholas E. Clifton</a></span>, <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >
      <a href="/authors/emily-e.-burke/">Emily E. Burke</a></span>, <span >
      <a href="/authors/antonio-f.-pardinas/">Antonio F. Pardiñas</a></span>, <span >
      <a href="/authors/janet-c.-harwood/">Janet C. Harwood</a></span>, <span >
      <a href="/authors/arianna-di-florio/">Arianna Di Florio</a></span>, <span >
      <a href="/authors/james-t.-r.-walters/">James T. R. Walters</a></span>, <span >
      <a href="/authors/michael-j.-owen/">Michael J. Owen</a></span>, <span >
      <a href="/authors/michael-c.-odonovan/">Michael C. O’Donovan</a></span>, <span >
      <a href="/authors/daniel-r.-weinberger/">Daniel R. Weinberger</a></span>, <span >
      <a href="/authors/peter-a.-holmans/">Peter A. Holmans</a></span>, <span >
      <a href="/authors/andrew-e.-jaffe-dagger/">Andrew E. Jaffe &amp;dagger;</a></span>, <span >
      <a href="/authors/jeremy-hall-dagger/">Jeremy Hall &amp;dagger;</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    March, 2021
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Biological Psychiatry</em>
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/2021_03_clifton/" >
      <div class="img-hover-zoom">
        <img src="/publication/2021_03_clifton/featured_hudaae9803c820c0e093baaf2448b9983d_63487_808x455_fill_q75_h2_lanczos_smart1.webp" height="455" width="808"
            class="article-banner" alt="Developmental Profile of Psychiatric Risk Associated With Voltage-Gated Cation Channel Activity" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/2021_03_clifton/" >Developmental Profile of Psychiatric Risk Associated With Voltage-Gated Cation Channel Activity</a>
  </div>

  
  <a href="/publication/2021_03_clifton/"  class="summary-link">
    <div class="article-style">
      <p><strong>Background</strong>. Recent breakthroughs in psychiatric genetics have implicated biological pathways onto which genetic risk for psychiatric disorders converges. However, these studies do not reveal the developmental time point(s) at which these pathways are relevant. <strong>Methods</strong>. We aimed to determine the relationship between psychiatric risk and developmental gene expression relating to discrete biological pathways. We used postmortem RNA sequencing data (BrainSeq and BrainSpan) from brain tissue at multiple prenatal and postnatal time points, with summary statistics from recent genome-wide association studies of schizophrenia, bipolar disorder, and major depressive disorder. We prioritized gene sets for overall enrichment of association with each disorder and then tested the relationship between the association of their constituent genes with their relative expression at each developmental stage. <strong>Results</strong>. We observed relationships between the expression of genes involved in voltage-gated cation channel activity during early midfetal, adolescence, and early adulthood time points and association with schizophrenia and bipolar disorder, such that genes more strongly associated with these disorders had relatively low expression during early midfetal development and higher expression during adolescence and early adulthood. The relationship with schizophrenia was strongest for the subset of genes related to calcium channel activity, while for bipolar disorder, the relationship was distributed between calcium and potassium channel activity genes. <strong>Conclusions</strong>. Our results indicate periods during development when biological pathways related to the activity of calcium and potassium channels may be most vulnerable to the effects of genetic variants conferring risk for psychiatric disorders. Furthermore, they indicate key time points and potential targets for disorder-specific therapeutic interventions.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1016/j.biopsych.2021.03.009" target="_blank" rel="noopener">
  PDF
</a>



<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/2021_03_clifton/cite.bib">
  Cite
</a>





  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/brainseq/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1016/j.biopsych.2021.03.009" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2021">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/christopher-wilks/">Christopher Wilks</a></span>, <span >
      <a href="/authors/omar-ahmed/">Omar Ahmed</a></span>, <span >
      <a href="/authors/daniel-n-baker/">Daniel N Baker</a></span>, <span >
      <a href="/authors/david-zhang/">David Zhang</a></span>, <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >
      <a href="/authors/ben-langmead/">Ben Langmead</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    March, 2021
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Oxford Bioinformatics</em>
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/2021_03_megadepth/" >
      <div class="img-hover-zoom">
        <img src="/publication/2021_03_megadepth/featured_hu2773785613cc3a641d879aa2cf77c415_200120_808x455_fill_q75_h2_lanczos_smart1_3.webp" height="455" width="808"
            class="article-banner" alt="Megadepth: efficient coverage quantification for BigWigs and BAMs" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/2021_03_megadepth/" >Megadepth: efficient coverage quantification for BigWigs and BAMs</a>
  </div>

  
  <a href="/publication/2021_03_megadepth/"  class="summary-link">
    <div class="article-style">
      <p><strong>Motivation</strong>. A common way to summarize sequencing datasets is to quantify data lying within genes or other genomic intervals. This can be slow and can require different tools for different input file types. <strong>Results</strong>. Megadepth is a fast tool for quantifying alignments and coverage for BigWig and BAM/CRAM input files, using substantially less memory than the next-fastest competitor. Megadepth can summarize coverage within all disjoint intervals of the Gencode V35 gene annotation for more than 19 000 GTExV8 BigWig files in approximately 1 h using 32 threads. Megadepth is available both as a command-line tool and as an R/Bioconductor package providing much faster quantification compared to the rtracklayer package. <strong>Availability and implementation</strong>: <a href="https://github.com/ChristopherWilks/megadepth" target="_blank" rel="noopener">https://github.com/ChristopherWilks/megadepth</a>, <a href="https://bioconductor.org/packages/megadepth" target="_blank" rel="noopener">https://bioconductor.org/packages/megadepth</a>.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.biorxiv.org/content/10.1101/2020.12.17.423317v1" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1093/bioinformatics/btab152" target="_blank" rel="noopener">
  PDF
</a>



<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/2021_03_megadepth/cite.bib">
  Cite
</a>


  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://bioconductor.org/packages/megadepth" target="_blank" rel="noopener">
  Code
</a>











  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/ChristopherWilks/megadepth" target="_blank" rel="noopener">
  Source Document
</a>


<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1093/bioinformatics/btab152" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2021">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/kristen-r-maynard-____/">Kristen R Maynard __*__</a></span>, <span >
      <a href="/authors/__leonardo-collado-torres__/authors/admin-____/">[__Leonardo Collado-Torres__](/authors/admin) __*__</a></span>, <span >
      <a href="/authors/lukas-m.-weber/">Lukas M. Weber</a></span>, <span >
      <a href="/authors/cedric-uytingco/">Cedric Uytingco</a></span>, <span >
      <a href="/authors/brianna-k.-barry/">Brianna K. Barry</a></span>, <span >
      <a href="/authors/stephen-r.-williams/">Stephen R. Williams</a></span>, <span >
      <a href="/authors/joseph-l.-catallini-ii/">Joseph L. Catallini II</a></span>, <span >
      <a href="/authors/matthew-n.-tran/">Matthew N. Tran</a></span>, <span >
      <a href="/authors/zachary-besich/">Zachary Besich</a></span>, <span >
      <a href="/authors/madhavi-tippani/">Madhavi Tippani</a></span>, <span >
      <a href="/authors/jennifer-chew/">Jennifer Chew</a></span>, <span >
      <a href="/authors/yifeng-yin/">Yifeng Yin</a></span>, <span >
      <a href="/authors/joel-e.-kleinman/">Joel E. Kleinman</a></span>, <span >
      <a href="/authors/thomas-m.-hyde/">Thomas M. Hyde</a></span>, <span >
      <a href="/authors/nikhil-rao/">Nikhil Rao</a></span>, <span >
      <a href="/authors/stephanie-c.-hicks/">Stephanie C. Hicks</a></span>, <span >
      <a href="/authors/keri-martinowich-dagger/">Keri Martinowich &amp;dagger;</a></span>, <span >
      <a href="/authors/andrew-e-jaffe-dagger/">Andrew E Jaffe &amp;dagger;</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    February, 2021
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Nature Neuroscience</em>
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/2021_02_spatiallibd/" >
      <div class="img-hover-zoom">
        <img src="/publication/2021_02_spatiallibd/featured_hu62124e2a037f7a24ac3aea52e85e2499_360394_808x455_fill_q75_h2_lanczos_smart1.webp" height="455" width="808"
            class="article-banner" alt="Transcriptome-scale spatial gene expression in the human dorsolateral prefrontal cortex" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/2021_02_spatiallibd/" >Transcriptome-scale spatial gene expression in the human dorsolateral prefrontal cortex</a>
  </div>

  
  <a href="/publication/2021_02_spatiallibd/"  class="summary-link">
    <div class="article-style">
      <p>We used the 10x Genomics Visium platform to define the spatial topography of gene expression in the six-layered human dorsolateral prefrontal cortex. We identified extensive layer-enriched expression signatures and refined associations to previous laminar markers. We overlaid our laminar expression signatures on large-scale single nucleus RNA-sequencing data, enhancing spatial annotation of expression-driven clusters. By integrating neuropsychiatric disorder gene sets, we showed differential layer-enriched expression of genes associated with schizophrenia and autism spectrum disorder, highlighting the clinical relevance of spatially defined expression. We then developed a data-driven framework to define unsupervised clusters in spatial transcriptomics data, which can be applied to other tissues or brain regions in which morphological architecture is not as well defined as cortical laminae. Last, we created a web application for the scientific community to explore these raw and summarized data to augment ongoing neuroscience and spatial transcriptomics research (<a href="http://research.libd.org/spatialLIBD%29" target="_blank" rel="noopener">http://research.libd.org/spatialLIBD)</a>.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.biorxiv.org/content/10.1101/2020.02.28.969931v1" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1038/s41593-020-00787-0" target="_blank" rel="noopener">
  PDF
</a>



<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/2021_02_spatiallibd/cite.bib">
  Cite
</a>


  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://bioconductor.org/packages/spatialLIBD" target="_blank" rel="noopener">
  Code
</a>


  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/LieberInstitute/HumanPilot" target="_blank" rel="noopener">
  Dataset
</a>



  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/spatial/">
    Project
  </a>
  








  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/LieberInstitute/spatialLIBD" target="_blank" rel="noopener">
  Source Document
</a>


<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1038/s41593-020-00787-0" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-0 year-2020">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    July, 2020
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Poster presented at BioC2020
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/poster2020bioc/" >
      <div class="img-hover-zoom">
        <img src="/publication/poster2020bioc/featured_hu02753402475fbe364fdae075c891378c_899871_808x455_fill_q75_h2_lanczos_smart1_3.webp" height="455" width="808"
            class="article-banner" alt="Transcriptome-scale spatial gene expression in the human dorsolateral prefrontal cortex" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/poster2020bioc/" >Transcriptome-scale spatial gene expression in the human dorsolateral prefrontal cortex</a>
  </div>

  
  <a href="/publication/poster2020bioc/"  class="summary-link">
    <div class="article-style">
      <p>Poster on the HumanPilot project for BioC2020 using spatialLIBD.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://f1000research.com/posters/9-762" target="_blank" rel="noopener">
  PDF
</a>







  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/spatial/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.7490/f1000research.1118079.1" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2020">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/kira-a-perzel-mandell/">Kira A Perzel Mandell</a></span>, <span >
      <a href="/authors/amanda-j-price/">Amanda J Price</a></span>, <span >
      <a href="/authors/richard-wilton/">Richard Wilton</a></span>, <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >
      <a href="/authors/ran-tao/">Ran Tao</a></span>, <span >
      <a href="/authors/nickeagles/">Nicholas J. Eagles</a></span>, <span >
      <a href="/authors/alexander-s-szalay/">Alexander S Szalay</a></span>, <span >
      <a href="/authors/thomas-m-hyde/">Thomas M Hyde</a></span>, <span >
      <a href="/authors/daniel-r-weinberger/">Daniel R Weinberger</a></span>, <span >
      <a href="/authors/joel-e-kleinman/">Joel E Kleinman</a></span>, <span >
      <a href="/authors/andrew-e-jaffe-dagger/">Andrew E Jaffe &amp;dagger;</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    July, 2020
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Epigenetics
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/2020-07_kira_prenatal/" >
      <div class="img-hover-zoom">
        <img src="/publication/2020-07_kira_prenatal/featured_hu168b5967c6d231d1a2049e6a98f492f8_639899_808x455_fill_q75_h2_lanczos_top_3.webp" height="455" width="808"
            class="article-banner" alt="Characterizing the dynamic and functional DNA methylation landscape in the developing human cortex" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/2020-07_kira_prenatal/" >Characterizing the dynamic and functional DNA methylation landscape in the developing human cortex</a>
  </div>

  
  <a href="/publication/2020-07_kira_prenatal/"  class="summary-link">
    <div class="article-style">
      <p>DNA methylation (DNAm) is a key epigenetic regulator of gene expression across development. The developing prenatal brain is a highly dynamic tissue, but our understanding of key drivers of epigenetic variability across development is limited. We, therefore, assessed genomic methylation at over 39 million sites in the prenatal cortex using whole-genome bisulfite sequencing and found loci and regions in which methylation levels are dynamic across development. We saw that DNAm at these loci was associated with nearby gene expression and enriched for enhancer chromatin states in prenatal brain tissue. Additionally, these loci were enriched for genes associated with neuropsychiatric disorders and genes involved with neurogenesis. We also found autosomal differences in DNAm between the sexes during prenatal development, though these have less clear functional consequences. We lastly confirmed that the dynamic methylation at this critical period is specifically CpG methylation, with generally low levels of CpH methylation. Our findings provide detailed insight into prenatal brain development as well as clues to the pathogenesis of psychiatric traits seen later in life.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.biorxiv.org/content/10.1101/823781v1" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1080/15592294.2020.1786304" target="_blank" rel="noopener">
  PDF
</a>



<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/2020-07_kira_prenatal/cite.bib">
  Cite
</a>





  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1080/15592294.2020.1786304" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2020">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/yoichi-araki/">Yoichi Araki</a></span>, <span >
      <a href="/authors/ingie-hong/">Ingie Hong</a></span>, <span >
      <a href="/authors/timothy-r-gamache/">Timothy R Gamache</a></span>, <span >
      <a href="/authors/shaowen-ju/">Shaowen Ju</a></span>, <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >
      <a href="/authors/joo-heon-shin/">Joo Heon Shin</a></span>, <span >
      <a href="/authors/richard-l-huganir-dagger/">Richard L Huganir &amp;dagger;</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    June, 2020
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      eLife
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/2020-06_syngap/" >
      <div class="img-hover-zoom">
        <img src="/publication/2020-06_syngap/featured_hu0b62227d5f5954724cad787018f45d6e_382929_808x455_fill_q75_h2_lanczos_top.webp" height="455" width="808"
            class="article-banner" alt="SynGAP isoforms differentially regulate synaptic plasticity and dendritic development" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/2020-06_syngap/" >SynGAP isoforms differentially regulate synaptic plasticity and dendritic development</a>
  </div>

  
  <a href="/publication/2020-06_syngap/"  class="summary-link">
    <div class="article-style">
      <p>SynGAP is a synaptic Ras GTPase-activating protein (GAP) with four C-terminal splice variants: α1, α2, β, and γ. Although studies have implicated SYNGAP1 in several cognitive disorders, it is not clear which SynGAP isoforms contribute to disease. Here, we demonstrate that SynGAP isoforms exhibit unique spatiotemporal expression patterns and play distinct roles in neuronal and synaptic development in mouse neurons. SynGAP-α1, which undergoes liquid-liquid phase separation with PSD-95, is highly enriched in synapses and is required for LTP. In contrast, SynGAP-β, which does not bind PSD-95 PDZ domains, is less synaptically targeted and promotes dendritic arborization. A mutation in SynGAP-α1 that disrupts phase separation and synaptic targeting abolishes its ability to regulate plasticity and instead causes it to drive dendritic development like SynGAP-β. These results demonstrate that distinct intrinsic biochemical properties of SynGAP isoforms determine their function, and individual isoforms may differentially contribute to the pathogenesis of SYNGAP1-related cognitive disorders.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.biorxiv.org/content/10.1101/2020.01.28.922013v2" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://elifesciences.org/articles/56273" target="_blank" rel="noopener">
  PDF
</a>



<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/2020-06_syngap/cite.bib">
  Cite
</a>





  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/brainseq/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.7554/eLife.56273" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2020">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/joselyn-chavez/">Joselyn Chávez</a></span>, <span >
      <a href="/authors/carmina-barberena-jonas/">Carmina Barberena-Jonas</a></span>, <span >
      <a href="/authors/jesus-e-sotelo-fonseca/">Jesus E Sotelo-Fonseca</a></span>, <span >
      <a href="/authors/jose-alquicira-hernandez/">José Alquicira-Hernández</a></span>, <span >
      <a href="/authors/heladia-salgado/">Heladia Salgado</a></span>, <span >
      <a href="/authors/__leonardo-collado-torres__/authors/admin-dagger/">[__Leonardo Collado-Torres__](/authors/admin) &amp;dagger;</a></span>, <span >
      <a href="/authors/alejandro-reyes-dagger/">Alejandro Reyes &amp;dagger;</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    June, 2020
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Bioinformatics
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/2020-06_regutools/" >
      <div class="img-hover-zoom">
        <img src="/publication/2020-06_regutools/featured_hu7cb23724d53b4f1bab5c2fdb7cc2d2f3_294111_808x455_fill_q75_h2_lanczos_top_3.webp" height="455" width="808"
            class="article-banner" alt="Programmatic access to bacterial regulatory networks with regutools" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/2020-06_regutools/" >Programmatic access to bacterial regulatory networks with regutools</a>
  </div>

  
  <a href="/publication/2020-06_regutools/"  class="summary-link">
    <div class="article-style">
      <p>RegulonDB has collected, harmonized and centralized data from hundreds of experiments for nearly two decades and is considered a point of reference for transcriptional regulation in Escherichia coli K12. Here, we present the regutools R package to facilitate programmatic access to RegulonDB data in computational biology. regutools gives researchers the possibility of writing reproducible workflows with automated queries to RegulonDB. The regutools package serves as a bridge between RegulonDB data and the Bioconductor ecosystem by reusing the data structures and statistical methods powered by other Bioconductor packages. We demonstrate the integration of regutools with Bioconductor by analyzing transcription factor DNA binding sites and transcriptional regulatory networks from RegulonDB. We anticipate that regutools will serve as a useful building block in our progress to further our understanding of gene regulatory networks.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1101/2020.04.29.068551" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1093/bioinformatics/btaa575" target="_blank" rel="noopener">
  PDF
</a>



<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/2020-06_regutools/cite.bib">
  Cite
</a>


  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://bioconductor.org/packages/regutools" target="_blank" rel="noopener">
  Code
</a>




  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/">
    Project
  </a>
  








  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/ComunidadBioInfo/regutools" target="_blank" rel="noopener">
  Source Document
</a>


<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1093/bioinformatics/btaa575" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2020">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/david-zhang-____/">David Zhang __*__</a></span>, <span >
      <a href="/authors/sebastian-guelfi-____/">Sebastian Guelfi __*__</a></span>, <span >
      <a href="/authors/sonia-garcia-ruiz/">Sonia Garcia Ruiz</a></span>, <span >
      <a href="/authors/beatrice-costa/">Beatrice Costa</a></span>, <span >
      <a href="/authors/regina-h-reynolds/">Regina H Reynolds</a></span>, <span >
      <a href="/authors/karishma-dsa/">Karishma D&#39;Sa</a></span>, <span >
      <a href="/authors/wenfei-liu/">Wenfei Liu</a></span>, <span >
      <a href="/authors/thomas-courtin/">Thomas Courtin</a></span>, <span >
      <a href="/authors/apeterson/">Amy Peterson</a></span>, <span >
      <a href="/authors/andrew-e-jaffe/">Andrew E Jaffe</a></span>, <span >
      <a href="/authors/john-hardy/">John Hardy</a></span>, <span >
      <a href="/authors/juan-botia/">Juan Botia</a></span>, <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >
      <a href="/authors/mina-ryten/">Mina Ryten</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    June, 2020
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Science Advances
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/2020-06_omimderfinder/" >
      <div class="img-hover-zoom">
        <img src="/publication/2020-06_omimderfinder/featured_hua363891e5ce89fa451d0d5de868c889a_190185_808x455_fill_q75_h2_lanczos_smart1.webp" height="455" width="808"
            class="article-banner" alt="Incomplete annotation has a disproportionate impact on our understanding of Mendelian and complex neurogenetic disorders" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/2020-06_omimderfinder/" >Incomplete annotation has a disproportionate impact on our understanding of Mendelian and complex neurogenetic disorders</a>
  </div>

  
  <a href="/publication/2020-06_omimderfinder/"  class="summary-link">
    <div class="article-style">
      <p>Growing evidence suggests that human gene annotation remains incomplete; however, it is unclear how this affects different tissues and our understanding of different disorders. Here, we detect previously unannotated transcription from Genotype-Tissue Expression RNA sequencing data across 41 human tissues. We connect this unannotated transcription to known genes, confirming that human gene annotation remains incomplete, even among well-studied genes including 63% of the Online Mendelian Inheritance in Man–morbid catalog and 317 neurodegeneration-associated genes. We find the greatest abundance of unannotated transcription in brain and genes highly expressed in brain are more likely to be reannotated. We explore examples of reannotated disease genes, such as SNCA, for which we experimentally validate a previously unidentified, brain-specific, potentially protein-coding exon. We release all tissue-specific transcriptomes through vizER: <a href="http://rytenlab.com/browser/app/vizER" target="_blank" rel="noopener">http://rytenlab.com/browser/app/vizER</a>. We anticipate that this resource will facilitate more accurate genetic analysis, with the greatest impact on our understanding of Mendelian and complex neurogenetic disorders.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.biorxiv.org/content/10.1101/499103v1" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1126/sciadv.aay8299" target="_blank" rel="noopener">
  PDF
</a>



<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/2020-06_omimderfinder/cite.bib">
  Cite
</a>





  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/derfinder/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1126/sciadv.aay8299" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2020">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/andrew-e-jaffe-____-dagger/">Andrew E Jaffe __*__ &amp;dagger;</a></span>, <span >
      <a href="/authors/daniel-j-hoeppner-____/">Daniel J Hoeppner __*__</a></span>, <span >
      <a href="/authors/takeshi-saito/">Takeshi Saito</a></span>, <span >
      <a href="/authors/lou-blanpain/">Lou Blanpain</a></span>, <span >
      <a href="/authors/joy-ukaigwe/">Joy Ukaigwe</a></span>, <span >
      <a href="/authors/emily-e-burke/">Emily E Burke</a></span>, <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >
      <a href="/authors/ran-rato/">Ran Rato</a></span>, <span >
      <a href="/authors/katsunori-tajinda/">Katsunori Tajinda</a></span>, <span >
      <a href="/authors/kristen-r-maynard/">Kristen R Maynard</a></span>, <span >
      <a href="/authors/matthew-n-tran/">Matthew N Tran</a></span>, <span >
      <a href="/authors/keri-martinowich/">Keri Martinowich</a></span>, <span >
      <a href="/authors/amy-deep-soboslay/">Amy Deep-Soboslay</a></span>, <span >
      <a href="/authors/joo-heon-shin/">Joo Heon Shin</a></span>, <span >
      <a href="/authors/joel-e-kleinman/">Joel E Kleinman</a></span>, <span >
      <a href="/authors/daniel-r-weinberger/">Daniel R Weinberger</a></span>, <span >
      <a href="/authors/mitsuyuki-matsumoto-dagger/">Mitsuyuki Matsumoto &amp;dagger;</a></span>, <span >
      <a href="/authors/thomas-m-hyde-dagger/">Thomas M Hyde &amp;dagger;</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    March, 2020
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Nature Neuroscience</em>
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/2020-03_dglcm/" >
      <div class="img-hover-zoom">
        <img src="/publication/2020-03_dglcm/featured_hud0c585195ca5ba87424a4f53610a5ab3_994655_808x455_fill_q75_h2_lanczos_smart1_3.webp" height="455" width="808"
            class="article-banner" alt="Profiling gene expression in the human dentate gyrus granule cell layer reveals insights into schizophrenia and its genetic risk" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/2020-03_dglcm/" >Profiling gene expression in the human dentate gyrus granule cell layer reveals insights into schizophrenia and its genetic risk</a>
  </div>

  
  <a href="/publication/2020-03_dglcm/"  class="summary-link">
    <div class="article-style">
      <p>Jaffe et al. profile the granule cell layer of the human hippocampus and find unique molecular associations for aging and genetic variation, as well as diagnosis with schizophrenia and its genetic risk, that were previously undiscovered in homogenate tissue.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1038/s41593-020-0604-z" target="_blank" rel="noopener">
  PDF
</a>



<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/2020-03_dglcm/cite.bib">
  Cite
</a>





  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/brainseq/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1038/s41593-020-0604-z" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2020">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/sebastian-guelfi-____/">Sebastian Guelfi __*__</a></span>, <span >
      <a href="/authors/karishma-dsa-____/">Karishma D’Sa __*__</a></span>, <span >
      <a href="/authors/juan-botia-____/">Juan Botía __*__</a></span>, <span >
      <a href="/authors/jana-vandrovcova/">Jana Vandrovcova</a></span>, <span >
      <a href="/authors/regina-h.-reynolds/">Regina H. Reynolds</a></span>, <span >
      <a href="/authors/david-zhang/">David Zhang</a></span>, <span >
      <a href="/authors/daniah-trabzuni/">Daniah Trabzuni</a></span>, <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >
      <a href="/authors/andrew-thomason/">Andrew Thomason</a></span>, <span >
      <a href="/authors/pedro-quijada-leyton/">Pedro Quijada Leyton</a></span>, <span >
      <a href="/authors/sarah-a.-gagliano/">Sarah A. Gagliano</a></span>, <span >
      <a href="/authors/mike-a.-nalls/">Mike A. Nalls</a></span>, <span >
      <a href="/authors/international-parkinsons-disease-genomics-consortium-ipdgc/">International Parkinson’s Disease Genomics Consortium (IPDGC)</a></span>, <span >
      <a href="/authors/uk-brain-expression-consortium/">UK Brain Expression Consortium</a></span>, <span >
      <a href="/authors/kerrin-s.-small/">Kerrin S. Small</a></span>, <span >
      <a href="/authors/colin-smith/">Colin Smith</a></span>, <span >
      <a href="/authors/adaikalavan-ramasamy/">Adaikalavan Ramasamy</a></span>, <span >
      <a href="/authors/john-hardy/">John Hardy</a></span>, <span >
      <a href="/authors/michael-e.-weale-dagger/">Michael E. Weale &amp;dagger;</a></span>, <span >
      <a href="/authors/mina-ryten-dagger/">Mina Ryten &amp;dagger;</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    February, 2020
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Nature Communications
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/2020-02_basalganglia/" >
      <div class="img-hover-zoom">
        <img src="/publication/2020-02_basalganglia/featured_hud2f61d6bc340e6eee2dfa5aae56c8cda_458522_808x455_fill_q75_h2_lanczos_smart1_3.webp" height="455" width="808"
            class="article-banner" alt="Regulatory sites for splicing in human basal ganglia are enriched for disease-relevant information" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/2020-02_basalganglia/" >Regulatory sites for splicing in human basal ganglia are enriched for disease-relevant information</a>
  </div>

  
  <a href="/publication/2020-02_basalganglia/"  class="summary-link">
    <div class="article-style">
      <p>Genome-wide association studies have generated an increasing number of common genetic variants associated with neurological and psychiatric disease risk. An improved understanding of the genetic control of gene expression in human brain is vital considering this is the likely modus operandum for many causal variants. However, human brain sampling complexities limit the explanatory power of brain-related expression quantitative trait loci (eQTL) and allele-specific expression (ASE) signals. We address this, using paired genomic and transcriptomic data from putamen and substantia nigra from 117 human brains, interrogating regulation at different RNA processing stages and uncovering novel transcripts. We identify disease-relevant regulatory loci, find that splicing eQTLs are enriched for regulatory information of neuron-specific genes, that ASEs provide cell-specific regulatory information with evidence for cellular specificity, and that incomplete annotation of the brain transcriptome limits interpretation of risk loci for neuropsychiatric disease. This resource of regulatory data is accessible through our web server, <a href="http://braineacv2.inf.um.es/" target="_blank" rel="noopener">http://braineacv2.inf.um.es/</a>.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.biorxiv.org/content/10.1101/591156v1" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1038/s41467-020-14483-x" target="_blank" rel="noopener">
  PDF
</a>



<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/2020-02_basalganglia/cite.bib">
  Cite
</a>





  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/recount2/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1038/s41467-020-14483-x" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2020">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/eddie-luidy-imada-____/">Eddie-Luidy Imada __*__</a></span>, <span >
      <a href="/authors/diego-fernando-sanchez-____/">Diego Fernando Sanchez __*__</a></span>, <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >
      <a href="/authors/christopher-wilks/">Christopher Wilks</a></span>, <span >
      <a href="/authors/tejasvi-matam/">Tejasvi Matam</a></span>, <span >
      <a href="/authors/wikum-dinalankara/">Wikum Dinalankara</a></span>, <span >
      <a href="/authors/aleksey-stupnikov/">Aleksey Stupnikov</a></span>, <span >
      <a href="/authors/francisco-lobo-pereira/">Francisco Lobo-Pereira</a></span>, <span >
      <a href="/authors/chi-wai-yip/">Chi-Wai Yip</a></span>, <span >
      <a href="/authors/kayoko-yasuzawa/">Kayoko Yasuzawa</a></span>, <span >
      <a href="/authors/naoto-kondo/">Naoto Kondo</a></span>, <span >
      <a href="/authors/masayoshi-itoh/">Masayoshi Itoh</a></span>, <span >
      <a href="/authors/harukazu-suzuki/">Harukazu Suzuki</a></span>, <span >
      <a href="/authors/takeya-kasukawa/">Takeya Kasukawa</a></span>, <span >
      <a href="/authors/chung-chau-hon/">Chung-Chau Hon</a></span>, <span >
      <a href="/authors/michiel-jl-de-hoon/">Michiel JL de Hoon</a></span>, <span >
      <a href="/authors/jay-w-shin/">Jay W Shin</a></span>, <span >
      <a href="/authors/piero-carninci/">Piero Carninci</a></span>, <span >
      <a href="/authors/andrew-e-jaffe/">Andrew E Jaffe</a></span>, <span >
      <a href="/authors/jeffrey-t-leek/">Jeffrey T Leek</a></span>, <span >
      <a href="/authors/alexander-favorov/">Alexander Favorov</a></span>, <span >
      <a href="/authors/gloria-r-franco/">Glória R Franco</a></span>, <span >
      <a href="/authors/benjamin-langmead-dagger/">Benjamin Langmead &amp;dagger;</a></span>, <span >
      <a href="/authors/luigi-marchionni-dagger/">Luigi Marchionni &amp;dagger;</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    February, 2020
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Genome Res
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/2020-02_fantomrecount2/" >
      <div class="img-hover-zoom">
        <img src="/publication/2020-02_fantomrecount2/featured_hu70239331444135becdd67206afa4f719_482326_808x455_fill_q75_h2_lanczos_top_3.webp" height="455" width="808"
            class="article-banner" alt="Recounting the FANTOM CAGE–Associated Transcriptome" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/2020-02_fantomrecount2/" >Recounting the FANTOM CAGE–Associated Transcriptome</a>
  </div>

  
  <a href="/publication/2020-02_fantomrecount2/"  class="summary-link">
    <div class="article-style">
      <p>Long noncoding RNAs (lncRNAs) have emerged as key coordinators of biological and cellular processes. Characterizing lncRNA expression across cells and tissues is key to understanding their role in determining phenotypes including human diseases. We present here FC-R2, a comprehensive expression atlas across a broadly defined human transcriptome, inclusive of over 109,000 coding and noncoding genes, as described in the FANTOM CAGE-Associated Transcriptome (FANTOM-CAT) study. This atlas greatly extends the gene annotation used in the original recount2 resource. We demonstrate the utility of the FC-R2 atlas by reproducing key findings from published large studies and by generating new results across normal and diseased human samples. In particular, we (a) identify tissue-specific transcription profiles for distinct classes of coding and noncoding genes, (b) perform differential expression analysis across thirteen cancer types, identifying novel noncoding genes potentially involved in tumor pathogenesis and progression, and (c) confirm the prognostic value for several enhancers lncRNAs expression in cancer. Our resource is instrumental for the systematic molecular characterization of lncRNA by the FANTOM6 Consortium. In conclusion, comprised of over 70,000 samples, the FC-R2 atlas will empower other researchers to investigate functions and biological roles of both known coding genes and novel lncRNAs.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.biorxiv.org/content/10.1101/659490v4" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="http://doi.org/10.1101/gr.254656.119" target="_blank" rel="noopener">
  PDF
</a>



<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/2020-02_fantomrecount2/cite.bib">
  Cite
</a>


  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/eddieimada/fcr2" target="_blank" rel="noopener">
  Code
</a>


  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://jhubiostatistics.shinyapps.io/recount/" target="_blank" rel="noopener">
  Dataset
</a>



  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/recount2/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1101/gr.254656" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2020">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/emily-e-burke-____/">Emily E Burke __*__</a></span>, <span >
      <a href="/authors/joshua-g-chenoweth-____/">Joshua G Chenoweth __*__</a></span>, <span >
      <a href="/authors/joo-heon-shin/">Joo Heon Shin</a></span>, <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >
      <a href="/authors/suel-kee-kim/">Suel Kee Kim</a></span>, <span >
      <a href="/authors/nicola-micali/">Nicola Micali</a></span>, <span >
      <a href="/authors/yanhong-wang/">Yanhong Wang</a></span>, <span >
      <a href="/authors/carlo-colantuoni/">Carlo Colantuoni</a></span>, <span >
      <a href="/authors/richard-e-straub/">Richard E Straub</a></span>, <span >
      <a href="/authors/daniel-j-hoeppner/">Daniel J Hoeppner</a></span>, <span >
      <a href="/authors/huei-ying-chen/">Huei-Ying Chen</a></span>, <span >
      <a href="/authors/alana-sellers/">Alana Sellers</a></span>, <span >
      <a href="/authors/kamel-shibbani/">Kamel Shibbani</a></span>, <span >
      <a href="/authors/gregory-r-hamersky/">Gregory R Hamersky</a></span>, <span >
      <a href="/authors/marcelo-diaz-bustamante/">Marcelo Diaz Bustamante</a></span>, <span >
      <a href="/authors/badoi-n-phan/">BaDoi N Phan</a></span>, <span >
      <a href="/authors/william-s-ulrich/">William S Ulrich</a></span>, <span >
      <a href="/authors/cristian-valencia/">Cristian Valencia</a></span>, <span >
      <a href="/authors/amritha-jaishankar/">Amritha Jaishankar</a></span>, <span >
      <a href="/authors/amanda-j-price/">Amanda J Price</a></span>, <span >
      <a href="/authors/anandita-rajpurohit/">Anandita Rajpurohit</a></span>, <span >
      <a href="/authors/stephen-a-semick/">Stephen A Semick</a></span>, <span >
      <a href="/authors/roland-burli/">Roland Bürli</a></span>, <span >
      <a href="/authors/james-c-barrow/">James C Barrow</a></span>, <span >
      <a href="/authors/daniel-j-hiler/">Daniel J Hiler</a></span>, <span >
      <a href="/authors/stephanie-cerceo-page/">Stephanie Cerceo Page</a></span>, <span >
      <a href="/authors/keri-martinowich/">Keri Martinowich</a></span>, <span >
      <a href="/authors/thomas-m-hyde/">Thomas M Hyde</a></span>, <span >
      <a href="/authors/joel-e-kleinman/">Joel E Kleinman</a></span>, <span >
      <a href="/authors/karen-f-berman/">Karen F Berman</a></span>, <span >
      <a href="/authors/jose-a-apud/">José A Apud</a></span>, <span >
      <a href="/authors/alan-j-cross/">Alan J Cross</a></span>, <span >
      <a href="/authors/nick-j-brandon/">Nick J Brandon</a></span>, <span >
      <a href="/authors/daniel-r-weinberger/">Daniel R Weinberger</a></span>, <span >
      <a href="/authors/brady-j-maher/">Brady J Maher</a></span>, <span >
      <a href="/authors/ronald-dg-mckay-dagger/">Ronald DG McKay &amp;dagger;</a></span>, <span >
      <a href="/authors/andrew-e-jaffe-dagger/">Andrew E Jaffe &amp;dagger;</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    January, 2020
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Nature Communications
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/2020-01_libdstemcell/" >
      <div class="img-hover-zoom">
        <img src="/publication/2020-01_libdstemcell/featured_huab1babfa32d082e3c5c715e55287e144_1130004_808x455_fill_q75_h2_lanczos_smart1_3.webp" height="455" width="808"
            class="article-banner" alt="Dissecting transcriptomic signatures of neuronal differentiation and maturation using iPSCs" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/2020-01_libdstemcell/" >Dissecting transcriptomic signatures of neuronal differentiation and maturation using iPSCs</a>
  </div>

  
  <a href="/publication/2020-01_libdstemcell/"  class="summary-link">
    <div class="article-style">
      <p>Human induced pluripotent stem cells (hiPSCs) are a powerful model of neural differentiation and maturation. We present a hiPSC transcriptomics resource on corticogenesis from 5 iPSC donor and 13 subclonal lines across 9 time points over 5 broad conditions: self-renewal, early neuronal differentiation, neural precursor cells (NPCs), assembled rosettes, and differentiated neuronal cells. We identify widespread changes in the expression of both individual features and global patterns of transcription. We next demonstrate that co-culturing human NPCs with rodent astrocytes results in mutually synergistic maturation, and that cell type-specific expression data can be extracted using only sequencing read alignments without cell sorting. We lastly adapt a previously generated RNA deconvolution approach to single-cell expression data to estimate the relative neuronal maturity of iPSC-derived neuronal cultures and human brain tissue. Using many public datasets, we demonstrate neuronal cultures are maturationally heterogeneous but contain subsets of neurons more mature than previously observed.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.biorxiv.org/content/early/2018/07/31/380758" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1038/s41467-019-14266-z" target="_blank" rel="noopener">
  PDF
</a>



<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/2020-01_libdstemcell/cite.bib">
  Cite
</a>





  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1038/s41467-019-14266-z" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2019">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/amanda-j.-price-____/">Amanda J. Price __*__</a></span>, <span >
      <a href="/authors/__leonardo-collado-torres__/authors/admin-____/">[__Leonardo Collado-Torres__](/authors/admin) __*__</a></span>, <span >
      <a href="/authors/nikolay-a.-ivanov/">Nikolay A. Ivanov</a></span>, <span >
      <a href="/authors/wei-xia/">Wei Xia</a></span>, <span >
      <a href="/authors/emily-e.-burke/">Emily E. Burke</a></span>, <span >
      <a href="/authors/joo-heon-shin/">Joo Heon Shin</a></span>, <span >
      <a href="/authors/ran-tao/">Ran Tao</a></span>, <span >
      <a href="/authors/liang-ma/">Liang Ma</a></span>, <span >
      <a href="/authors/yankai-jia/">Yankai Jia</a></span>, <span >
      <a href="/authors/thomas-m.-hyde/">Thomas M. Hyde</a></span>, <span >
      <a href="/authors/joel-e.-kleinman/">Joel E. Kleinman</a></span>, <span >
      <a href="/authors/daniel-r.-weinberger/">Daniel R. Weinberger</a></span>, <span >
      <a href="/authors/andrew-e-jaffe-dagger/">Andrew E Jaffe &amp;dagger;</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    September, 2019
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Genome Biology
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/2019-09_dnam_mcph_neuron_glia/" >
      <div class="img-hover-zoom">
        <img src="/publication/2019-09_dnam_mcph_neuron_glia/featured_hu60fca4d401fa100a1ee1e2e878287825_334774_808x455_fill_q75_h2_lanczos_smart1_3.webp" height="455" width="808"
            class="article-banner" alt="Divergent neuronal DNA methylation patterns across human cortical development reveal critical periods and a unique role of CpH methylation" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/2019-09_dnam_mcph_neuron_glia/" >Divergent neuronal DNA methylation patterns across human cortical development reveal critical periods and a unique role of CpH methylation</a>
  </div>

  
  <a href="/publication/2019-09_dnam_mcph_neuron_glia/"  class="summary-link">
    <div class="article-style">
      <p><strong>Background</strong>: DNA methylation (DNAm) is a critical regulator of both development and cellular identity and shows unique patterns in neurons. To better characterize maturational changes in DNAm patterns in these cells, we profile the DNAm landscape at single-base resolution across the first two decades of human neocortical development in NeuN+ neurons using whole-genome bisulfite sequencing and compare them to non-neurons (primarily glia) and prenatal homogenate cortex. <strong>Results</strong>: We show that DNAm changes more dramatically during the first 5 years of postnatal life than during the entire remaining period. We further refine global patterns of increasingly divergent neuronal CpG and CpH methylation (mCpG and mCpH) into six developmental trajectories and find that in contrast to genome-wide patterns, neighboring mCpG and mCpH levels within these regions are highly correlated. We integrate paired RNAseq data and identify putative regulation of hundreds of transcripts and their splicing events exclusively by mCpH levels, independently from mCpG levels, across this period. We finally explore the relationship between DNAm patterns and development of brain-related phenotypes and find enriched heritability for many phenotypes within identified DNAm features. <strong>Conclusions</strong>: By profiling DNAm changes in NeuN-sorted neurons over the span of human cortical development, we identify novel, dynamic regions of DNAm that would be masked in homogenate DNAm data; expand on the relationship between CpG methylation, CpH methylation, and gene expression; and find enrichment particularly for neuropsychiatric diseases in genomic regions with cell type-specific, developmentally dynamic DNAm patterns. <strong>Keywords</strong>: DNA methylation, Neurodevelopment, Gene expression, Non-CpG methylation.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.biorxiv.org/content/10.1101/428391v1" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://genomebiology.biomedcentral.com/track/pdf/10.1186/s13059-019-1805-1" target="_blank" rel="noopener">
  PDF
</a>



<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/2019-09_dnam_mcph_neuron_glia/cite.bib">
  Cite
</a>


  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/LieberInstitute/brain-epigenomics/" target="_blank" rel="noopener">
  Code
</a>




<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://jhubiostatistics.shinyapps.io/wgbsExprs/" target="_blank" rel="noopener">
  Project
</a>









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1186/s13059-019-1805-1" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2019">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/carrie-wright-____/">Carrie Wright __*__</a></span>, <span >
      <a href="/authors/anandita-rajpurohit-____/">Anandita Rajpurohit __*__</a></span>, <span >
      <a href="/authors/emily-e.-burke/">Emily E. Burke</a></span>, <span >
      <a href="/authors/courtney-williams/"> Courtney Williams</a></span>, <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >
      <a href="/authors/martha-kimos/">Martha Kimos</a></span>, <span >
      <a href="/authors/nicholas-j.-brandon/">Nicholas J. Brandon</a></span>, <span >
      <a href="/authors/alan-j.-cross/">Alan J. Cross</a></span>, <span >
      <a href="/authors/andrew-e.-jaffe/">Andrew E. Jaffe</a></span>, <span >
      <a href="/authors/daniel-r.-weinberger-dagger/">Daniel R. Weinberger  &amp;dagger;</a></span>, <span >
      <a href="/authors/joo-heon-shin-dagger/">Joo Heon Shin  &amp;dagger;</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    June, 2019
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      BMC Genomics
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/2019-06_mirna_kit_comp/" >
      <div class="img-hover-zoom">
        <img src="/publication/2019-06_mirna_kit_comp/featured_hu9400eccb33424d02ad41838c423b2097_394531_808x455_fill_q75_h2_lanczos_smart1_3.webp" height="455" width="808"
            class="article-banner" alt="Comprehensive assessment of multiple biases in small RNA sequencing reveals significant differences in the performance of widely used methods" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/2019-06_mirna_kit_comp/" >Comprehensive assessment of multiple biases in small RNA sequencing reveals significant differences in the performance of widely used methods</a>
  </div>

  
  <a href="/publication/2019-06_mirna_kit_comp/"  class="summary-link">
    <div class="article-style">
      <p>Background: RNA sequencing offers advantages over other quantification methods for microRNA (miRNA), yet numerous biases make reliable quantification challenging. Previous evaluations of these biases have focused on adapter ligation bias with limited evaluation of reverse transcription bias or amplification bias. Furthermore, evaluations of the quantification of isomiRs (miRNA isoforms) or the influence of starting amount on performance have been very limited. No study had yet evaluated the quantification of isomiRs of altered length or compared the consistency of results derived from multiple moderate starting inputs. We therefore evaluated quantifications of miRNA and isomiRs using four library preparation kits, with various starting amounts, as well as quantifications following removal of duplicate reads using unique molecular identifiers (UMIs) to mitigate reverse transcription and amplification biases. Results: All methods resulted in false isomiR detection; however, the adapter-free method tested was especially prone to false isomiR detection. We demonstrate that using UMIs improves accuracy and we provide a guide for input amounts to improve consistency. Conclusions: Our data show differences and limitations of current methods, thus raising concerns about the validity of quantification of miRNA and isomiRs across studies. We advocate for the use of UMIs to improve accuracy and reliability of miRNA quantifications.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.biorxiv.org/content/10.1101/445437v3" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-019-5870-3f" target="_blank" rel="noopener">
  PDF
</a>



<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/2019-06_mirna_kit_comp/cite.bib">
  Cite
</a>


  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/LieberInstitute/miRNA_Kit_Comparison" target="_blank" rel="noopener">
  Code
</a>




  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1186/s12864-019-5870-3" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2019">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >
      <a href="/authors/emily-e-burke/">Emily E Burke</a></span>, <span >
      <a href="/authors/apeterson/">Amy Peterson</a></span>, <span >
      <a href="/authors/jooheon-shin/">JooHeon Shin</a></span>, <span >
      <a href="/authors/richard-e-straub/">Richard E Straub</a></span>, <span >
      <a href="/authors/anandita-rajpurohit/">Anandita Rajpurohit</a></span>, <span >
      <a href="/authors/stephen-a-semick/">Stephen A Semick</a></span>, <span >
      <a href="/authors/william-s-ulrich/">William S Ulrich</a></span>, <span >
      <a href="/authors/brainseq-consortium/">BrainSeq Consortium</a></span>, <span >
      <a href="/authors/amanda-j-price/">Amanda J Price</a></span>, <span >
      <a href="/authors/cristian-valencia/">Cristian Valencia</a></span>, <span >
      <a href="/authors/ran-tao/">Ran Tao</a></span>, <span >
      <a href="/authors/amy-deep-soboslay/">Amy Deep-Soboslay</a></span>, <span >
      <a href="/authors/thomas-m-hyde/">Thomas M Hyde</a></span>, <span >
      <a href="/authors/joel-e-kleinman/">Joel E Kleinman</a></span>, <span >
      <a href="/authors/daniel-r-weinberger-dagger/">Daniel R Weinberger &amp;dagger;</a></span>, <span >
      <a href="/authors/andrew-e-jaffe-dagger/">Andrew E Jaffe &amp;dagger;</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    June, 2019
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Neuron
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/2019-06_bsp2/" >
      <div class="img-hover-zoom">
        <img src="/publication/2019-06_bsp2/featured_hu169a505711757f29f1e543af28f6c854_18730_808x455_fill_q75_h2_lanczos_top_3.webp" height="455" width="808"
            class="article-banner" alt="Regional heterogeneity in gene expression, regulation, and coherence in the frontal cortex and hippocampus across development and schizophrenia" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/2019-06_bsp2/" >Regional heterogeneity in gene expression, regulation, and coherence in the frontal cortex and hippocampus across development and schizophrenia</a>
  </div>

  
  <a href="/publication/2019-06_bsp2/"  class="summary-link">
    <div class="article-style">
      <p>The hippocampus formation, although prominently implicated in schizophrenia pathogenesis, has been overlooked in large-scale genomics efforts in the schizophrenic brain. We performed RNA-seq in hippocampi and dorsolateral prefrontal cortices (DLPFCs) from 551 individuals (286 with schizophrenia). We identified substantial regional differences in gene expression and found widespread developmental differences that were independent of cellular composition. We identified 48 and 245 differentially expressed genes (DEGs) associated with schizophrenia within the hippocampus and DLPFC, with little overlap between the brain regions. 124 of 163 (76.6%) of schizophrenia GWAS risk loci contained eQTLs in any region. Transcriptome-wide association studies in each region identified many novel schizophrenia risk features that were brain region-specific. Last, we identified potential molecular correlates of in vivo evidence of altered prefrontal-hippocampal functional coherence in schizophrenia. These results underscore the complexity and regional heterogeneity of the transcriptional correlates of schizophrenia and offer new insights into potentially causative biology.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.biorxiv.org/content/early/2018/09/26/426213" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1016/j.neuron.2019.05.013" target="_blank" rel="noopener">
  PDF
</a>



<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/2019-06_bsp2/cite.bib">
  Cite
</a>


  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/LieberInstitute/brainseq_phase2" target="_blank" rel="noopener">
  Code
</a>


  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://data.mendeley.com/datasets/3j93ybf4md/" target="_blank" rel="noopener">
  Dataset
</a>



  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/brainseq/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1016/j.neuron.2019.05.013" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-0 year-2019">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    May, 2019
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/poster2019bog/" >
      <div class="img-hover-zoom">
        <img src="/publication/poster2019bog/featured_hu9b0a18854455e950ff0a7d3159e5a719_897436_808x455_fill_q75_h2_lanczos_top_3.webp" height="455" width="808"
            class="article-banner" alt="recount-brain: a curated repository of human brain RNA-seq datasets metadata" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/poster2019bog/" >recount-brain: a curated repository of human brain RNA-seq datasets metadata</a>
  </div>

  
  <a href="/publication/poster2019bog/"  class="summary-link">
    <div class="article-style">
      <p>Poster presented at BoG2019</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="/recount-brain-bog19/" target="_blank" rel="noopener">
  PDF
</a>




  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/lcolladotor/recount-brain-bog19/" target="_blank" rel="noopener">
  Code
</a>




  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/recount2/">
    Project
  </a>
  











  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-3 year-2019">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/arazmara/">Ashkaun Razmara</a></span>, <span >
      <a href="/authors/ellis-se/">Ellis SE</a></span>, <span >
      <a href="/authors/sokolowski-dj/">Sokolowski DJ</a></span>, <span >
      <a href="/authors/davis-s/">Davis S</a></span>, <span >
      <a href="/authors/wilson-md/">Wilson MD</a></span>, <span >
      <a href="/authors/leek-jt/">Leek JT</a></span>, <span >
      <a href="/authors/jaffe-ae/">Jaffe AE</a></span>, <span >
      <a href="/authors/__l-collado-torres__/authors/admin-dagger/">[__L Collado-Torres__](/authors/admin) &amp;dagger;</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    April, 2019
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      bioRxiv 618025
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/preprint_recount_brain/" >
      <div class="img-hover-zoom">
        <img src="/publication/preprint_recount_brain/featured_huaa449328070568318c86c41bb75a245f_103596_808x455_fill_q75_h2_lanczos_smart1.webp" height="455" width="808"
            class="article-banner" alt="recount-brain: a curated repository of human brain RNA-seq datasets metadata" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/preprint_recount_brain/" >recount-brain: a curated repository of human brain RNA-seq datasets metadata</a>
  </div>

  
  <a href="/publication/preprint_recount_brain/"  class="summary-link">
    <div class="article-style">
      <p>The usability of publicly-available gene expression data is often limited by the availability of high-quality, standardized biological phenotype and experimental condition information (“metadata”). We released the recount2 project, which involved re-processing ∼70,000 samples in the Sequencing Read Archive (SRA), Genotype-Tissue Expression (GTEx), and The Cancer Genome Atlas (TCGA) projects. While samples from the latter two projects are well-characterized with extensive metadata, the ∼50,000 RNA-seq samples from SRA in recount2 are inconsistently annotated with metadata. Tissue type, sex, and library type can be estimated from the RNA sequencing (RNA-seq) data itself. However, more detailed and harder to predict metadata, like age and diagnosis, must ideally be provided by labs that deposit the data. To facilitate more analyses within human brain tissue data, we have complemented phenotype predictions by manually constructing a uniformly-curated database of public RNA-seq samples present in SRA and recount2. We describe the reproducible curation process for constructing recount-brain that involves systematic review of the primary manuscript, which can serve as a guide to annotate other studies and tissues. We further expanded recount-brain by merging it with GTEx and TCGA brain samples as well as linking to controlled vocabulary terms for tissue, Brodmann area and disease. Furthermore, we illustrate how to integrate the sample metadata in recount-brain with the gene expression data in recount2 to perform differential expression analysis. We then provide three analysis examples involving modeling postmortem interval, glioblastoma, and meta-analyses across GTEx and TCGA. Overall, recount-brain facilitates expression analyses and improves their reproducibility as individual researchers do not have to manually curate the sample metadata. recount-brain is available via the add_metadata() function from the recount Bioconductor package at bioconductor.org/packages/recount.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.biorxiv.org/content/10.1101/618025v1" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.biorxiv.org/content/early/2019/04/24/618025.full.pdf" target="_blank" rel="noopener">
  PDF
</a>



<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/preprint_recount_brain/cite.bib">
  Cite
</a>


  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/LieberInstitute/recount-brain" target="_blank" rel="noopener">
  Code
</a>




  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/recount2/">
    Project
  </a>
  



  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="/recount-brain-bog19/" target="_blank" rel="noopener">
  Poster
</a>







<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1101/618025" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2019">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/anil-k-madugundu/">Anil K Madugundu</a></span>, <span >
      <a href="/authors/chan-hyun-na/">Chan Hyun Na</a></span>, <span >
      <a href="/authors/raja-sekhar-nirujogi/">Raja Sekhar Nirujogi</a></span>, <span >
      <a href="/authors/santosh-renuse/">Santosh Renuse</a></span>, <span >
      <a href="/authors/kwang-pyo-kim/">Kwang Pyo Kim</a></span>, <span >
      <a href="/authors/kathleen-h.-burns/">Kathleen H. Burns</a></span>, <span >
      <a href="/authors/christopher-wilks/">Christopher Wilks</a></span>, <span >
      <a href="/authors/ben-langmead/">Ben Langmead</a></span>, <span >
      <a href="/authors/shannon-e.-ellis/">Shannon E. Ellis</a></span>, <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >
      <a href="/authors/marc-k.-halushka/">Marc K. Halushka</a></span>, <span >
      <a href="/authors/min-sik-kim/">Min-Sik Kim</a></span>, <span >
      <a href="/authors/akhilesh-pandey-dagger/">Akhilesh Pandey &amp;dagger;</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    April, 2019
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Proteomics
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/2019-04_recount_proteomics/" >
      <div class="img-hover-zoom">
        <img src="/publication/2019-04_recount_proteomics/featured_hu9af44dfb87f3cd13fd97284f076ba1d4_140736_808x455_fill_q75_h2_lanczos_smart1.webp" height="455" width="808"
            class="article-banner" alt="Integrated Transcriptomic and Proteomic Analysis of Primary Human Umbilical Vein Endothelial Cells" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/2019-04_recount_proteomics/" >Integrated Transcriptomic and Proteomic Analysis of Primary Human Umbilical Vein Endothelial Cells</a>
  </div>

  
  <a href="/publication/2019-04_recount_proteomics/"  class="summary-link">
    <div class="article-style">
      <p>Understanding the molecular profile of every human cell type is essential for understanding its role in normal physi-ology and disease. Technological advancements in DNA sequencing, mass spectrometry, and computational methodsallow us to carry out multiomics analyses although such approaches are not routine yet. Human umbilical vein endothe-lial cells (HUVECs) are a widely used model system to study pathological and physiological processes associated withthe cardiovascular system. In this study, next-generation sequencing and high-resolution mass spectrometry to profilethe transcriptome and proteome of primary HUVECs is employed. Analysis of 145 million paired-end reads from next-generation sequencing confirmed expression of 12 186 protein-coding genes (FPKM &gt;= 0.1), 439 novel long non-codingRNAs, and revealed 6089 novel isoforms that were not annotated in GENCODE. Proteomics analysis identifies 6477proteins including confirmation ofN-termini for 1091 proteins, isoforms for 149 proteins, and 1034 phosphosites. Adatabase search to specifically identify other post-translational modifications provide evidence for a number of modifi-cation sites on 117 proteins which include ubiquitylation, lysine acetylation, and mono-, di- and tri-methylation events.Evidence for 11 &lsquo;missing proteins&rsquo;, which are proteins for which there was insufficient or no protein level evidence, isprovided. Peptides supporting missing protein and novel events are validated by comparison of MS/MS fragmentationpatterns with synthetic peptides. Finally, 245 variant peptides derived from 207 expressed proteins in addition to alternatetranslational start sites for seven proteins and evidence for novel proteoforms for five proteins resulting from alternativesplicing are identified. Overall, it is believed that the integrated approach employed in this study is widely applicable tostudy any primary cell type for deeper molecular characterization.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://onlinelibrary.wiley.com/doi/epdf/10.1002/pmic.201800315" target="_blank" rel="noopener">
  PDF
</a>



<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/2019-04_recount_proteomics/cite.bib">
  Cite
</a>





  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1002/pmic.201800315" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2019">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/stephen-a-semick/">Stephen A Semick</a></span>, <span >
      <a href="/authors/rahul-a-bharadwaj/">Rahul A Bharadwaj</a></span>, <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >
      <a href="/authors/ran-tao/">Ran Tao</a></span>, <span >
      <a href="/authors/joo-heon-shin/">Joo Heon Shin</a></span>, <span >
      <a href="/authors/amy-deep-soboslay/">Amy Deep-Soboslay</a></span>, <span >
      <a href="/authors/james-weiss/">James Weiss</a></span>, <span >
      <a href="/authors/daniel-r-weinberger/">Daniel R Weinberger</a></span>, <span >
      <a href="/authors/thomas-m-hyde/">Thomas M Hyde</a></span>, <span >
      <a href="/authors/joel-e-kleinman/">Joel E Kleinman</a></span>, <span >
      <a href="/authors/andrew-e-jaffe-dagger/">Andrew E Jaffe &amp;dagger;</a></span>, <span >
      <a href="/authors/venkata-s-mattay-dagger/">Venkata S Mattay &amp;dagger;</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    February, 2019
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Acta Neuropathologica
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/2019-02_dnam_alzheimer/" >
      <div class="img-hover-zoom">
        <img src="/publication/2019-02_dnam_alzheimer/featured_hu1e066182b6f82934619930922facd138_606837_808x455_fill_q75_h2_lanczos_smart1_3.webp" height="455" width="808"
            class="article-banner" alt="Integrated DNA methylation and gene expression profiling across multiple brain regions implicate novel genes in Alzheimer&#39;s disease" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/2019-02_dnam_alzheimer/" >Integrated DNA methylation and gene expression profiling across multiple brain regions implicate novel genes in Alzheimer&#39;s disease</a>
  </div>

  
  <a href="/publication/2019-02_dnam_alzheimer/"  class="summary-link">
    <div class="article-style">
      <p>Late-onset Alzheimer’s disease (AD) is a complex age-related neurodegenerative disorder that likely involves epigenetic factors. To better understand the epigenetic state associated with AD, we surveyed 420,852 DNA methylation (DNAm) sites from neurotypical controls (N=49) and late-onset AD patients (N=24) across four brain regions (hippocampus, entorhinal cortex, dorsolateral prefrontal cortex and cerebellum). We identified 858 sites with robust differential methylation collectively annotated to 772 possible genes (FDR&lt;5%, within 10 kb). These sites were overrepresented in AD genetic risk loci (p=0.00655) and were enriched for changes during normal aging (p&lt;2.2×10−16), and nearby genes were enriched for processes related to cell-adhesion, immunity, and calcium homeostasis (FDR&lt;5%). To functionally validate these associations, we generated and analyzed corresponding transcriptome data to prioritize 130 genes within 10 kb of the differentially methylated sites. These 130 genes were differentially expressed between AD cases and controls and their expression was associated with nearby DNAm (p&lt;0.05). This integrated analysis implicates novel genes in Alzheimer’s disease, such as ANKRD30B. These results highlight DNAm differences in Alzheimer’s disease that have gene expression correlates, further implicating DNAm as an epigenetic mechanism underlying pathological molecular changes associated with AD. Furthermore, our framework illustrates the value of integrating epigenetic and transcriptomic data for understanding complex disease.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.biorxiv.org/content/early/2018/09/29/430603" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1007/s00401-019-01966-5" target="_blank" rel="noopener">
  PDF
</a>



<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/2019-02_dnam_alzheimer/cite.bib">
  Cite
</a>


  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/LieberInstitute/Alzheimers_DNAm" target="_blank" rel="noopener">
  Code
</a>




  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1007/s00401-019-01966-5" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2018">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/helena-kuri-magana/">Helena Kuri-Magaña</a></span>, <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >
      <a href="/authors/andrew-e-jaffe/">Andrew E Jaffe</a></span>, <span >
      <a href="/authors/humberto-valdovinos-torres/">Humberto Valdovinos-Torres</a></span>, <span >
      <a href="/authors/marbella-ovilla-munoz/">Marbella Ovilla-Muñoz</a></span>, <span >
      <a href="/authors/juan-m-tellez-sosa/">Juan M Téllez-Sosa</a></span>, <span >
      <a href="/authors/laura-c-bonifaz-alfonzo/">Laura C Bonifaz-Alfonzo</a></span>, <span >
      <a href="/authors/jesus-martinez-barnetche/">Jesús Martínez-Barnetche</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    November, 2018
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Frontiers in Immunology
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/2018-11_nccsr/" >
      <div class="img-hover-zoom">
        <img src="/publication/2018-11_nccsr/featured_hubb959b6f36053d217f6324d4d26fd23c_793639_808x455_fill_q75_h2_lanczos_top_3.webp" height="455" width="808"
            class="article-banner" alt="Non-coding Class Switch Recombination-Related Transcription in Human Normal and Pathological Immune Responses" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/2018-11_nccsr/" >Non-coding Class Switch Recombination-Related Transcription in Human Normal and Pathological Immune Responses</a>
  </div>

  
  <a href="/publication/2018-11_nccsr/"  class="summary-link">
    <div class="article-style">
      <p>Background: Antibody class switch recombination (CSR) to IgG, IgA or IgE is a hallmark of adaptive immunity, allowing antibody function diversification beyond IgM. CSR involves a deletion of the IgM/IgD constant region genes placing a new acceptor Constant (CH) gene, downstream of the VDJH exon. CSR depends on non-coding (CSRnc) transcription of donor Iμ and acceptor IH exons, located 5′ upstream of each CH coding gene. Although our knowledge of the role of CSRnc transcription has advanced greatly, its extension and importance in healthy and diseased humans is scarce. Methods: We analyzed CSRnc transcription in 70,603 publicly available RNA-seq samples, including GTEx, TCGA and the Sequence Read Archive (SRA) using recount2, an online resource consisting of normalized RNA-seq gene and exon counts, as well as coverage BigWig files that can be programmatically accessed through R. CSRnc transcription was validated with a qRT-PCR assay for Iμ, Iγ1 and Iγ3 in humans in response to vaccination. Results: We mapped IH transcription for the human IgH locus, including the less understood IGHD gene. CSRnc transcription was restricted to B cells and is widely distributed in normal adult tissues, but predominant in blood, spleen, MALT-containing tissues, visceral adipose tissue and some so-called &lsquo;immune privileged&rsquo; tissues. However, significant Iγ4 expression was found even in non-lymphoid fetal tissues. CSRnc expression in cancer tissues mimicked the expression of their normal counterparts, with notable pattern changes in some common cancer subsets. CSRnc transcription in tumors appears to result from tumor infiltration by B cells, since CSRnc transcription was not detected in corresponding tumor-derived immortal cell lines. Additionally, significantly increased Iδ transcription in ileal mucosa in Crohn&rsquo;s disease with ulceration was found. Conclusions: CSRnc transcription occurs in multiple anatomical locations beyond classical secondary lymphoid organs, representing a potentially useful marker of effector B cell responses in normal and pathological immune responses. The pattern of IH exon expression may reveal clues of the local immune response (i.e. cytokine milieu) in health and disease. This is a great example of how the public recount2 data can be used to further our understanding of transcription, including regions outside the known transcriptome.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.biorxiv.org/content/early/2018/08/03/384172" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.frontiersin.org/articles/10.3389/fimmu.2018.02679/full" target="_blank" rel="noopener">
  PDF
</a>



<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/2018-11_nccsr/cite.bib">
  Cite
</a>





  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/recount2/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.3389/fimmu.2018.02679" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-0 year-2018">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    October, 2018
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/poster2018ashg/" >
      <div class="img-hover-zoom">
        <img src="/publication/poster2018ashg/featured_huca91b25d278d606855f49f135164430e_1275613_808x455_fill_q75_h2_lanczos_top_3.webp" height="455" width="808"
            class="article-banner" alt="Regional heterogeneity in gene expression, regulation and coherence in hippocampus and dorsolateral prefrontal cortex across development and in schizophrenia" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/poster2018ashg/" >Regional heterogeneity in gene expression, regulation and coherence in hippocampus and dorsolateral prefrontal cortex across development and in schizophrenia</a>
  </div>

  
  <a href="/publication/poster2018ashg/"  class="summary-link">
    <div class="article-style">
      <p>Poster presented at ASHG2018, biodata18 and PSB2019</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="http://lcolladotor.github.io/Posters/ASHG2018.pdf" target="_blank" rel="noopener">
  PDF
</a>







  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/brainseq/">
    Project
  </a>
  











  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2018">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/semick-sa/">Semick SA</a></span>, <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >
      <a href="/authors/markunas-ca/">Markunas CA</a></span>, <span >
      <a href="/authors/shin-jh/">Shin JH</a></span>, <span >
      <a href="/authors/deep-soboslay-a/">Deep-Soboslay A</a></span>, <span >
      <a href="/authors/tao-r/">Tao R</a></span>, <span >
      <a href="/authors/huestis-ma/">Huestis MA</a></span>, <span >
      <a href="/authors/bierut-lj/">Bierut LJ</a></span>, <span >
      <a href="/authors/maher-bs/">Maher BS</a></span>, <span >
      <a href="/authors/johnson-eo/">Johnson EO</a></span>, <span >
      <a href="/authors/hyde-tm/">Hyde TM</a></span>, <span >
      <a href="/authors/weinberger-dr/">Weinberger DR</a></span>, <span >
      <a href="/authors/hancock-db/">Hancock DB</a></span>, <span >
      <a href="/authors/kleinman-je-dagger/">Kleinman JE &amp;dagger;</a></span>, <span >
      <a href="/authors/jaffe-ae-dagger/">Jaffe AE &amp;dagger;</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    August, 2018
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Molecular Psychiatry
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/2018-08_maternalsmoking/" >
      <div class="img-hover-zoom">
        <img src="/publication/2018-08_maternalsmoking/featured_hue8dcb2fbcff4e0f70f3f9b884c84d020_179424_808x455_fill_q75_h2_lanczos_smart1_3.webp" height="455" width="808"
            class="article-banner" alt="Developmental effects of maternal smoking during pregnancy on the human frontal cortex transcriptome" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/2018-08_maternalsmoking/" >Developmental effects of maternal smoking during pregnancy on the human frontal cortex transcriptome</a>
  </div>

  
  <a href="/publication/2018-08_maternalsmoking/"  class="summary-link">
    <div class="article-style">
      <p>This paper is on smoking and its relation to gene expression at different life stages. These genes are also related to autism spectrum disorder.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.biorxiv.org/content/early/2017/12/20/236968" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.nature.com/articles/s41380-018-0223-1.pdf" target="_blank" rel="noopener">
  PDF
</a>



<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/2018-08_maternalsmoking/cite.bib">
  Cite
</a>


  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/LieberInstitute/Smoking_DLPFC_Devel" target="_blank" rel="noopener">
  Code
</a>




  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/brainseq/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1038/s41380-018-0223-1" target="_blank" rel="noopener">
  DOI
</a>


  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="https://t.co/0RsJ1kDbOF8" target="_blank" rel="noopener">
    Read for free</a>


  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2018">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/jaffe-ae/">Jaffe AE</a></span>, <span >
      <a href="/authors/straub-r/">Straub R</a></span>, <span >
      <a href="/authors/shin-jh/">Shin JH</a></span>, <span >
      <a href="/authors/tao-r/">Tao R</a></span>, <span >
      <a href="/authors/gao-y/">Gao Y</a></span>, <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >
      <a href="/authors/kam-thong-t/">Kam-Thong T</a></span>, <span >
      <a href="/authors/xi-hs/">Xi HS</a></span>, <span >
      <a href="/authors/quan-j/">Quan J</a></span>, <span >
      <a href="/authors/chen-q/">Chen Q</a></span>, <span >
      <a href="/authors/colantuoni-c/">Colantuoni C</a></span>, <span >
      <a href="/authors/ulrich-ws/">Ulrich WS</a></span>, <span >
      <a href="/authors/maher-bj/">Maher BJ</a></span>, <span >
      <a href="/authors/deep-soboslay-a/">Deep-Soboslay A</a></span>, <span >
      <a href="/authors/the-brainseq-consortium/">The BrainSeq Consortium</a></span>, <span >
      <a href="/authors/cross-aj/">Cross AJ</a></span>, <span >
      <a href="/authors/brandon-nj/">Brandon NJ</a></span>, <span >
      <a href="/authors/leek-jt/">Leek JT</a></span>, <span >
      <a href="/authors/hyde-tm/">Hyde TM</a></span>, <span >
      <a href="/authors/kleinman-je/">Kleinman JE</a></span>, <span >
      <a href="/authors/weinberger-dr/">Weinberger DR</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    July, 2018
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Nature Neuroscience
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/2018-07_bsp1/" >
      <div class="img-hover-zoom">
        <img src="/publication/2018-07_bsp1/featured_huff893b7c4781f4483265f7536c65ba15_202241_808x455_fill_q75_h2_lanczos_smart1_3.webp" height="455" width="808"
            class="article-banner" alt="Developmental and genetic regulation of the human cortex transcriptome illuminate schizophrenia pathogenesis" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/2018-07_bsp1/" >Developmental and genetic regulation of the human cortex transcriptome illuminate schizophrenia pathogenesis</a>
  </div>

  
  <a href="/publication/2018-07_bsp1/"  class="summary-link">
    <div class="article-style">
      <p>Genome-wide association studies have identified 108 schizophrenia risk loci, but biological mechanisms for individual loci are largely unknown. Using developmental, genetic and illness-based RNA sequencing expression analysis in human brain, we characterized the human brain transcriptome around these loci and found enrichment for developmentally regulated genes with novel examples of shifting isoform usage across pre- and postnatal life. We found widespread expression quantitative trait loci (eQTLs), including many with transcript specificity and previously unannotated sequence that were independently replicated. We leveraged this general eQTL database to show that 48.1% of risk variants for schizophrenia associate with nearby expression. We lastly found 237 genes significantly differentially expressed between patients and controls, which replicated in an independent dataset, implicated synaptic processes, and were strongly regulated in early development. These findings together offer genetics- and diagnosis-related targets for better modeling of schizophrenia risk. This resource is publicly available at <a href="http://eqtl.brainseq.org/phase1" target="_blank" rel="noopener">http://eqtl.brainseq.org/phase1</a>.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.biorxiv.org/content/10.1101/124321v2" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.nature.com/articles/s41593-018-0197-y.pdf" target="_blank" rel="noopener">
  PDF
</a>



<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/2018-07_bsp1/cite.bib">
  Cite
</a>





  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/brainseq/">
    Project
  </a>
  

  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/derfinder/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1038/s41593-018-0197-y" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-0 year-2018">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    June, 2018
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/poster2018sages/" >
      <div class="img-hover-zoom">
        <img src="/publication/poster2018sages/featured_hu113101afe2b5a6c3452c664971211c9a_1402911_808x455_fill_q75_h2_lanczos_smart1_3.webp" height="455" width="808"
            class="article-banner" alt="BrainSeq Phase II: schizophrenia-associated expression differences between the hippocampus and the dorsolateral prefrontal cortex" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/poster2018sages/" >BrainSeq Phase II: schizophrenia-associated expression differences between the hippocampus and the dorsolateral prefrontal cortex</a>
  </div>

  
  <a href="/publication/poster2018sages/"  class="summary-link">
    <div class="article-style">
      <p>Poster presented at SAGES2018</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="http://lcolladotor.github.io/Posters/SAGES2018.pdf" target="_blank" rel="noopener">
  PDF
</a>







  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/brainseq/">
    Project
  </a>
  











  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2018">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/shannon-se/">Shannon SE</a></span>, <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >
      <a href="/authors/jaffe-ae/">Jaffe AE</a></span>, <span >
      <a href="/authors/leek-jt/">Leek JT</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    March, 2018
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Nucleic Acids Research
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/2018-03_rnaseq_prediction/" >
      <div class="img-hover-zoom">
        <img src="/publication/2018-03_rnaseq_prediction/featured_hua4d9fb4833bf342dfdc873622df802de_235231_808x455_fill_q75_h2_lanczos_smart1.webp" height="455" width="808"
            class="article-banner" alt="Improving the value of public RNA-seq expression data by phenotype prediction" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/2018-03_rnaseq_prediction/" >Improving the value of public RNA-seq expression data by phenotype prediction</a>
  </div>

  
  <a href="/publication/2018-03_rnaseq_prediction/"  class="summary-link">
    <div class="article-style">
      <p>Background: Publicly available genomic data are a valuable resource for studying normal human variation and disease, but these data are often not well labeled or annotated. The lack of phenotype information for public genomic data severely limits their utility for addressing targeted biological questions. Results: We develop an in silico phenotyping approach for predicting critical missing annotation directly from genomic measurements using, well-annotated genomic and phenotypic data produced by consortia like TCGA and GTEx as training data. We apply in silico phenotyping to a set of 70,000 RNA-seq samples we recently processed on a common pipeline as part of the recount2 project (<a href="https://jhubiostatistics.shinyapps.io/recount/%29" target="_blank" rel="noopener">https://jhubiostatistics.shinyapps.io/recount/)</a>. We use gene expression data to build and evaluate predictors for both biological phenotypes (sex, tissue, sample source) and experimental conditions (sequencing strategy). We demonstrate how these predictions can be used to study cross-sample properties of public genomic data, select genomic projects with specific characteristics, and perform downstream analyses using predicted phenotypes. The methods to perform phenotype prediction are available in the phenopredict R package (<a href="https://github.com/leekgroup/phenopredict" target="_blank" rel="noopener">https://github.com/leekgroup/phenopredict</a>) and the predictions for recount2 are available from the recount R package (<a href="https://bioconductor.org/packages/release/bioc/html/recount.html%29" target="_blank" rel="noopener">https://bioconductor.org/packages/release/bioc/html/recount.html)</a>. Conclusion: Having leveraging massive public data sets to generate a well-phenotyped set of expression data for more than 70,000 human samples, expression data is available for use on a scale that was not previously feasible.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.biorxiv.org/content/10.1101/145656v1" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gky102/4920847" target="_blank" rel="noopener">
  PDF
</a>



<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/2018-03_rnaseq_prediction/cite.bib">
  Cite
</a>


  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/ShanEllis/phenopredict_usecase" target="_blank" rel="noopener">
  Code
</a>




  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/recount2/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1093/nar/gky102" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-3 year-2018">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/fu-j/">Fu J</a></span>, <span >
      <a href="/authors/kammers-k/">Kammers K</a></span>, <span >
      <a href="/authors/nellore-a/">Nellore A</a></span>, <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >
      <a href="/authors/leek-jt/">Leek JT</a></span>, <span >
      <a href="/authors/taub-ma/">Taub MA</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    January, 2018
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      bioRxiv 247346
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/preprint_recount_tx/" >
      <div class="img-hover-zoom">
        <img src="/publication/preprint_recount_tx/featured_hua06a972d1bb66609511008f642f307a0_136615_808x455_fill_q75_h2_lanczos_smart1_3.webp" height="455" width="808"
            class="article-banner" alt="RNA-seq transcript quantification from reduced-representation data in recount2" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/preprint_recount_tx/" >RNA-seq transcript quantification from reduced-representation data in recount2</a>
  </div>

  
  <a href="/publication/preprint_recount_tx/"  class="summary-link">
    <div class="article-style">
      <p>More than 70,000 short-read RNA-sequencing samples are publicly available through the recount2 project, a curated database of summary coverage data. However, no current methods can estimate transcript-level abundances using the reduced-representation information stored in this database. Here we present a linear model utilizing coverage of junctions and subdivided exons to generate transcript abundance estimates of comparable accuracy to those obtained from methods requiring read-level data. Our approach flexibly models bias, produces standard errors, and is easy to refresh given updated annotation. We illustrate our method on simulated and real data and release transcript abundance estimates for the samples in recount2.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.biorxiv.org/content/10.1101/247346v2" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.biorxiv.org/content/early/2018/01/12/247346.full.pdf" target="_blank" rel="noopener">
  PDF
</a>



<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/preprint_recount_tx/cite.bib">
  Cite
</a>





  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/recount2/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1101/247346" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-0 year-2017">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    October, 2017
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/poster2017idies/" >
      <div class="img-hover-zoom">
        <img src="/publication/poster2017idies/featured_hu87ecd3455c6fe7e504a939f3e07970e2_1416677_808x455_fill_q75_h2_lanczos_top_3.webp" height="455" width="808"
            class="article-banner" alt="Getting started with recount2 and accessing it via R" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/poster2017idies/" >Getting started with recount2 and accessing it via R</a>
  </div>

  
  <a href="/publication/poster2017idies/"  class="summary-link">
    <div class="article-style">
      <p>Poster presented at IDIES2017</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="http://lcolladotor.github.io/Posters/IDIES2017.pdf" target="_blank" rel="noopener">
  PDF
</a>







  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/recount2/">
    Project
  </a>
  











  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2017">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/__l-collado-torres__/authors/admin-dagger/">[__L Collado-Torres__](/authors/admin) &amp;dagger;</a></span>, <span >
      <a href="/authors/a-nellore/">A Nellore</a></span>, <span >
      <a href="/authors/ae-jaffe/">AE Jaffe</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    August, 2017
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      F1000Research
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/2017-08_recountworkflow/" >
      <div class="img-hover-zoom">
        <img src="/publication/2017-08_recountworkflow/featured_hu3895338370ae321089968d378f88b52e_7454_808x455_fill_q75_h2_lanczos_smart1_3.webp" height="455" width="808"
            class="article-banner" alt="recount workflow: Accessing over 70,000 human RNA-seq samples with Bioconductor [version 1; referees: 1 approved, 2 approved with reservations]" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/2017-08_recountworkflow/" >recount workflow: Accessing over 70,000 human RNA-seq samples with Bioconductor [version 1; referees: 1 approved, 2 approved with reservations]</a>
  </div>

  
  <a href="/publication/2017-08_recountworkflow/"  class="summary-link">
    <div class="article-style">
      <p>The recount2 resource is composed of over 70,000 uniformly processed human RNA-seq samples spanning TCGA and SRA, including GTEx. The processed data can be accessed via the recount2 website and the recount Bioconductor package. This workflow explains in detail how to use the recount package and how to integrate it with other Bioconductor packages for several analyses that can be carried out with the recount2 resource. In particular, we describe how the coverage count matrices were computed in recount2 as well as different ways of obtaining public metadata, which can facilitate downstream analyses. Step-by-step directions show how to do a gene-level differential expression analysis, visualize base-level genome coverage data, and perform an analyses at multiple feature levels. This workflow thus provides further information to understand the data in recount2 and a compendium of R code to use the data.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://f1000research.com/articles/6-1558/v1" target="_blank" rel="noopener">
  PDF
</a>



<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/2017-08_recountworkflow/cite.bib">
  Cite
</a>


  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/LieberInstitute/recountWorkflow" target="_blank" rel="noopener">
  Code
</a>




  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/recount2/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.12688/f1000research.12223.1" target="_blank" rel="noopener">
  DOI
</a>


  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="http://bioconductor.org/packages/release/workflows/html/recountWorkflow.html" target="_blank" rel="noopener">
    Bioconductor</a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="http://LieberInstitute.github.io/recountWorkshop/" target="_blank" rel="noopener">
    BioC2017 Workshop</a>


  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2017">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/c-wright/">C Wright</a></span>, <span >
      <a href="/authors/jh-shin/">JH Shin</a></span>, <span >
      <a href="/authors/a-rajpurohit/">A Rajpurohit</a></span>, <span >
      <a href="/authors/a-deep-soboslay/">A Deep-Soboslay</a></span>, <span >
      <a href="/authors/__l-collado-torres__/authors/admin/">[__L Collado-Torres__](/authors/admin)</a></span>, <span >
      <a href="/authors/nj-brandon/">NJ Brandon</a></span>, <span >
      <a href="/authors/tm-hyde/">TM Hyde</a></span>, <span >
      <a href="/authors/je-kleinman/">JE Kleinman</a></span>, <span >
      <a href="/authors/ae-jaffe/">AE Jaffe</a></span>, <span >
      <a href="/authors/aj-cross/">AJ Cross</a></span>, <span >
      <a href="/authors/dr-weinberger/">DR Weinberger</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    May, 2017
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Translational Psychiatry
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/2017-05_histamine/" >
      <div class="img-hover-zoom">
        <img src="/publication/2017-05_histamine/featured_hu6d32d65792e3bc285bcd06e6b134828f_172962_808x455_fill_q75_h2_lanczos_smart1_3.webp" height="455" width="808"
            class="article-banner" alt="Altered expression of histamine signaling genes in autism spectrum disorder" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/2017-05_histamine/" >Altered expression of histamine signaling genes in autism spectrum disorder</a>
  </div>

  
  <a href="/publication/2017-05_histamine/"  class="summary-link">
    <div class="article-style">
      <p>Expression of the gene set of HNMT, HRH1, HRH2 and HRH3 was significantly altered between ASD and matched controls, and this finding was replicated with an independent data set.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.nature.com/tp/journal/v7/n5/full/tp201787a.html" target="_blank" rel="noopener">
  PDF
</a>



<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/2017-05_histamine/cite.bib">
  Cite
</a>


  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/LieberInstitute/Wright_2017_ASD_Histamine" target="_blank" rel="noopener">
  Code
</a>




  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1038/tp.2017.87" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2017">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/__l-collado-torres__/authors/admin-____/">[__L Collado-Torres__](/authors/admin) __*__</a></span>, <span >
      <a href="/authors/a-nellore-____/">A Nellore __*__</a></span>, <span >
      <a href="/authors/k-kammers/">K Kammers</a></span>, <span >
      <a href="/authors/se-ellis/">SE Ellis</a></span>, <span >
      <a href="/authors/ma-taub/">MA Taub</a></span>, <span >
      <a href="/authors/kd-hansen/">KD Hansen</a></span>, <span >
      <a href="/authors/ae-jaffe/">AE Jaffe</a></span>, <span >
      <a href="/authors/b-langmead/">B Langmead</a></span>, <span >
      <a href="/authors/jt-leek/">JT Leek</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    April, 2017
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Nature Biotechnology
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/2017-04_recount/" >
      <div class="img-hover-zoom">
        <img src="/publication/2017-04_recount/featured_hu38d48fdc126e5a16b49d21329437221e_293563_808x455_fill_q75_h2_lanczos_bottom_3.webp" height="455" width="808"
            class="article-banner" alt="Reproducible RNA-seq analysis using recount2" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/2017-04_recount/" >Reproducible RNA-seq analysis using recount2</a>
  </div>

  
  <a href="/publication/2017-04_recount/"  class="summary-link">
    <div class="article-style">
      <p>recount2 is a resource of processed and summarized expression data spanning over 70,000 human RNA-seq samples from the Sequence Read Archive (SRA). The associated recount Bioconductor package provides a convenient API for querying, downloading, and analyzing the data. Each processed study consists of meta/phenotype data, the expression levels of genes and their underlying exons and splice junctions, and corresponding genomic annotation. We also provide data summarization types for quantifying novel transcribed sequence including base-resolution coverage and potentially unannotated splice junctions. We present workflows illustrating how to use recount to perform differential expression analysis including meta-analysis, annotation-free base-level analysis, and replication of smaller studies using data from larger studies. recount provides a valuable and user-friendly resource of processed RNA-seq datasets to draw additional biological insights from existing public data. The resource is available at <a href="https://jhubiostatistics.shinyapps.io/recount/" target="_blank" rel="noopener">https://jhubiostatistics.shinyapps.io/recount/</a>.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.biorxiv.org/content/10.1101/068478v1" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="http://www.nature.com/nbt/journal/v35/n4/full/nbt.3838.html" target="_blank" rel="noopener">
  PDF
</a>



<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/2017-04_recount/cite.bib">
  Cite
</a>


  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/leekgroup/recount-website" target="_blank" rel="noopener">
  Code
</a>




  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/recount2/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1038/nbt.3838" target="_blank" rel="noopener">
  DOI
</a>


  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="http://bioconductor.org/packages/recount" target="_blank" rel="noopener">
    Bioconductor</a>


  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2016">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/a-nellore/">A Nellore</a></span>, <span >
      <a href="/authors/ae-jaffe/">AE Jaffe</a></span>, <span >
      <a href="/authors/jp-fortin/">JP Fortin</a></span>, <span >
      <a href="/authors/j-alquicira-hernandez/"> J Alquicira-Hernández</a></span>, <span >
      <a href="/authors/__l-collado-torres__/authors/admin/">[__L Collado-Torres__](/authors/admin)</a></span>, <span >
      <a href="/authors/s-wang/">S Wang</a></span>, <span >
      <a href="/authors/ra-phillips/">RA Phillips</a></span>, <span >
      <a href="/authors/n-karbhari/">N Karbhari</a></span>, <span >
      <a href="/authors/kd-hansen/">KD Hansen</a></span>, <span >
      <a href="/authors/b-langmead/">B Langmead</a></span>, <span >
      <a href="/authors/jt-leek/">JT Leek</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    December, 2016
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Genome Biology
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/2016-12_intropolis/" >
      <div class="img-hover-zoom">
        <img src="/publication/2016-12_intropolis/featured_hu86bb6472eb05fa8968374278fa4ec260_236812_808x455_fill_q75_h2_lanczos_smart1_3.webp" height="455" width="808"
            class="article-banner" alt="Human splicing diversity and the extent of unannotated splice junctions across human RNA-seq samples on the Sequence Read Archive" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/2016-12_intropolis/" >Human splicing diversity and the extent of unannotated splice junctions across human RNA-seq samples on the Sequence Read Archive</a>
  </div>

  
  <a href="/publication/2016-12_intropolis/"  class="summary-link">
    <div class="article-style">
      <p>We found 56,861 junctions (18.6%) in at least 1000 samples that were not annotated out of 21,504 samples, and their expression associated with tissue type. We compiled junction data into a resource called intropolis available at <a href="http://intropolis.rail.bio" target="_blank" rel="noopener">http://intropolis.rail.bio</a>.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.biorxiv.org/content/10.1101/038224v1" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="http://genomebiology.biomedcentral.com/track/pdf/10.1186/s13059-016-1118-6?site=genomebiology.biomedcentral.com" target="_blank" rel="noopener">
  PDF
</a>



<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/2016-12_intropolis/cite.bib">
  Cite
</a>


  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://figshare.com/articles/Scripts_outputs_for_paper_Human_splicing_diversity_across_the_Sequence_Read_Archive_by_Nellore_et_al_/3811629/1" target="_blank" rel="noopener">
  Code
</a>


  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://figshare.com/articles/intropolis_exon-exon_junctions_across_publicly_available_RNA-seq_samples_on_the_Sequence_Read_Archive/3811680/1" target="_blank" rel="noopener">
  Dataset
</a>



  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/recount2/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1186/s13059-016-1118-6" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2016">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/__l-collado-torres__/authors/admin/">[__L Collado-Torres__](/authors/admin)</a></span>, <span >
      <a href="/authors/a-nellore/">A Nellore</a></span>, <span >
      <a href="/authors/ac-frazee/">AC Frazee</a></span>, <span >
      <a href="/authors/c-wilks/">C Wilks</a></span>, <span >
      <a href="/authors/mi-love/">MI Love</a></span>, <span >
      <a href="/authors/b-langmead/">B Langmead</a></span>, <span >
      <a href="/authors/ra-irizarry/">RA Irizarry</a></span>, <span >
      <a href="/authors/jt-leek/">JT Leek</a></span>, <span >
      <a href="/authors/ae-jaffe/">AE Jaffe</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    September, 2016
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      NAR
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/2016-09_derfinder/" >
      <div class="img-hover-zoom">
        <img src="/publication/2016-09_derfinder/featured_hu80015c1f13253d2568ff2a092af02f6b_231747_808x455_fill_q75_h2_lanczos_smart1_3.webp" height="455" width="808"
            class="article-banner" alt="Flexible expressed region analysis for RNA-seq with derfinder" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/2016-09_derfinder/" >Flexible expressed region analysis for RNA-seq with derfinder</a>
  </div>

  
  <a href="/publication/2016-09_derfinder/"  class="summary-link">
    <div class="article-style">
      <p>derfinder analysis using expressed region-level and single base-level approaches provides a compromise between full transcript reconstruction and feature-level analysis. The package is available from Bioconductor.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="http://biorxiv.org/content/early/2016/05/19/015370" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="http://nar.oxfordjournals.org/content/early/2016/09/29/nar.gkw852.full.pdf&#43;html" target="_blank" rel="noopener">
  PDF
</a>



<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/2016-09_derfinder/cite.bib">
  Cite
</a>


  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/leekgroup/derSupplement" target="_blank" rel="noopener">
  Code
</a>




  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/derfinder/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1093/nar/gkw852" target="_blank" rel="noopener">
  DOI
</a>


  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="http://bioconductor.org/packages/derfinder" target="_blank" rel="noopener">
    Bioconductor</a>


  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2016">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/a-nellore/">A Nellore</a></span>, <span >
      <a href="/authors/__l-collado-torres__/authors/admin/">[__L Collado-Torres__](/authors/admin)</a></span>, <span >
      <a href="/authors/ae-jaffe/">AE Jaffe</a></span>, <span >
      <a href="/authors/j-alquicira-hernandez/"> J Alquicira-Hernández</a></span>, <span >
      <a href="/authors/c-wilks/">C Wilks</a></span>, <span >
      <a href="/authors/j-pritt/">J Pritt</a></span>, <span >
      <a href="/authors/j-morton/">J Morton</a></span>, <span >
      <a href="/authors/jt-leek/">JT Leek</a></span>, <span >
      <a href="/authors/b-langmead/">B Langmead</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    September, 2016
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Bioinformatics
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/2016-09_rail/" >
      <div class="img-hover-zoom">
        <img src="/publication/2016-09_rail/featured_hub94be178e119e17f0204dfbeac3b7990_213872_808x455_fill_q75_h2_lanczos_smart1_3.webp" height="455" width="808"
            class="article-banner" alt="Rail-RNA: Scalable analysis of RNA-seq splicing and coverage" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/2016-09_rail/" >Rail-RNA: Scalable analysis of RNA-seq splicing and coverage</a>
  </div>

  
  <a href="/publication/2016-09_rail/"  class="summary-link">
    <div class="article-style">
      <p>We describe Rail-RNA, a cloud-enabled spliced aligner that analyzes many samples at once. Rail-RNA eliminates redundant work across samples, making it more efficient as samples are added. For many samples, Rail-RNA is more accurate than annotation-assisted aligners.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.biorxiv.org/content/10.1101/019067v2" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="http://bioinformatics.oxfordjournals.org/content/early/2016/09/02/bioinformatics.btw575.full.pdf&#43;html" target="_blank" rel="noopener">
  PDF
</a>



<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/2016-09_rail/cite.bib">
  Cite
</a>





  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/derfinder/">
    Project
  </a>
  

  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/recount2/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1093/bioinformatics/btw575" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2016">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/__l-collado-torres__/authors/admin/">[__L Collado-Torres__](/authors/admin)</a></span>, <span >
      <a href="/authors/ae-jaffe/">AE Jaffe</a></span>, <span >
      <a href="/authors/jt-leek/">JT Leek</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    June, 2016
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      F1000Research 2016, 4:105
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/2016-06_regionreport/" >
      <div class="img-hover-zoom">
        <img src="/publication/2016-06_regionreport/featured_hu2bab50de816c0b9ea4d83a70e2362e56_299655_808x455_fill_q75_h2_lanczos_smart1_3.webp" height="455" width="808"
            class="article-banner" alt="regionReport: Interactive reports for region-level and feature-level genomic analyses" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/2016-06_regionreport/" >regionReport: Interactive reports for region-level and feature-level genomic analyses</a>
  </div>

  
  <a href="/publication/2016-06_regionreport/"  class="summary-link">
    <div class="article-style">
      <p>regionReport is an R package for generating detailed interactive reports from region-level genomic analyses as well as feature-level RNA-seq. The report includes quality-control checks, an overview of the results, an interactive table of the genomic regions or features of interest and reproducibility information. regionReport provides specialised reports for exploring DESeq2, edgeR, or derfinder differential expression analyses results. regionReport is also flexible and can easily be expanded with report templates for other analysis pipelines.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://www.biorxiv.org/content/10.1101/016659v1" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://f1000research.com/articles/4-105/v2/pdf" target="_blank" rel="noopener">
  PDF
</a>



<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/2016-06_regionreport/cite.bib">
  Cite
</a>


  
  
    
  
<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://github.com/leekgroup/regionReportSupp" target="_blank" rel="noopener">
  Code
</a>




  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/derfinder/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.12688/f1000research.6379.2" target="_blank" rel="noopener">
  DOI
</a>


  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="http://bioconductor.org/packages/regionReport" target="_blank" rel="noopener">
    Bioconductor</a>


  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-0 year-2015">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    October, 2015
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/poster2015/" >
      <div class="img-hover-zoom">
        <img src="/publication/poster2015/featured_hue6a40410f569f5345fa305581f98a591_993972_808x455_fill_q75_h2_lanczos_smart1_3.webp" height="455" width="808"
            class="article-banner" alt="Annotation-agnostic RNA-seq differential expression analysis software" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/poster2015/" >Annotation-agnostic RNA-seq differential expression analysis software</a>
  </div>

  
  <a href="/publication/poster2015/"  class="summary-link">
    <div class="article-style">
      <p>Poster presented at ASHG2015 and IDIES2015</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="http://lcolladotor.github.io/Posters/ASHG2015.pdf" target="_blank" rel="noopener">
  PDF
</a>







  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/derfinder/">
    Project
  </a>
  











  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2014">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/jaffe-ae/">Jaffe AE</a></span>, <span >
      <a href="/authors/shin-j/">Shin J</a></span>, <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >
      <a href="/authors/leek-jt/">Leek JT</a></span>, <span >
      <a href="/authors/tao-r/">Tao R</a></span>, <span >
      <a href="/authors/li-c/">Li C</a></span>, <span >
      <a href="/authors/gao-y/">Gao Y</a></span>, <span >
      <a href="/authors/jia-y/">Jia Y</a></span>, <span >
      <a href="/authors/maher-bj/">Maher BJ</a></span>, <span >
      <a href="/authors/hyde-tm/">Hyde TM</a></span>, <span >
      <a href="/authors/kleinman-je/">Kleinman JE</a></span>, <span >
      <a href="/authors/weinberger-dr/">Weinberger DR</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    December, 2014
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Nature Neuroscience
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/2014-12_brainders/" >
      <div class="img-hover-zoom">
        <img src="/publication/2014-12_brainders/featured_hu55d5bd73db4245a0b08a0597e0f0bedc_376739_808x455_fill_q75_h2_lanczos_smart1_3.webp" height="455" width="808"
            class="article-banner" alt="Developmental regulation of human cortex transcription and its clinical relevance at single base resolution" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/2014-12_brainders/" >Developmental regulation of human cortex transcription and its clinical relevance at single base resolution</a>
  </div>

  
  <a href="/publication/2014-12_brainders/"  class="summary-link">
    <div class="article-style">
      <p>Transcriptome analysis of human brain provides fundamental insight into development and disease, but it largely relies on existing annotation. We sequenced transcriptomes of 72 prefrontal cortex samples across six life stages and identified 50,650 differentially expression regions (DERs) associated with developmental and aging, agnostic of annotation. While many DERs annotated to non-exonic sequence (41.1%), most were similarly regulated in cytosolic mRNA extracted from independent samples. The DERs were developmentally conserved across 16 brain regions and in the developing mouse cortex, and were expressed in diverse cell and tissue types. The DERs were further enriched for active chromatin marks and clinical risk for neurodevelopmental disorders such as schizophrenia. Lastly, we demonstrate quantitatively that these DERs associate with a changing neuronal phenotype related to differentiation and maturation. These data show conserved molecular signatures of transcriptional dynamics across brain development, have potential clinical relevance and highlight the incomplete annotation of the human brain transcriptome.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="http://www.nature.com/neuro/journal/v18/n1/full/nn.3898.html" target="_blank" rel="noopener">
  PDF
</a>



<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/2014-12_brainders/cite.bib">
  Cite
</a>





  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/derfinder/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1038/nn.3898" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-6 year-2014">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/ac-frazee/">AC Frazee</a></span>, <span >
      <a href="/authors/l-collado-torres/">L Collado-Torres</a></span>, <span >
      <a href="/authors/ae-jaffe/">AE Jaffe</a></span>, <span >
      <a href="/authors/b-langmead/">B Langmead</a></span>, <span >
      <a href="/authors/jt-leek/">JT Leek</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    June, 2014
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Springer
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/2014-06_rnaseqbook/" >
      <div class="img-hover-zoom">
        <img src="/publication/2014-06_rnaseqbook/featured_hueaa2dadef3cbf156b33855f895f55cec_80877_808x455_fill_q75_h2_lanczos_smart1_3.webp" height="455" width="808"
            class="article-banner" alt="Measurement, Summary, and Methodological Variation in RNA-sequencing in Statistical Analysis of Next Generation Sequencing Data" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/2014-06_rnaseqbook/" >Measurement, Summary, and Methodological Variation in RNA-sequencing in Statistical Analysis of Next Generation Sequencing Data</a>
  </div>

  
  <a href="/publication/2014-06_rnaseqbook/"  class="summary-link">
    <div class="article-style">
      <p>There has been a major shift from microarrays to RNA-sequencing (RNA-seq) for measuring gene expression as the price per measurement between these technologies has become comparable. The advantages of RNA-seq are increased measurement flexibility to detect alternative transcription, allele specific transcription, or transcription outside of known coding regions. The price of this increased flexibility is: (a) an increase in raw data size and (b) more decisions that must be made by the data analyst. Here we provide a selective review and extension of our previous work in attempting to measure variability in results due to different choices about how to summarize and analyze RNA-sequencing data. We discuss a standard model for gene expression measurements that breaks variability down into variation due to technology, biology, and measurement error. Finally, wee show the importance of gene model selection, normalization, and choice for statistical model on the ultimate results of an RNA-sequencing experiment.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="http://link.springer.com/chapter/10.1007/978-3-319-07212-8_6" target="_blank" rel="noopener">
  PDF
</a>



<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/2014-06_rnaseqbook/cite.bib">
  Cite
</a>





  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/derfinder/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1007/978-3-319-07212-8_6" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-0 year-2014">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    March, 2014
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/poster2014enar/" >
      <div class="img-hover-zoom">
        <img src="/publication/poster2014enar/featured_hu00faf59d0f9890e844071e418f06e48c_1174197_808x455_fill_q75_h2_lanczos_smart1_3.webp" height="455" width="808"
            class="article-banner" alt="Fast Annotation-Agnostic Differential Expression Analysis" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/poster2014enar/" >Fast Annotation-Agnostic Differential Expression Analysis</a>
  </div>

  
  <a href="/publication/poster2014enar/"  class="summary-link">
    <div class="article-style">
      <p>Poster presented at ENAR2014</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="http://lcolladotor.github.io/Posters/ENAR2014.pdf" target="_blank" rel="noopener">
  PDF
</a>







  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/derfinder/">
    Project
  </a>
  











  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-0 year-2013">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    October, 2013
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/poster2013gbs/" >
      <div class="img-hover-zoom">
        <img src="/publication/poster2013gbs/featured_hu2a58cdc5db42ffc8f03e7853a2b9f97b_1018007_808x455_fill_q75_h2_lanczos_smart1_3.webp" height="455" width="808"
            class="article-banner" alt="Fast Annotation-Agnostic Differential Expression Analysis" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/poster2013gbs/" >Fast Annotation-Agnostic Differential Expression Analysis</a>
  </div>

  
  <a href="/publication/poster2013gbs/"  class="summary-link">
    <div class="article-style">
      <p>Poster presented at Genomics and Bioinformatics Symposium 2013, Delta Omega Poster Competition</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="http://lcolladotor.github.io/Posters/GBS2013.pdf" target="_blank" rel="noopener">
  PDF
</a>







  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/derfinder/">
    Project
  </a>
  











  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-0 year-2013">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    April, 2013
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/poster2013retreat/" >
      <div class="img-hover-zoom">
        <img src="/publication/poster2013retreat/featured_huc50708a1d3886bade62f59f9e11ba8ea_1273134_808x455_fill_q75_h2_lanczos_smart1_3.webp" height="455" width="808"
            class="article-banner" alt="Differential expression RNA-seq analysis with a large data set from brain samples" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/poster2013retreat/" >Differential expression RNA-seq analysis with a large data set from brain samples</a>
  </div>

  
  <a href="/publication/poster2013retreat/"  class="summary-link">
    <div class="article-style">
      <p>Poster presented at JHU Biostatistics Department Retreat</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="http://lcolladotor.github.io/Posters/retreat2013.pdf" target="_blank" rel="noopener">
  PDF
</a>







  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/derfinder/">
    Project
  </a>
  











  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2011">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/shank-ea/">Shank EA</a></span>, <span >
      <a href="/authors/klepac-ceraj-v/">Klepac-Ceraj V</a></span>, <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >
      <a href="/authors/powers-ge/">Powers GE</a></span>, <span >
      <a href="/authors/losick-r/">Losick R</a></span>, <span >
      <a href="/authors/kolter-r/">Kolter R</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    November, 2011
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      PNAS
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/2011-11_bsubtilis/" >
      <div class="img-hover-zoom">
        <img src="/publication/2011-11_bsubtilis/featured_hu2c325411603e74e59221b38ee16d0a7c_125553_808x455_fill_q75_h2_lanczos_center_3.webp" height="455" width="808"
            class="article-banner" alt="Interspecies interactions that result in Bacillus subtilis forming biofilms are mediated mainly by members of its own genus" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/2011-11_bsubtilis/" >Interspecies interactions that result in Bacillus subtilis forming biofilms are mediated mainly by members of its own genus</a>
  </div>

  
  <a href="/publication/2011-11_bsubtilis/"  class="summary-link">
    <div class="article-style">
      <p>Many different systems of bacterial interactions have been described. However, relatively few studies have explored how interactions between different microorganisms might influence bacterial development. To explore such interspecies interactions, we focused on Bacillus subtilis, which characteristically develops into matrix-producing cannibals before entering sporulation. We investigated whether organisms from the natural environment of B. subtilis—the soil—were able to alter the development of B.subtilis. To test this possibility, we developed a coculture microcolony screen in which we used fluorescent reporters to identify soil bacteria able to induce matrix production in B. subtilis. Most of the bacteria that influence matrix production in B. subtilis are members of the genus Bacillus, suggesting that such interactions may be predominantly with close relatives. The interactions we observed were mediated via two different mechanisms. One resulted in increased expression of matrix genes via the activation of a sensor histidine kinase, KinD. The second was kinase independent and conceivably functions by altering the relative subpopulations of B. subtilis cell types by preferentially killing noncannibals. These two mechanisms were grouped according to the inducing strain’s relatedness to B. subtilis. Our results suggest that bacteria preferentially alter their development in response to secreted molecules from closely related bacteria and do so using mechanisms that depend on the phylogenetic relatedness of the interacting bacteria.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="http://www.pnas.org/content/108/48/E1236.full.pdf&#43;html" target="_blank" rel="noopener">
  PDF
</a>



<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/2011-11_bsubtilis/cite.bib">
  Cite
</a>





  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1073/pnas.1103630108" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2011">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/gama-castro-s/">Gama-Castro S</a></span>, <span >
      <a href="/authors/salgado-h/">Salgado H</a></span>, <span >
      <a href="/authors/peralta-gil-m/">Peralta-Gil M</a></span>, <span >
      <a href="/authors/santos-zavaleta-a/">Santos-Zavaleta A</a></span>, <span >
      <a href="/authors/muniz-rascado-l/">Muñiz-Rascado L</a></span>, <span >
      <a href="/authors/solano-lira-h/">Solano-Lira H</a></span>, <span >
      <a href="/authors/jimenez-jacinto-v/">Jimenez-Jacinto V</a></span>, <span >
      <a href="/authors/weiss-v/">Weiss V</a></span>, <span >
      <a href="/authors/garcia-sotelo-js/">García-Sotelo JS</a></span>, <span >
      <a href="/authors/lopez-fuentes-a/">López-Fuentes A</a></span>, <span >
      <a href="/authors/porron-sotelo-l/">Porrón-Sotelo L</a></span>, <span >
      <a href="/authors/alquicira-hernandez-s/">Alquicira-Hernández S</a></span>, <span >
      <a href="/authors/medina-rivera-a/">Medina-Rivera A</a></span>, <span >
      <a href="/authors/martinez-flores-i/">Martínez-Flores I</a></span>, <span >
      <a href="/authors/alquicira-hernandez-k/">Alquicira-Hernández K</a></span>, <span >
      <a href="/authors/martinez-adame-r/">Martínez-Adame R</a></span>, <span >
      <a href="/authors/bonavides-martinez-c/">Bonavides-Martínez C</a></span>, <span >
      <a href="/authors/miranda-rios-j/">Miranda-Ríos J</a></span>, <span >
      <a href="/authors/huerta-am/">Huerta AM</a></span>, <span >
      <a href="/authors/mendoza-vargas-a/">Mendoza-Vargas A</a></span>, <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >
      <a href="/authors/taboada-b/">Taboada B</a></span>, <span >
      <a href="/authors/vega-alvarado-l/">Vega-Alvarado L</a></span>, <span >
      <a href="/authors/olvera-m/">Olvera M</a></span>, <span >
      <a href="/authors/olvera-l/">Olvera L</a></span>, <span >
      <a href="/authors/grande-r/">Grande R</a></span>, <span >
      <a href="/authors/morett-e/">Morett E</a></span>, <span >
      <a href="/authors/collado-vides-j/">Collado-Vides J</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    January, 2011
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      NAR
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/2011-01_regulondbv7/" >
      <div class="img-hover-zoom">
        <img src="/publication/2011-01_regulondbv7/featured_hu1353773f91cd92b58099ed7d1d0a81b4_200773_808x455_fill_q75_h2_lanczos_smart1_3.webp" height="455" width="808"
            class="article-banner" alt="RegulonDB version 7.0: transcriptional regulation of Escherichia coli K-12 integrated within genetic sensory response units (Gensor Units)" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/2011-01_regulondbv7/" >RegulonDB version 7.0: transcriptional regulation of Escherichia coli K-12 integrated within genetic sensory response units (Gensor Units)</a>
  </div>

  
  <a href="/publication/2011-01_regulondbv7/"  class="summary-link">
    <div class="article-style">
      <p>RegulonDB (<a href="http://regulondb.ccg.unam.mx/" target="_blank" rel="noopener">http://regulondb.ccg.unam.mx/</a>) is the primary reference database of the best-known regulatory network of any free-living organism, that of Escherichia coli K-12. The major conceptual change since 3 years ago is an expanded biological context so that transcriptional regulation is now part of a unit that initiates with the signal and continues with the signal transduction to the core of regulation, modifying expression of the affected target genes responsible for the response. We call these genetic sensory response units, or Gensor Units. We have initiated their high-level curation, with graphic maps and superreactions with links to other databases. Additional connectivity uses expandable submaps. RegulonDB has summaries for every transcription factor (TF) and TF-binding sites with internal symmetry. Several DNA-binding motifs and their sizes have been redefined and relocated. In addition to data from the literature, we have incorporated our own information on transcription start sites (TSSs) and transcriptional units (TUs), obtained by using high-throughput whole-genome sequencing technologies. A new portable drawing tool for genomic features is also now available, as well as new ways to download the data, including web services, files for several relational database manager systems and text files including BioPAX format.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="http://nar.oxfordjournals.org/content/39/suppl_1/D98.full.pdf&#43;html" target="_blank" rel="noopener">
  PDF
</a>



<a href="#" class="btn btn-outline-primary btn-page-header btn-sm js-cite-modal"
        data-filename="/publication/2011-01_regulondbv7/cite.bib">
  Cite
</a>





  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/">
    Project
  </a>
  









<a class="btn btn-outline-primary btn-page-header btn-sm" href="https://doi.org/10.1093/nar/gkq1110" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-0 year-2010">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    October, 2010
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/poster2010/" >
      <div class="img-hover-zoom">
        <img src="/publication/poster2010/featured_hu3512b051a4aa8904e759044e180c5020_645445_808x455_fill_q75_h2_lanczos_smart1_3.webp" height="455" width="808"
            class="article-banner" alt="Global Analysis of Transcription Start Sites and Transcription Units in Bacterial Genomes" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/poster2010/" >Global Analysis of Transcription Start Sites and Transcription Units in Bacterial Genomes</a>
  </div>

  
  <a href="/publication/poster2010/"  class="summary-link">
    <div class="article-style">
      <p>Poster presented at EMBO Conference Series from Functional Genomics to Systems Biology 2010</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="http://lcolladotor.github.io/Posters/EMBL2010poster.pdf" target="_blank" rel="noopener">
  PDF
</a>







  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/">
    Project
  </a>
  











  </div>
  

</div>

        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-0 year-2010">
          











  


<div class="card-simple view-card">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      <a href="/authors/admin/">Leonardo Collado-Torres</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    July, 2010
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
  </span>
  

  

  
  
  
  

  
  

</div>

  

  
  
  
    
    <a href="/publication/poster2010bioc/" >
      <div class="img-hover-zoom">
        <img src="/publication/poster2010bioc/featured_hu9839a251d184bb07f5bb7a746a32907b_1438410_808x455_fill_q75_h2_lanczos_smart1_3.webp" height="455" width="808"
            class="article-banner" alt="Global Analysis of Transcription Start Sites and Transcription Units in Bacterial Genomes" loading="lazy">
      </div>
    </a>
  

  <div class="section-subheading article-title mb-1 mt-3">
    <a href="/publication/poster2010bioc/" >Global Analysis of Transcription Start Sites and Transcription Units in Bacterial Genomes</a>
  </div>

  
  <a href="/publication/poster2010bioc/"  class="summary-link">
    <div class="article-style">
      <p>Poster presented at BioC2010</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary btn-page-header btn-sm" href="http://lcolladotor.github.io/Posters/BioC2010-v4.pdf" target="_blank" rel="noopener">
  PDF
</a>







  
  <a class="btn btn-outline-primary btn-page-header btn-sm" href="/project/">
    Project
  </a>
  











  </div>
  

</div>

        </div>

        
      </div>

    </div>
  </div>
</div>
  </div>

  <div class="page-footer">
    
    
    <div class="container">
      <footer class="site-footer">

  












  
  
  
  
  













  
  
  

  
  
    
  
  
    
  

  

  
  <p class="powered-by copyright-license-text">
    All thoughts and opinions here are my own (Leonardo Collado Torres). The icon was designed by <a href="https://www.linkedin.com/in/mauricio-guzman-6529b551/" target="_blank" rel="noopener">Mauricio Guzmán</a> and is inspired by <a href="https://en.wikipedia.org/wiki/Huichol" target="_blank" rel="noopener">Huichol culture</a>; it represents my community building interests. © 2011-2024 LCT. This work is licensed under <a href="https://creativecommons.org/licenses/by-nc-sa/4.0" rel="noopener noreferrer" target="_blank">CC BY NC SA 4.0</a>
  </p>
  

  <p class="powered-by footer-license-icons">
    <a href="https://creativecommons.org/licenses/by-nc-sa/4.0" rel="noopener noreferrer" target="_blank" aria-label="Creative Commons">
      <i class="fab fa-creative-commons fa-2x" aria-hidden="true"></i>
      <i class="fab fa-creative-commons-by fa-2x" aria-hidden="true"></i>
      
        <i class="fab fa-creative-commons-nc fa-2x" aria-hidden="true"></i>
      
      
        <i class="fab fa-creative-commons-sa fa-2x" aria-hidden="true"></i>
      
    </a>
  </p>




  
  <p>
      
      
      <script type="text/javascript" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=ra-4ef23e545d41b87c"></script>
      
      

      <script type="text/javascript">
      var addthis_config = {
      	data_ga_property: 'UA-45299226-1',
      	data_ga_social : true,
      	data_track_addressbar: true,
        data_track_addressbar_paths: ['/201*/*/*/*/*']
      };
      var addthis_share = {
      	
      	url_transforms : {
      		shorten: {
      			twitter: 'bitly'
      		}
      	}, 
      	shorteners : {
      		bitly : {}
      	}
      }
      </script>
      
      <center>	 
      <div id="clustrmaps-widget"></div><script type='text/javascript' id='clustrmaps' src='//cdn.clustrmaps.com/map_v2.js?cl=ffffff&w=300&t=m&d=Zo-deincaJGsqhqjIWsujtdoNOMJsnnZLefKmxDKVv8&co=124b73&cmo=dd53ff&cmn=4ec724'></script>
      </center>
  </p>
  

  <p class="powered-by">
    
    
    
      
      
      
      
      
      
      Published with <a href="https://wowchemy.com/?utm_campaign=poweredby" target="_blank" rel="noopener">Wowchemy</a> — the free, <a href="https://github.com/wowchemy/wowchemy-hugo-themes" target="_blank" rel="noopener">open source</a> website builder that empowers creators.
    
  </p>
</footer>

    </div>
    
  </div>

  


<script src="/js/vendor-bundle.min.938a3a7554cd9f6602290411f64d2617.js"></script>




  
    <script src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script>
    <script src="https://cdn.jsdelivr.net/gh/metafizzy/isotope@v3.0.6/dist/isotope.pkgd.min.js" integrity="sha512-Zq2BOxyhvnRFXu0+WE6ojpZLOU2jdnqbrM1hmVdGzyeCa1DgM3X5Q4A/Is9xA1IkbUeDd7755dNNI/PzSf2Pew==" crossorigin="anonymous"></script>
  

  
  

  






  <script src="https://cdn.jsdelivr.net/npm/leaflet@1.7.1/dist/leaflet.min.js" integrity="" crossorigin="anonymous"></script>








  
  <script id="search-hit-fuse-template" type="text/x-template">
    <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
    </div>
  </script>
  
    <script src="https://cdn.jsdelivr.net/gh/krisk/Fuse@v3.2.1/dist/fuse.min.js" integrity="sha512-o38bmzBGX+hD3JHWUFCDA09btWaqrNmoJ3RXLlrysA7PP01Kgs4UlE4MhelE1v5dJR3+cxlR4qQlotsW7jKsnw==" crossorigin="anonymous"></script>
    <script src="https://cdn.jsdelivr.net/gh/julmot/mark.js@8.11.1/dist/jquery.mark.min.js" integrity="sha512-mhbv5DqBMgrWL+32MmsDOt/OAvqr/cHimk6B8y/bx/xS88MVkYGPiVv2ixKVrkywF2qHplNRUvFsAHUdxZ3Krg==" crossorigin="anonymous"></script>
  









  <script id="dsq-count-scr" src="https://fellgernonbit.disqus.com/count.js" async></script>




  
  
  
  
  
  
  







<script id="page-data" type="application/json">{"use_headroom":true}</script>



  <script src="/js/wowchemy-headroom.db4755770454eb63685f8de785c0a172.js" type="module"></script>









  
  


<script src="/en/js/wowchemy.min.2945731fa88332cb0ec33cf47d29879f.js"></script>



  <script src="/js/wowchemy-map.a26e9d2f7238ba5b868384f1c5bc6477.js" type="module"></script>




  
<div id="modal" class="modal fade" role="dialog">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <h5 class="modal-title">Cite</h5>
        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
      </div>
      <div class="modal-body">
        
        <pre><code></code></pre>
      </div>
      <div class="modal-footer">
        <a class="btn btn-outline-primary my-1 js-copy-cite" href="#" target="_blank">
          <i class="fas fa-copy"></i> Copy
        </a>
        <a class="btn btn-outline-primary my-1 js-download-cite" href="#" target="_blank">
          <i class="fas fa-download"></i> Download
        </a>
        <div id="modal-error"></div>
      </div>
    </div>
  </div>
</div>


  <script src="/js/wowchemy-publication.af9327db0521d4a01354bfc8b77a4324.js" type="module"></script>


















</body>
</html>
