<!DOCTYPE html><html lang="en-us" >

<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="generator" content="Source Themes Academic 4.8.0">

  

  
  

  
  
  
  
  
    
    
    
  
  

  <meta name="author" content="Leonardo Collado-Torres">

  
  
  
    
  
  <meta name="description" content="Investigator">

  
  <link rel="alternate" hreflang="en-us" href="https://lcolladotor.github.io/publication/">

  


  
  
  
  <meta name="theme-color" content="#e51744">
  

  
  
  
  <script src="/js/mathjax-config.js"></script>
  

  
  
  
  
    
    
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.14.0/css/all.min.css" integrity="sha256-FMvZuGapsJLjouA6k7Eo2lusoAX9i0ShlWFG6qt7SLc=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.css" integrity="sha256-Vzbj7sDDS/woiFS3uNKo8eIuni59rjyNGtXfstRzStA=" crossorigin="anonymous">

    
    
    
      
    
    
      
      
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.1.2/styles/monokai.min.css" crossorigin="anonymous" title="hl-light">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.1.2/styles/monokai.min.css" crossorigin="anonymous" title="hl-dark" disabled>
      
    

    
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.5.1/leaflet.css" integrity="sha256-SHMGCYmST46SoyGgo4YR/9AlK1vf3ff84Aq9yK4hdqM=" crossorigin="anonymous">
    

    

    
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
        <script src="https://cdnjs.cloudflare.com/ajax/libs/lazysizes/5.1.2/lazysizes.min.js" integrity="sha256-Md1qLToewPeKjfAHU1zyPwOutccPAm5tahnaw7Osw0A=" crossorigin="anonymous" async></script>
      
    
      

      
      

      
    
      

      
      

      
    
      

      
      

      
        <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml.js" integrity="" crossorigin="anonymous" async></script>
      
    
      

      
      

      
    

  

  
  
  
    
      
      
      <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat:400,700%7CRoboto:400,400italic,700%7CRoboto+Mono%7CKarma%7CMina&display=swap">
    
  

  
  
  
  
  <link rel="stylesheet" href="/css/academic.css">

  





<script async src="https://www.googletagmanager.com/gtag/js?id=UA-45299226-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];

  function gtag() {
      dataLayer.push(arguments);
  }

  function trackOutboundLink(url, target) {
    gtag('event', 'click', {
         'event_category': 'outbound',
         'event_label': url,
         'transport_type': 'beacon',
         'event_callback': function () {
           if (target !== '_blank') {
             document.location = url;
           }
         }
    });
    console.debug("Outbound link clicked: " + url);
  }

  function onClickCallback(event) {
    if ((event.target.tagName !== 'A') || (event.target.host === window.location.host)) {
      return;
    }
    trackOutboundLink(event.target, event.target.getAttribute('target'));  
  }

  gtag('js', new Date());
  gtag('config', 'UA-45299226-1', {});

  
  document.addEventListener('click', onClickCallback, false);
</script>


  


  
  

  
  <link rel="alternate" href="/publication/index.xml" type="application/rss+xml" title="L. Collado-Torres">
  

  <link rel="manifest" href="/index.webmanifest">
  <link rel="icon" type="image/png" href="/images/icon_hu2b32c4ab415f12f472f73c7b0301b0d1_19400_32x32_fill_lanczos_center_2.png">
  <link rel="apple-touch-icon" type="image/png" href="/images/icon_hu2b32c4ab415f12f472f73c7b0301b0d1_19400_192x192_fill_lanczos_center_2.png">

  <link rel="canonical" href="https://lcolladotor.github.io/publication/">

  
  
  
  
  
  
  
    
    
  
  
  <meta property="twitter:card" content="summary">
  
  <meta property="twitter:site" content="@lcolladotor">
  <meta property="twitter:creator" content="@lcolladotor">
  
  <meta property="og:site_name" content="L. Collado-Torres">
  <meta property="og:url" content="https://lcolladotor.github.io/publication/">
  <meta property="og:title" content="Publications | L. Collado-Torres">
  <meta property="og:description" content="Investigator"><meta property="og:image" content="https://lcolladotor.github.io/images/icon_hu2b32c4ab415f12f472f73c7b0301b0d1_19400_512x512_fill_lanczos_center_2.png">
  <meta property="twitter:image" content="https://lcolladotor.github.io/images/icon_hu2b32c4ab415f12f472f73c7b0301b0d1_19400_512x512_fill_lanczos_center_2.png"><meta property="og:locale" content="en-us">
  
    <meta property="og:updated_time" content="2020-07-29T13:00:00-05:00">
  

  




  


  





  <title>Publications | L. Collado-Torres</title>

</head>


<body id="top" data-spy="scroll" data-offset="70" data-target="#TableOfContents" class=" ">

  <aside class="search-results" id="search">
  <div class="container">
    <section class="search-header">

      <div class="row no-gutters justify-content-between mb-3">
        <div class="col-6">
          <h1>Search</h1>
        </div>
        <div class="col-6 col-search-close">
          <a class="js-search" href="#"><i class="fas fa-times-circle text-muted" aria-hidden="true"></i></a>
        </div>
      </div>

      <div id="search-box">
        
        <input name="q" id="search-query" placeholder="Search..." autocapitalize="off"
        autocomplete="off" autocorrect="off" spellcheck="false" type="search" class="form-control">
        
      </div>

    </section>
    <section class="section-search-results">

      <div id="search-hits">
        
      </div>

    </section>
  </div>
</aside>


  












<nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id="navbar-main">
  <div class="container">

    
    <div class="d-none d-lg-inline-flex">
      <a class="navbar-brand" href="/">L. Collado-Torres</a>
    </div>
    

    
    <button type="button" class="navbar-toggler" data-toggle="collapse"
            data-target="#navbar-content" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
    <span><i class="fas fa-bars"></i></span>
    </button>
    

    
    <div class="navbar-brand-mobile-wrapper d-inline-flex d-lg-none">
      <a class="navbar-brand" href="/">L. Collado-Torres</a>
    </div>
    

    
    
    <div class="navbar-collapse main-menu-item collapse justify-content-start" id="navbar-content">

      
      <ul class="navbar-nav d-md-inline-flex">
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#about"><span>Home</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#projects"><span>Projects</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#publications"><span>Publications</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#talks_favorite"><span>Talks</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#blog"><span>Blog</span></a>
        </li>

        
        

        

        
        
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#cv"><span>CV</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#mentoring"><span>Mentoring</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#teaching"><span>Teaching</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#contact"><span>Contact</span></a>
        </li>

        
        

      

        
      </ul>
    </div>

    <ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2">
      
      <li class="nav-item">
        <a class="nav-link js-search" href="#" aria-label="Search"><i class="fas fa-search" aria-hidden="true"></i></a>
      </li>
      

      
      <li class="nav-item dropdown theme-dropdown">
        <a href="#" class="nav-link js-theme-selector" data-toggle="dropdown" aria-haspopup="true">
          <i class="fas fa-palette" aria-hidden="true"></i>
        </a>
        <div class="dropdown-menu">
          <a href="#" class="dropdown-item js-set-theme-light">
            <span>Light</span>
          </a>
          <a href="#" class="dropdown-item js-set-theme-dark">
            <span>Dark</span>
          </a>
          <a href="#" class="dropdown-item js-set-theme-auto">
            <span>Automatic</span>
          </a>
        </div>
      </li>
      

      

    </ul>

  </div>
</nav>



  












  

  
  
  
    
  
<div class="universal-wrapper pt-3">
  <h1>Publications</h1>

  

  
</div>



<div class="universal-wrapper">
  <div class="row">
    <div class="col-lg-12">

      

      
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      
        
        
      

      <div class="form-row mb-4">
        <div class="col-auto">
          <input type="search" class="filter-search" placeholder="Search..." autocapitalize="off"
          autocomplete="off" autocorrect="off" role="textbox" spellcheck="false">
        </div>
        <div class="col-auto">
          <select class="pub-filters pubtype-select form-control form-control-sm" data-filter-group="pubtype">
            <option value="*">Type</option>
            
            
            <option value=".pubtype-0">
              Uncategorized
            </option>
            
            <option value=".pubtype-2">
              Journal article
            </option>
            
            <option value=".pubtype-3">
              Preprint
            </option>
            
            <option value=".pubtype-6">
              Book section
            </option>
            
          </select>
        </div>
        <div class="col-auto">
          <select class="pub-filters form-control form-control-sm" data-filter-group="year">
            <option value="*">Date</option>
            
            
            
            <option value=".year-2020">
              2020
            </option>
            
            <option value=".year-2019">
              2019
            </option>
            
            <option value=".year-2018">
              2018
            </option>
            
            <option value=".year-2017">
              2017
            </option>
            
            <option value=".year-2016">
              2016
            </option>
            
            <option value=".year-2015">
              2015
            </option>
            
            <option value=".year-2014">
              2014
            </option>
            
            <option value=".year-2013">
              2013
            </option>
            
            <option value=".year-2011">
              2011
            </option>
            
            <option value=".year-2010">
              2010
            </option>
            
            
          </select>
        </div>
      </div>

      <div id="container-publications">
        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-0 year-2020">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span class="font-weight-bold"><a href="/authors/admin/">Leonardo Collado-Torres</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    Wed, Jul 29, 2020
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Poster presented at BioC2020
    
  </span>
  

  

  
  
  
  <span class="middot-divider"></span>
  <a href="/publication/poster2020bioc/#disqus_thread"></a>
  

  
  

</div>

  

  
  
  
  
  <a href="/publication/poster2020bioc/">
      <img src="/publication/poster2020bioc/featured_hu02753402475fbe364fdae075c891378c_899871_918x517_fill_q90_lanczos_smart1_2.png" class="article-banner" alt="Transcriptome-scale spatial gene expression in the human dorsolateral prefrontal cortex">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/poster2020bioc/">Transcriptome-scale spatial gene expression in the human dorsolateral prefrontal cortex</a>
  </h3>

  
  <a href="/publication/poster2020bioc/" class="summary-link">
    <div class="article-style">
      <p>Poster on the spatialLIBD project for BioC2020.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://f1000research.com/posters/9-762" target="_blank" rel="noopener">
  PDF
</a>







  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/project/spatial/">
    Project
  </a>
  









<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.7490/f1000research.1118079.1" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2020">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Kira A Perzel Mandell</span>, <span >Amanda J Price</span>, <span >Richard Wilton</span>, <span class="font-weight-bold"><a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >Ran Tao</span>, <span >Nicholas J Eagles</span>, <span >Alexander S Szalay</span>, <span >Thomas M Hyde</span>, <span >Daniel R Weinberger</span>, <span >Joel E Kleinman</span>, <span >Andrew E Jaffe &amp;dagger;</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    Wed, Jul 15, 2020
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Epigenetics
    
  </span>
  

  

  
  
  
  <span class="middot-divider"></span>
  <a href="/publication/2020-07_kira_prenatal/#disqus_thread"></a>
  

  
  

</div>

  

  
  
  
  
  <a href="/publication/2020-07_kira_prenatal/">
      <img src="/publication/2020-07_kira_prenatal/featured_hu168b5967c6d231d1a2049e6a98f492f8_639899_918x517_fill_q90_lanczos_top_2.png" class="article-banner" alt="Characterizing the dynamic and functional DNA methylation landscape in the developing human cortex">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/2020-07_kira_prenatal/">Characterizing the dynamic and functional DNA methylation landscape in the developing human cortex</a>
  </h3>

  
  <a href="/publication/2020-07_kira_prenatal/" class="summary-link">
    <div class="article-style">
      <p>DNA methylation (DNAm) is a key epigenetic regulator of gene expression across development. The developing prenatal brain is a highly dynamic tissue, but our understanding of key drivers of epigenetic variability across development is limited. We, therefore, assessed genomic methylation at over 39 million sites in the prenatal cortex using whole-genome bisulfite sequencing and found loci and regions in which methylation levels are dynamic across development. We saw that DNAm at these loci was associated with nearby gene expression and enriched for enhancer chromatin states in prenatal brain tissue. Additionally, these loci were enriched for genes associated with neuropsychiatric disorders and genes involved with neurogenesis. We also found autosomal differences in DNAm between the sexes during prenatal development, though these have less clear functional consequences. We lastly confirmed that the dynamic methylation at this critical period is specifically CpG methylation, with generally low levels of CpH methylation. Our findings provide detailed insight into prenatal brain development as well as clues to the pathogenesis of psychiatric traits seen later in life.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www.biorxiv.org/content/10.1101/823781v1" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1080/15592294.2020.1786304" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2020-07_kira_prenatal/cite.bib">
  Cite
</button>





  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/project/">
    Project
  </a>
  









<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1080/15592294.2020.1786304" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2020">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Yoichi Araki</span>, <span >Ingie Hong</span>, <span >Timothy R Gamache</span>, <span >Shaowen Ju</span>, <span class="font-weight-bold"><a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >Joo Heon Shin</span>, <span >Richard L Huganir &amp;dagger;</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    Wed, Jun 24, 2020
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      eLife
    
  </span>
  

  

  
  
  
  <span class="middot-divider"></span>
  <a href="/publication/2020-06_syngap/#disqus_thread"></a>
  

  
  

</div>

  

  
  
  
  
  <a href="/publication/2020-06_syngap/">
      <img src="/publication/2020-06_syngap/featured_hu0b62227d5f5954724cad787018f45d6e_382929_918x517_fill_q90_lanczos_top.jpg" class="article-banner" alt="SynGAP isoforms differentially regulate synaptic plasticity and dendritic development">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/2020-06_syngap/">SynGAP isoforms differentially regulate synaptic plasticity and dendritic development</a>
  </h3>

  
  <a href="/publication/2020-06_syngap/" class="summary-link">
    <div class="article-style">
      <p>SynGAP is a synaptic Ras GTPase-activating protein (GAP) with four C-terminal splice variants: α1, α2, β, and γ. Although studies have implicated SYNGAP1 in several cognitive disorders, it is not clear which SynGAP isoforms contribute to disease. Here, we demonstrate that SynGAP isoforms exhibit unique spatiotemporal expression patterns and play distinct roles in neuronal and synaptic development in mouse neurons. SynGAP-α1, which undergoes liquid-liquid phase separation with PSD-95, is highly enriched in synapses and is required for LTP. In contrast, SynGAP-β, which does not bind PSD-95 PDZ domains, is less synaptically targeted and promotes dendritic arborization. A mutation in SynGAP-α1 that disrupts phase separation and synaptic targeting abolishes its ability to regulate plasticity and instead causes it to drive dendritic development like SynGAP-β. These results demonstrate that distinct intrinsic biochemical properties of SynGAP isoforms determine their function, and individual isoforms may differentially contribute to the pathogenesis of SYNGAP1-related cognitive disorders.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www.biorxiv.org/content/10.1101/2020.01.28.922013v2" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://elifesciences.org/articles/56273" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2020-06_syngap/cite.bib">
  Cite
</button>





  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/project/brainseq/">
    Project
  </a>
  









<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.7554/eLife.56273" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2020">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Joselyn Chávez</span>, <span >Carmina Barberena-Jonas</span>, <span >Jesus E Sotelo-Fonseca</span>, <span >José Alquicira-Hernández</span>, <span >Heladia Salgado</span>, <span >[__Leonardo Collado-Torres__](/authors/admin) &amp;dagger;</span>, <span >Alejandro Reyes &amp;dagger;</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    Tue, Jun 23, 2020
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Bioinformatics
    
  </span>
  

  

  
  
  
  <span class="middot-divider"></span>
  <a href="/publication/2020-06_regutools/#disqus_thread"></a>
  

  
  

</div>

  

  
  
  
  
  <a href="/publication/2020-06_regutools/">
      <img src="/publication/2020-06_regutools/featured_hu7cb23724d53b4f1bab5c2fdb7cc2d2f3_294111_918x517_fill_q90_lanczos_top_2.png" class="article-banner" alt="Programmatic access to bacterial regulatory networks with regutools">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/2020-06_regutools/">Programmatic access to bacterial regulatory networks with regutools</a>
  </h3>

  
  <a href="/publication/2020-06_regutools/" class="summary-link">
    <div class="article-style">
      <p>RegulonDB has collected, harmonized and centralized data from hundreds of experiments for nearly two decades and is considered a point of reference for transcriptional regulation in Escherichia coli K12. Here, we present the regutools R package to facilitate programmatic access to RegulonDB data in computational biology. regutools gives researchers the possibility of writing reproducible workflows with automated queries to RegulonDB. The regutools package serves as a bridge between RegulonDB data and the Bioconductor ecosystem by reusing the data structures and statistical methods powered by other Bioconductor packages. We demonstrate the integration of regutools with Bioconductor by analyzing transcription factor DNA binding sites and transcriptional regulatory networks from RegulonDB. We anticipate that regutools will serve as a useful building block in our progress to further our understanding of gene regulatory networks.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1101/2020.04.29.068551" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1093/bioinformatics/btaa575" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2020-06_regutools/cite.bib">
  Cite
</button>


  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://bioconductor.org/packages/regutools" target="_blank" rel="noopener">
  Code
</a>




  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/project/">
    Project
  </a>
  








  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://github.com/ComunidadBioInfo/regutools" target="_blank" rel="noopener">
  Source Document
</a>


<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1093/bioinformatics/btaa575" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2020">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >David Zhang __*__</span>, <span >Sebastian Guelfi __*__</span>, <span >Sonia Garcia Ruiz</span>, <span >Beatrice Costa</span>, <span >Regina H Reynolds</span>, <span >Karishma D&#39;Sa</span>, <span >Wenfei Liu</span>, <span >Thomas Courtin</span>, <span >Amy Peterson</span>, <span >Andrew E Jaffe</span>, <span >John Hardy</span>, <span >Juan Botia</span>, <span class="font-weight-bold"><a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >Mina Ryten</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    Wed, Jun 10, 2020
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Science Advances
    
  </span>
  

  

  
  
  
  <span class="middot-divider"></span>
  <a href="/publication/2020-06_omimderfinder/#disqus_thread"></a>
  

  
  

</div>

  

  
  
  
  
  <a href="/publication/2020-06_omimderfinder/">
      <img src="/publication/2020-06_omimderfinder/featured_hua363891e5ce89fa451d0d5de868c889a_190185_918x517_fill_q90_lanczos_smart1.jpg" class="article-banner" alt="Incomplete annotation has a disproportionate impact on our understanding of Mendelian and complex neurogenetic disorders">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/2020-06_omimderfinder/">Incomplete annotation has a disproportionate impact on our understanding of Mendelian and complex neurogenetic disorders</a>
  </h3>

  
  <a href="/publication/2020-06_omimderfinder/" class="summary-link">
    <div class="article-style">
      <p>Growing evidence suggests that human gene annotation remains incomplete; however, it is unclear how this affects different tissues and our understanding of different disorders. Here, we detect previously unannotated transcription from Genotype-Tissue Expression RNA sequencing data across 41 human tissues. We connect this unannotated transcription to known genes, confirming that human gene annotation remains incomplete, even among well-studied genes including 63% of the Online Mendelian Inheritance in Man–morbid catalog and 317 neurodegeneration-associated genes. We find the greatest abundance of unannotated transcription in brain and genes highly expressed in brain are more likely to be reannotated. We explore examples of reannotated disease genes, such as SNCA, for which we experimentally validate a previously unidentified, brain-specific, potentially protein-coding exon. We release all tissue-specific transcriptomes through vizER: <a href="http://rytenlab.com/browser/app/vizER">http://rytenlab.com/browser/app/vizER</a>. We anticipate that this resource will facilitate more accurate genetic analysis, with the greatest impact on our understanding of Mendelian and complex neurogenetic disorders.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www.biorxiv.org/content/10.1101/499103v1" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1126/sciadv.aay8299" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2020-06_omimderfinder/cite.bib">
  Cite
</button>





  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/project/derfinder/">
    Project
  </a>
  









<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1126/sciadv.aay8299" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2020">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Andrew E Jaffe __*__ &amp;dagger;</span>, <span >Daniel J Hoeppner __*__</span>, <span >Takeshi Saito</span>, <span >Lou Blanpain</span>, <span >Joy Ukaigwe</span>, <span >Emily E Burke</span>, <span class="font-weight-bold"><a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >Ran Rato</span>, <span >Katsunori Tajinda</span>, <span >Kristen R Maynard</span>, <span >Matthew N Tran</span>, <span >Keri Martinowich</span>, <span >Amy Deep-Soboslay</span>, <span >Joo Heon Shin</span>, <span >Joel E Kleinman</span>, <span >Daniel R Weinberger</span>, <span >Mitsuyuki Matsumoto &amp;dagger;</span>, <span >Thomas M Hyde &amp;dagger;</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    Mon, Mar 16, 2020
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>Nature Neuroscience</em>
    
  </span>
  

  

  
  
  
  <span class="middot-divider"></span>
  <a href="/publication/2020-03_dglcm/#disqus_thread"></a>
  

  
  

</div>

  

  
  
  
  
  <a href="/publication/2020-03_dglcm/">
      <img src="/publication/2020-03_dglcm/featured_hud0c585195ca5ba87424a4f53610a5ab3_994655_918x517_fill_q90_lanczos_smart1_2.png" class="article-banner" alt="Profiling gene expression in the human dentate gyrus granule cell layer reveals insights into schizophrenia and its genetic risk">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/2020-03_dglcm/">Profiling gene expression in the human dentate gyrus granule cell layer reveals insights into schizophrenia and its genetic risk</a>
  </h3>

  
  <a href="/publication/2020-03_dglcm/" class="summary-link">
    <div class="article-style">
      <p>Jaffe et al. profile the granule cell layer of the human hippocampus and find unique molecular associations for aging and genetic variation, as well as diagnosis with schizophrenia and its genetic risk, that were previously undiscovered in homogenate tissue.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1038/s41593-020-0604-z" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2020-03_dglcm/cite.bib">
  Cite
</button>





  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/project/brainseq/">
    Project
  </a>
  









<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1038/s41593-020-0604-z" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-3 year-2020">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Kristen R Maynard __*__</span>, <span >[__Leonardo Collado-Torres__](/authors/admin) __*__</span>, <span >Lukas M. Weber</span>, <span >Cedric Uytingco</span>, <span >Brianna K. Barry</span>, <span >Stephen R. Williams</span>, <span >Joseph L. Catallini II</span>, <span >Matthew N. Tran</span>, <span >Zachary Besich</span>, <span >Madhavi Tippani</span>, <span >Jennifer Chew</span>, <span >Yifeng Yin</span>, <span >Joel E. Kleinman</span>, <span >Thomas M. Hyde</span>, <span >Nikhil Rao</span>, <span >Stephanie C. Hicks</span>, <span >Keri Martinowich &amp;dagger;</span>, <span >Andrew E Jaffe &amp;dagger;</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    Fri, Feb 28, 2020
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      <em>bioRxiv</em>
    
  </span>
  

  

  
  
  
  <span class="middot-divider"></span>
  <a href="/publication/preprint_spatiallibd/#disqus_thread"></a>
  

  
  

</div>

  

  
  
  
  
  <a href="/publication/preprint_spatiallibd/">
      <img src="/publication/preprint_spatiallibd/featured_hu62124e2a037f7a24ac3aea52e85e2499_360394_918x517_fill_q90_lanczos_smart1.jpg" class="article-banner" alt="Transcriptome-scale spatial gene expression in the human dorsolateral prefrontal cortex">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/preprint_spatiallibd/">Transcriptome-scale spatial gene expression in the human dorsolateral prefrontal cortex</a>
  </h3>

  
  <a href="/publication/preprint_spatiallibd/" class="summary-link">
    <div class="article-style">
      <p>We used the 10x Genomics Visium platform to define the spatial topography of gene expression in the six-layered human dorsolateral prefrontal cortex (DLPFC). We identified extensive layer-enriched expression signatures, and refined associations to previous laminar markers. We overlaid our laminar expression signatures onto large-scale single nuclei RNA sequencing data, enhancing spatial annotation of expression-driven clusters. By integrating neuropsychiatric disorder gene sets, we showed differential layer-enriched expression of genes associated with schizophrenia and autism spectrum disorder, highlighting the clinical relevance of spatially-defined expression. We then developed a data-driven framework to define unsupervised clusters in spatial transcriptomics data, which can be applied to other tissues or brain regions where morphological architecture is not as well-defined as cortical laminae. We lastly created a web application for the scientific community to explore these raw and summarized data to augment ongoing neuroscience and spatial transcriptomics research (<a href="http://research.libd.org/spatialLIBD)">http://research.libd.org/spatialLIBD)</a>.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www.biorxiv.org/content/10.1101/2020.02.28.969931v1" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/preprint_spatiallibd/cite.bib">
  Cite
</button>


  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://bioconductor.org/packages/spatialLIBD" target="_blank" rel="noopener">
  Code
</a>


  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://github.com/LieberInstitute/HumanPilot" target="_blank" rel="noopener">
  Dataset
</a>



  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/project/spatial/">
    Project
  </a>
  








  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://github.com/LieberInstitute/spatialLIBD" target="_blank" rel="noopener">
  Source Document
</a>


<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1101/2020.02.28.969931" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2020">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Sebastian Guelfi __*__</span>, <span >Karishma D’Sa __*__</span>, <span >Juan Botía __*__</span>, <span >Jana Vandrovcova</span>, <span >Regina H. Reynolds</span>, <span >David Zhang</span>, <span >Daniah Trabzuni</span>, <span class="font-weight-bold"><a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >Andrew Thomason</span>, <span >Pedro Quijada Leyton</span>, <span >Sarah A. Gagliano</span>, <span >Mike A. Nalls</span>, <span >International Parkinson’s Disease Genomics Consortium (IPDGC)</span>, <span >UK Brain Expression Consortium</span>, <span >Kerrin S. Small</span>, <span >Colin Smith</span>, <span >Adaikalavan Ramasamy</span>, <span >John Hardy</span>, <span >Michael E. Weale &amp;dagger;</span>, <span >Mina Ryten &amp;dagger;</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    Tue, Feb 25, 2020
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Nature Communications
    
  </span>
  

  

  
  
  
  <span class="middot-divider"></span>
  <a href="/publication/2020-02_basalganglia/#disqus_thread"></a>
  

  
  

</div>

  

  
  
  
  
  <a href="/publication/2020-02_basalganglia/">
      <img src="/publication/2020-02_basalganglia/featured_hud2f61d6bc340e6eee2dfa5aae56c8cda_458522_918x517_fill_q90_lanczos_smart1_2.png" class="article-banner" alt="Regulatory sites for splicing in human basal ganglia are enriched for disease-relevant information">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/2020-02_basalganglia/">Regulatory sites for splicing in human basal ganglia are enriched for disease-relevant information</a>
  </h3>

  
  <a href="/publication/2020-02_basalganglia/" class="summary-link">
    <div class="article-style">
      <p>Genome-wide association studies have generated an increasing number of common genetic variants associated with neurological and psychiatric disease risk. An improved understanding of the genetic control of gene expression in human brain is vital considering this is the likely modus operandum for many causal variants. However, human brain sampling complexities limit the explanatory power of brain-related expression quantitative trait loci (eQTL) and allele-specific expression (ASE) signals. We address this, using paired genomic and transcriptomic data from putamen and substantia nigra from 117 human brains, interrogating regulation at different RNA processing stages and uncovering novel transcripts. We identify disease-relevant regulatory loci, find that splicing eQTLs are enriched for regulatory information of neuron-specific genes, that ASEs provide cell-specific regulatory information with evidence for cellular specificity, and that incomplete annotation of the brain transcriptome limits interpretation of risk loci for neuropsychiatric disease. This resource of regulatory data is accessible through our web server, <a href="http://braineacv2.inf.um.es/">http://braineacv2.inf.um.es/</a>.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www.biorxiv.org/content/10.1101/591156v1" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1038/s41467-020-14483-x" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2020-02_basalganglia/cite.bib">
  Cite
</button>





  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/project/recount2/">
    Project
  </a>
  









<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1038/s41467-020-14483-x" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2020">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Eddie-Luidy Imada __*__</span>, <span >Diego Fernando Sanchez __*__</span>, <span class="font-weight-bold"><a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >Christopher Wilks</span>, <span >Tejasvi Matam</span>, <span >Wikum Dinalankara</span>, <span >Aleksey Stupnikov</span>, <span >Francisco Lobo-Pereira</span>, <span >Chi-Wai Yip</span>, <span >Kayoko Yasuzawa</span>, <span >Naoto Kondo</span>, <span >Masayoshi Itoh</span>, <span >Harukazu Suzuki</span>, <span >Takeya Kasukawa</span>, <span >Chung-Chau Hon</span>, <span >Michiel JL de Hoon</span>, <span >Jay W Shin</span>, <span >Piero Carninci</span>, <span >Andrew E Jaffe</span>, <span >Jeffrey T Leek</span>, <span >Alexander Favorov</span>, <span >Glória R Franco</span>, <span >Benjamin Langmead &amp;dagger;</span>, <span >Luigi Marchionni &amp;dagger;</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    Thu, Feb 20, 2020
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Genome Res
    
  </span>
  

  

  
  
  
  <span class="middot-divider"></span>
  <a href="/publication/2020-02_fantomrecount2/#disqus_thread"></a>
  

  
  

</div>

  

  
  
  
  
  <a href="/publication/2020-02_fantomrecount2/">
      <img src="/publication/2020-02_fantomrecount2/featured_hu70239331444135becdd67206afa4f719_482326_918x517_fill_q90_lanczos_top_2.png" class="article-banner" alt="Recounting the FANTOM CAGE–Associated Transcriptome">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/2020-02_fantomrecount2/">Recounting the FANTOM CAGE–Associated Transcriptome</a>
  </h3>

  
  <a href="/publication/2020-02_fantomrecount2/" class="summary-link">
    <div class="article-style">
      <p>Long noncoding RNAs (lncRNAs) have emerged as key coordinators of biological and cellular processes. Characterizing lncRNA expression across cells and tissues is key to understanding their role in determining phenotypes including human diseases. We present here FC-R2, a comprehensive expression atlas across a broadly defined human transcriptome, inclusive of over 109,000 coding and noncoding genes, as described in the FANTOM CAGE-Associated Transcriptome (FANTOM-CAT) study. This atlas greatly extends the gene annotation used in the original recount2 resource. We demonstrate the utility of the FC-R2 atlas by reproducing key findings from published large studies and by generating new results across normal and diseased human samples. In particular, we (a) identify tissue-specific transcription profiles for distinct classes of coding and noncoding genes, (b) perform differential expression analysis across thirteen cancer types, identifying novel noncoding genes potentially involved in tumor pathogenesis and progression, and (c) confirm the prognostic value for several enhancers lncRNAs expression in cancer. Our resource is instrumental for the systematic molecular characterization of lncRNA by the FANTOM6 Consortium. In conclusion, comprised of over 70,000 samples, the FC-R2 atlas will empower other researchers to investigate functions and biological roles of both known coding genes and novel lncRNAs.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www.biorxiv.org/content/10.1101/659490v4" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="http://doi.org/10.1101/gr.254656.119" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2020-02_fantomrecount2/cite.bib">
  Cite
</button>


  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://github.com/eddieimada/fcr2" target="_blank" rel="noopener">
  Code
</a>


  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://jhubiostatistics.shinyapps.io/recount/" target="_blank" rel="noopener">
  Dataset
</a>



  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/project/recount2/">
    Project
  </a>
  









<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1101/gr.254656" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2020">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Emily E Burke __*__</span>, <span >Joshua G Chenoweth __*__</span>, <span >Joo Heon Shin</span>, <span class="font-weight-bold"><a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >Suel Kee Kim</span>, <span >Nicola Micali</span>, <span >Yanhong Wang</span>, <span >Carlo Colantuoni</span>, <span >Richard E Straub</span>, <span >Daniel J Hoeppner</span>, <span >Huei-Ying Chen</span>, <span >Alana Sellers</span>, <span >Kamel Shibbani</span>, <span >Gregory R Hamersky</span>, <span >Marcelo Diaz Bustamante</span>, <span >BaDoi N Phan</span>, <span >William S Ulrich</span>, <span >Cristian Valencia</span>, <span >Amritha Jaishankar</span>, <span >Amanda J Price</span>, <span >Anandita Rajpurohit</span>, <span >Stephen A Semick</span>, <span >Roland Bürli</span>, <span >James C Barrow</span>, <span >Daniel J Hiler</span>, <span >Stephanie Cerceo Page</span>, <span >Keri Martinowich</span>, <span >Thomas M Hyde</span>, <span >Joel E Kleinman</span>, <span >Karen F Berman</span>, <span >José A Apud</span>, <span >Alan J Cross</span>, <span >Nick J Brandon</span>, <span >Daniel R Weinberger</span>, <span >Brady J Maher</span>, <span >Ronald DG McKay &amp;dagger;</span>, <span >Andrew E Jaffe &amp;dagger;</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    Thu, Jan 23, 2020
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Nature Communications
    
  </span>
  

  

  
  
  
  <span class="middot-divider"></span>
  <a href="/publication/2020-01_libdstemcell/#disqus_thread"></a>
  

  
  

</div>

  

  
  
  
  
  <a href="/publication/2020-01_libdstemcell/">
      <img src="/publication/2020-01_libdstemcell/featured_huab1babfa32d082e3c5c715e55287e144_1130004_918x517_fill_q90_lanczos_smart1_2.png" class="article-banner" alt="Dissecting transcriptomic signatures of neuronal differentiation and maturation using iPSCs">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/2020-01_libdstemcell/">Dissecting transcriptomic signatures of neuronal differentiation and maturation using iPSCs</a>
  </h3>

  
  <a href="/publication/2020-01_libdstemcell/" class="summary-link">
    <div class="article-style">
      <p>Human induced pluripotent stem cells (hiPSCs) are a powerful model of neural differentiation and maturation. We present a hiPSC transcriptomics resource on corticogenesis from 5 iPSC donor and 13 subclonal lines across 9 time points over 5 broad conditions: self-renewal, early neuronal differentiation, neural precursor cells (NPCs), assembled rosettes, and differentiated neuronal cells. We identify widespread changes in the expression of both individual features and global patterns of transcription. We next demonstrate that co-culturing human NPCs with rodent astrocytes results in mutually synergistic maturation, and that cell type-specific expression data can be extracted using only sequencing read alignments without cell sorting. We lastly adapt a previously generated RNA deconvolution approach to single-cell expression data to estimate the relative neuronal maturity of iPSC-derived neuronal cultures and human brain tissue. Using many public datasets, we demonstrate neuronal cultures are maturationally heterogeneous but contain subsets of neurons more mature than previously observed.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www.biorxiv.org/content/early/2018/07/31/380758" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1038/s41467-019-14266-z" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2020-01_libdstemcell/cite.bib">
  Cite
</button>





  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/project/">
    Project
  </a>
  









<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1038/s41467-019-14266-z" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2019">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Amanda J. Price __*__</span>, <span >[__Leonardo Collado-Torres__](/authors/admin) __*__</span>, <span >Nikolay A. Ivanov</span>, <span >Wei Xia</span>, <span >Emily E. Burke</span>, <span >Joo Heon Shin</span>, <span >Ran Tao</span>, <span >Liang Ma</span>, <span >Yankai Jia</span>, <span >Thomas M. Hyde</span>, <span >Joel E. Kleinman</span>, <span >Daniel R. Weinberger</span>, <span >Andrew E Jaffe &amp;dagger;</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    Thu, Sep 26, 2019
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Genome Biology
    
  </span>
  

  

  
  
  
  <span class="middot-divider"></span>
  <a href="/publication/2019-09_dnam_mcph_neuron_glia/#disqus_thread"></a>
  

  
  

</div>

  

  
  
  
  
  <a href="/publication/2019-09_dnam_mcph_neuron_glia/">
      <img src="/publication/2019-09_dnam_mcph_neuron_glia/featured_hu60fca4d401fa100a1ee1e2e878287825_334774_918x517_fill_q90_lanczos_smart1_2.png" class="article-banner" alt="Divergent neuronal DNA methylation patterns across human cortical development reveal critical periods and a unique role of CpH methylation">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/2019-09_dnam_mcph_neuron_glia/">Divergent neuronal DNA methylation patterns across human cortical development reveal critical periods and a unique role of CpH methylation</a>
  </h3>

  
  <a href="/publication/2019-09_dnam_mcph_neuron_glia/" class="summary-link">
    <div class="article-style">
      <p><strong>Background</strong>: DNA methylation (DNAm) is a critical regulator of both development and cellular identity and shows unique patterns in neurons. To better characterize maturational changes in DNAm patterns in these cells, we profile the DNAm landscape at single-base resolution across the first two decades of human neocortical development in NeuN+ neurons using whole-genome bisulfite sequencing and compare them to non-neurons (primarily glia) and prenatal homogenate cortex. <strong>Results</strong>: We show that DNAm changes more dramatically during the first 5 years of postnatal life than during the entire remaining period. We further refine global patterns of increasingly divergent neuronal CpG and CpH methylation (mCpG and mCpH) into six developmental trajectories and find that in contrast to genome-wide patterns, neighboring mCpG and mCpH levels within these regions are highly correlated. We integrate paired RNAseq data and identify putative regulation of hundreds of transcripts and their splicing events exclusively by mCpH levels, independently from mCpG levels, across this period. We finally explore the relationship between DNAm patterns and development of brain-related phenotypes and find enriched heritability for many phenotypes within identified DNAm features. <strong>Conclusions</strong>: By profiling DNAm changes in NeuN-sorted neurons over the span of human cortical development, we identify novel, dynamic regions of DNAm that would be masked in homogenate DNAm data; expand on the relationship between CpG methylation, CpH methylation, and gene expression; and find enrichment particularly for neuropsychiatric diseases in genomic regions with cell type-specific, developmentally dynamic DNAm patterns. <strong>Keywords</strong>: DNA methylation, Neurodevelopment, Gene expression, Non-CpG methylation.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www.biorxiv.org/content/10.1101/428391v1" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://genomebiology.biomedcentral.com/track/pdf/10.1186/s13059-019-1805-1" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2019-09_dnam_mcph_neuron_glia/cite.bib">
  Cite
</button>


  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://github.com/LieberInstitute/brain-epigenomics/" target="_blank" rel="noopener">
  Code
</a>




  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/project/">
    Project
  </a>
  









<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1186/s13059-019-1805-1" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2019">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Carrie Wright __*__</span>, <span >Anandita Rajpurohit __*__</span>, <span >Emily E. Burke</span>, <span > Courtney Williams</span>, <span class="font-weight-bold"><a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >Martha Kimos</span>, <span >Nicholas J. Brandon</span>, <span >Alan J. Cross</span>, <span >Andrew E. Jaffe</span>, <span >Daniel R. Weinberger  &amp;dagger;</span>, <span >Joo Heon Shin  &amp;dagger;</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    Fri, Jun 21, 2019
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      BMC Genomics
    
  </span>
  

  

  
  
  
  <span class="middot-divider"></span>
  <a href="/publication/2019-06_mirna_kit_comp/#disqus_thread"></a>
  

  
  

</div>

  

  
  
  
  
  <a href="/publication/2019-06_mirna_kit_comp/">
      <img src="/publication/2019-06_mirna_kit_comp/featured_hu9400eccb33424d02ad41838c423b2097_394531_918x517_fill_q90_lanczos_smart1_2.png" class="article-banner" alt="Comprehensive assessment of multiple biases in small RNA sequencing reveals significant differences in the performance of widely used methods">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/2019-06_mirna_kit_comp/">Comprehensive assessment of multiple biases in small RNA sequencing reveals significant differences in the performance of widely used methods</a>
  </h3>

  
  <a href="/publication/2019-06_mirna_kit_comp/" class="summary-link">
    <div class="article-style">
      <p>Background: RNA sequencing offers advantages over other quantification methods for microRNA (miRNA), yet numerous biases make reliable quantification challenging. Previous evaluations of these biases have focused on adapter ligation bias with limited evaluation of reverse transcription bias or amplification bias. Furthermore, evaluations of the quantification of isomiRs (miRNA isoforms) or the influence of starting amount on performance have been very limited. No study had yet evaluated the quantification of isomiRs of altered length or compared the consistency of results derived from multiple moderate starting inputs. We therefore evaluated quantifications of miRNA and isomiRs using four library preparation kits, with various starting amounts, as well as quantifications following removal of duplicate reads using unique molecular identifiers (UMIs) to mitigate reverse transcription and amplification biases. Results: All methods resulted in false isomiR detection; however, the adapter-free method tested was especially prone to false isomiR detection. We demonstrate that using UMIs improves accuracy and we provide a guide for input amounts to improve consistency. Conclusions: Our data show differences and limitations of current methods, thus raising concerns about the validity of quantification of miRNA and isomiRs across studies. We advocate for the use of UMIs to improve accuracy and reliability of miRNA quantifications.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www.biorxiv.org/content/10.1101/445437v3" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-019-5870-3f" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2019-06_mirna_kit_comp/cite.bib">
  Cite
</button>


  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://github.com/LieberInstitute/miRNA_Kit_Comparison" target="_blank" rel="noopener">
  Code
</a>




  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/project/">
    Project
  </a>
  









<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1186/s12864-019-5870-3" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2019">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span class="font-weight-bold"><a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >Emily E Burke</span>, <span >Amy Peterson</span>, <span >JooHeon Shin</span>, <span >Richard E Straub</span>, <span >Anandita Rajpurohit</span>, <span >Stephen A Semick</span>, <span >William S Ulrich</span>, <span >BrainSeq Consortium</span>, <span >Amanda J Price</span>, <span >Cristian Valencia</span>, <span >Ran Tao</span>, <span >Amy Deep-Soboslay</span>, <span >Thomas M Hyde</span>, <span >Joel E Kleinman</span>, <span >Daniel R Weinberger &amp;dagger;</span>, <span >Andrew E Jaffe &amp;dagger;</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    Tue, Jun 4, 2019
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Neuron
    
  </span>
  

  

  
  
  
  <span class="middot-divider"></span>
  <a href="/publication/2019-06_bsp2/#disqus_thread"></a>
  

  
  

</div>

  

  
  
  
  
  <a href="/publication/2019-06_bsp2/">
      <img src="/publication/2019-06_bsp2/featured_hu169a505711757f29f1e543af28f6c854_18730_918x517_fill_q90_lanczos_top_2.png" class="article-banner" alt="Regional heterogeneity in gene expression, regulation, and coherence in the frontal cortex and hippocampus across development and schizophrenia">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/2019-06_bsp2/">Regional heterogeneity in gene expression, regulation, and coherence in the frontal cortex and hippocampus across development and schizophrenia</a>
  </h3>

  
  <a href="/publication/2019-06_bsp2/" class="summary-link">
    <div class="article-style">
      <p>The hippocampus formation, although prominently implicated in schizophrenia pathogenesis, has been overlooked in large-scale genomics efforts in the schizophrenic brain. We performed RNA-seq in hippocampi and dorsolateral prefrontal cortices (DLPFCs) from 551 individuals (286 with schizophrenia). We identified substantial regional differences in gene expression and found widespread developmental differences that were independent of cellular composition. We identified 48 and 245 differentially expressed genes (DEGs) associated with schizophrenia within the hippocampus and DLPFC, with little overlap between the brain regions. 124 of 163 (76.6%) of schizophrenia GWAS risk loci contained eQTLs in any region. Transcriptome-wide association studies in each region identified many novel schizophrenia risk features that were brain region-specific. Last, we identified potential molecular correlates of in vivo evidence of altered prefrontal-hippocampal functional coherence in schizophrenia. These results underscore the complexity and regional heterogeneity of the transcriptional correlates of schizophrenia and offer new insights into potentially causative biology.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www.biorxiv.org/content/early/2018/09/26/426213" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1016/j.neuron.2019.05.013" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2019-06_bsp2/cite.bib">
  Cite
</button>


  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://github.com/LieberInstitute/brainseq_phase2" target="_blank" rel="noopener">
  Code
</a>


  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://data.mendeley.com/datasets/3j93ybf4md/" target="_blank" rel="noopener">
  Dataset
</a>



  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/project/brainseq/">
    Project
  </a>
  









<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1016/j.neuron.2019.05.013" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-0 year-2019">
          
            





  






<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span class="font-weight-bold"><a href="/authors/admin/">Leonardo Collado-Torres</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    Wed, May 8, 2019
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Poster presented at BoG2019
    
  </span>
  

  

  
  
  
  <span class="middot-divider"></span>
  <a href="/publication/poster2019bog/#disqus_thread"></a>
  

  
  

</div>

  

  
  
  
  
  <a href="/publication/poster2019bog/">
      <img src="/publication/poster2019bog/featured_hu9b0a18854455e950ff0a7d3159e5a719_897436_918x517_fill_q90_lanczos_top_2.png" class="article-banner" alt="recount-brain: a curated repository of human brain RNA-seq datasets metadata">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/poster2019bog/">recount-brain: a curated repository of human brain RNA-seq datasets metadata</a>
  </h3>

  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/recount-brain-bog19/" target="_blank" rel="noopener">
  PDF
</a>




  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://github.com/lcolladotor/recount-brain-bog19/" target="_blank" rel="noopener">
  Code
</a>




  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/project/recount2/">
    Project
  </a>
  











  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-3 year-2019">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span ><a href="/authors/arazmara/">Ashkaun Razmara</a></span>, <span >Ellis SE</span>, <span >Sokolowski DJ</span>, <span >Davis S</span>, <span >Wilson MD</span>, <span >Leek JT</span>, <span >Jaffe AE</span>, <span >[__L Collado-Torres__](/authors/admin) &amp;dagger;</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    Wed, Apr 24, 2019
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      bioRxiv 618025
    
  </span>
  

  

  
  
  
  <span class="middot-divider"></span>
  <a href="/publication/preprint_recount_brain/#disqus_thread"></a>
  

  
  

</div>

  

  
  
  
  
  <a href="/publication/preprint_recount_brain/">
      <img src="/publication/preprint_recount_brain/featured_huaa449328070568318c86c41bb75a245f_103596_918x517_fill_q90_lanczos_smart1.jpg" class="article-banner" alt="recount-brain: a curated repository of human brain RNA-seq datasets metadata">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/preprint_recount_brain/">recount-brain: a curated repository of human brain RNA-seq datasets metadata</a>
  </h3>

  
  <a href="/publication/preprint_recount_brain/" class="summary-link">
    <div class="article-style">
      <p>The usability of publicly-available gene expression data is often limited by the availability of high-quality, standardized biological phenotype and experimental condition information (“metadata”). We released the recount2 project, which involved re-processing ∼70,000 samples in the Sequencing Read Archive (SRA), Genotype-Tissue Expression (GTEx), and The Cancer Genome Atlas (TCGA) projects. While samples from the latter two projects are well-characterized with extensive metadata, the ∼50,000 RNA-seq samples from SRA in recount2 are inconsistently annotated with metadata. Tissue type, sex, and library type can be estimated from the RNA sequencing (RNA-seq) data itself. However, more detailed and harder to predict metadata, like age and diagnosis, must ideally be provided by labs that deposit the data. To facilitate more analyses within human brain tissue data, we have complemented phenotype predictions by manually constructing a uniformly-curated database of public RNA-seq samples present in SRA and recount2. We describe the reproducible curation process for constructing recount-brain that involves systematic review of the primary manuscript, which can serve as a guide to annotate other studies and tissues. We further expanded recount-brain by merging it with GTEx and TCGA brain samples as well as linking to controlled vocabulary terms for tissue, Brodmann area and disease. Furthermore, we illustrate how to integrate the sample metadata in recount-brain with the gene expression data in recount2 to perform differential expression analysis. We then provide three analysis examples involving modeling postmortem interval, glioblastoma, and meta-analyses across GTEx and TCGA. Overall, recount-brain facilitates expression analyses and improves their reproducibility as individual researchers do not have to manually curate the sample metadata. recount-brain is available via the add_metadata() function from the recount Bioconductor package at bioconductor.org/packages/recount.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www.biorxiv.org/content/10.1101/618025v1" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www.biorxiv.org/content/early/2019/04/24/618025.full.pdf" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/preprint_recount_brain/cite.bib">
  Cite
</button>


  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://github.com/LieberInstitute/recount-brain" target="_blank" rel="noopener">
  Code
</a>




  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/project/recount2/">
    Project
  </a>
  



  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/recount-brain-bog19/" target="_blank" rel="noopener">
  Poster
</a>







<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1101/618025" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2019">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Anil K Madugundu</span>, <span >Chan Hyun Na</span>, <span >Raja Sekhar Nirujogi</span>, <span >Santosh Renuse</span>, <span >Kwang Pyo Kim</span>, <span >Kathleen H. Burns</span>, <span >Christopher Wilks</span>, <span >Ben Langmead</span>, <span >Shannon E. Ellis</span>, <span class="font-weight-bold"><a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >Marc K. Halushka</span>, <span >Min-Sik Kim</span>, <span >Akhilesh Pandey &amp;dagger;</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    Sun, Apr 14, 2019
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Proteomics
    
  </span>
  

  

  
  
  
  <span class="middot-divider"></span>
  <a href="/publication/2019-04_recount_proteomics/#disqus_thread"></a>
  

  
  

</div>

  

  
  
  
  
  <a href="/publication/2019-04_recount_proteomics/">
      <img src="/publication/2019-04_recount_proteomics/featured_hu9af44dfb87f3cd13fd97284f076ba1d4_140736_918x517_fill_q90_lanczos_smart1.jpg" class="article-banner" alt="Integrated Transcriptomic and Proteomic Analysis of Primary Human Umbilical Vein Endothelial Cells">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/2019-04_recount_proteomics/">Integrated Transcriptomic and Proteomic Analysis of Primary Human Umbilical Vein Endothelial Cells</a>
  </h3>

  
  <a href="/publication/2019-04_recount_proteomics/" class="summary-link">
    <div class="article-style">
      <p>Understanding the molecular profile of every human cell type is essential for understanding its role in normal physi-ology and disease. Technological advancements in DNA sequencing, mass spectrometry, and computational methodsallow us to carry out multiomics analyses although such approaches are not routine yet. Human umbilical vein endothe-lial cells (HUVECs) are a widely used model system to study pathological and physiological processes associated withthe cardiovascular system. In this study, next-generation sequencing and high-resolution mass spectrometry to profilethe transcriptome and proteome of primary HUVECs is employed. Analysis of 145 million paired-end reads from next-generation sequencing confirmed expression of 12 186 protein-coding genes (FPKM &gt;= 0.1), 439 novel long non-codingRNAs, and revealed 6089 novel isoforms that were not annotated in GENCODE. Proteomics analysis identifies 6477proteins including confirmation ofN-termini for 1091 proteins, isoforms for 149 proteins, and 1034 phosphosites. Adatabase search to specifically identify other post-translational modifications provide evidence for a number of modifi-cation sites on 117 proteins which include ubiquitylation, lysine acetylation, and mono-, di- and tri-methylation events.Evidence for 11 &lsquo;missing proteins&rsquo;, which are proteins for which there was insufficient or no protein level evidence, isprovided. Peptides supporting missing protein and novel events are validated by comparison of MS/MS fragmentationpatterns with synthetic peptides. Finally, 245 variant peptides derived from 207 expressed proteins in addition to alternatetranslational start sites for seven proteins and evidence for novel proteoforms for five proteins resulting from alternativesplicing are identified. Overall, it is believed that the integrated approach employed in this study is widely applicable tostudy any primary cell type for deeper molecular characterization.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://onlinelibrary.wiley.com/doi/epdf/10.1002/pmic.201800315" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2019-04_recount_proteomics/cite.bib">
  Cite
</button>





  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/project/">
    Project
  </a>
  









<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1002/pmic.201800315" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2019">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Stephen A Semick</span>, <span >Rahul A Bharadwaj</span>, <span class="font-weight-bold"><a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >Ran Tao</span>, <span >Joo Heon Shin</span>, <span >Amy Deep-Soboslay</span>, <span >James Weiss</span>, <span >Daniel R Weinberger</span>, <span >Thomas M Hyde</span>, <span >Joel E Kleinman</span>, <span >Andrew E Jaffe &amp;dagger;</span>, <span >Venkata S Mattay &amp;dagger;</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    Sat, Feb 2, 2019
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Acta Neuropathologica
    
  </span>
  

  

  
  
  
  <span class="middot-divider"></span>
  <a href="/publication/2019-02_dnam_alzheimer/#disqus_thread"></a>
  

  
  

</div>

  

  
  
  
  
  <a href="/publication/2019-02_dnam_alzheimer/">
      <img src="/publication/2019-02_dnam_alzheimer/featured_hu1e066182b6f82934619930922facd138_606837_918x517_fill_q90_lanczos_smart1_2.png" class="article-banner" alt="Integrated DNA methylation and gene expression profiling across multiple brain regions implicate novel genes in Alzheimer&#39;s disease">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/2019-02_dnam_alzheimer/">Integrated DNA methylation and gene expression profiling across multiple brain regions implicate novel genes in Alzheimer&#39;s disease</a>
  </h3>

  
  <a href="/publication/2019-02_dnam_alzheimer/" class="summary-link">
    <div class="article-style">
      <p>Late-onset Alzheimer’s disease (AD) is a complex age-related neurodegenerative disorder that likely involves epigenetic factors. To better understand the epigenetic state associated with AD, we surveyed 420,852 DNA methylation (DNAm) sites from neurotypical controls (N=49) and late-onset AD patients (N=24) across four brain regions (hippocampus, entorhinal cortex, dorsolateral prefrontal cortex and cerebellum). We identified 858 sites with robust differential methylation collectively annotated to 772 possible genes (FDR&lt;5%, within 10 kb). These sites were overrepresented in AD genetic risk loci (p=0.00655) and were enriched for changes during normal aging (p&lt;2.2×10−16), and nearby genes were enriched for processes related to cell-adhesion, immunity, and calcium homeostasis (FDR&lt;5%). To functionally validate these associations, we generated and analyzed corresponding transcriptome data to prioritize 130 genes within 10 kb of the differentially methylated sites. These 130 genes were differentially expressed between AD cases and controls and their expression was associated with nearby DNAm (p&lt;0.05). This integrated analysis implicates novel genes in Alzheimer’s disease, such as ANKRD30B. These results highlight DNAm differences in Alzheimer’s disease that have gene expression correlates, further implicating DNAm as an epigenetic mechanism underlying pathological molecular changes associated with AD. Furthermore, our framework illustrates the value of integrating epigenetic and transcriptomic data for understanding complex disease.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www.biorxiv.org/content/early/2018/09/29/430603" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1007/s00401-019-01966-5" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2019-02_dnam_alzheimer/cite.bib">
  Cite
</button>


  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://github.com/LieberInstitute/Alzheimers_DNAm" target="_blank" rel="noopener">
  Code
</a>




  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/project/">
    Project
  </a>
  









<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1007/s00401-019-01966-5" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2018">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Helena Kuri-Magaña</span>, <span class="font-weight-bold"><a href="/authors/admin/">Leonardo Collado-Torres</a></span>, <span >Andrew E Jaffe</span>, <span >Humberto Valdovinos-Torres</span>, <span >Marbella Ovilla-Muñoz</span>, <span >Juan M Téllez-Sosa</span>, <span >Laura C Bonifaz-Alfonzo</span>, <span >Jesús Martínez-Barnetche</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    Wed, Nov 21, 2018
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Frontiers in Immunology
    
  </span>
  

  

  
  
  
  <span class="middot-divider"></span>
  <a href="/publication/2018-11_nccsr/#disqus_thread"></a>
  

  
  

</div>

  

  
  
  
  
  <a href="/publication/2018-11_nccsr/">
      <img src="/publication/2018-11_nccsr/featured_hubb959b6f36053d217f6324d4d26fd23c_793639_918x517_fill_q90_lanczos_top_2.png" class="article-banner" alt="Non-coding Class Switch Recombination-Related Transcription in Human Normal and Pathological Immune Responses">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/2018-11_nccsr/">Non-coding Class Switch Recombination-Related Transcription in Human Normal and Pathological Immune Responses</a>
  </h3>

  
  <a href="/publication/2018-11_nccsr/" class="summary-link">
    <div class="article-style">
      <p>Background: Antibody class switch recombination (CSR) to IgG, IgA or IgE is a hallmark of adaptive immunity, allowing antibody function diversification beyond IgM. CSR involves a deletion of the IgM/IgD constant region genes placing a new acceptor Constant (CH) gene, downstream of the VDJH exon. CSR depends on non-coding (CSRnc) transcription of donor Iμ and acceptor IH exons, located 5′ upstream of each CH coding gene. Although our knowledge of the role of CSRnc transcription has advanced greatly, its extension and importance in healthy and diseased humans is scarce. Methods: We analyzed CSRnc transcription in 70,603 publicly available RNA-seq samples, including GTEx, TCGA and the Sequence Read Archive (SRA) using recount2, an online resource consisting of normalized RNA-seq gene and exon counts, as well as coverage BigWig files that can be programmatically accessed through R. CSRnc transcription was validated with a qRT-PCR assay for Iμ, Iγ1 and Iγ3 in humans in response to vaccination. Results: We mapped IH transcription for the human IgH locus, including the less understood IGHD gene. CSRnc transcription was restricted to B cells and is widely distributed in normal adult tissues, but predominant in blood, spleen, MALT-containing tissues, visceral adipose tissue and some so-called &lsquo;immune privileged&rsquo; tissues. However, significant Iγ4 expression was found even in non-lymphoid fetal tissues. CSRnc expression in cancer tissues mimicked the expression of their normal counterparts, with notable pattern changes in some common cancer subsets. CSRnc transcription in tumors appears to result from tumor infiltration by B cells, since CSRnc transcription was not detected in corresponding tumor-derived immortal cell lines. Additionally, significantly increased Iδ transcription in ileal mucosa in Crohn&rsquo;s disease with ulceration was found. Conclusions: CSRnc transcription occurs in multiple anatomical locations beyond classical secondary lymphoid organs, representing a potentially useful marker of effector B cell responses in normal and pathological immune responses. The pattern of IH exon expression may reveal clues of the local immune response (i.e. cytokine milieu) in health and disease. This is a great example of how the public recount2 data can be used to further our understanding of transcription, including regions outside the known transcriptome.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www.biorxiv.org/content/early/2018/08/03/384172" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www.frontiersin.org/articles/10.3389/fimmu.2018.02679/full" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2018-11_nccsr/cite.bib">
  Cite
</button>





  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/project/recount2/">
    Project
  </a>
  









<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.3389/fimmu.2018.02679" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-0 year-2018">
          
            





  






<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span class="font-weight-bold"><a href="/authors/admin/">Leonardo Collado-Torres</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    Tue, Oct 16, 2018
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Poster presented at ASHG2018, biodata18 and PSB2019
    
  </span>
  

  

  
  
  
  <span class="middot-divider"></span>
  <a href="/publication/poster2018ashg/#disqus_thread"></a>
  

  
  

</div>

  

  
  
  
  
  <a href="/publication/poster2018ashg/">
      <img src="/publication/poster2018ashg/featured_huca91b25d278d606855f49f135164430e_1275613_918x517_fill_q90_lanczos_top_2.png" class="article-banner" alt="Regional heterogeneity in gene expression, regulation and coherence in hippocampus and dorsolateral prefrontal cortex across development and in schizophrenia">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/poster2018ashg/">Regional heterogeneity in gene expression, regulation and coherence in hippocampus and dorsolateral prefrontal cortex across development and in schizophrenia</a>
  </h3>

  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="http://lcolladotor.github.io/Posters/ASHG2018.pdf" target="_blank" rel="noopener">
  PDF
</a>







  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/project/brainseq/">
    Project
  </a>
  











  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2018">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Semick SA</span>, <span >__Collado-Torres L__</span>, <span >Markunas CA</span>, <span >Shin JH</span>, <span >Deep-Soboslay A</span>, <span >Tao R</span>, <span >Huestis MA</span>, <span >Bierut LJ</span>, <span >Maher BS</span>, <span >Johnson EO</span>, <span >Hyde TM</span>, <span >Weinberger DR</span>, <span >Hancock DB</span>, <span >Kleinman JE &amp;dagger;</span>, <span >Jaffe AE &amp;dagger;</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    Tue, Aug 21, 2018
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Molecular Psychiatry
    
  </span>
  

  

  
  
  
  <span class="middot-divider"></span>
  <a href="/publication/2018-08_maternalsmoking/#disqus_thread"></a>
  

  
  

</div>

  

  
  
  
  
  <a href="/publication/2018-08_maternalsmoking/">
      <img src="/publication/2018-08_maternalsmoking/featured_hue8dcb2fbcff4e0f70f3f9b884c84d020_179424_918x517_fill_q90_lanczos_smart1_2.png" class="article-banner" alt="Developmental effects of maternal smoking during pregnancy on the human frontal cortex transcriptome">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/2018-08_maternalsmoking/">Developmental effects of maternal smoking during pregnancy on the human frontal cortex transcriptome</a>
  </h3>

  
  <a href="/publication/2018-08_maternalsmoking/" class="summary-link">
    <div class="article-style">
      <p>This paper is on smoking and its relation to gene expression at different life stages. These genes are also related to autism spectrum disorder.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www.biorxiv.org/content/early/2017/12/20/236968" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www.nature.com/articles/s41380-018-0223-1.pdf" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2018-08_maternalsmoking/cite.bib">
  Cite
</button>


  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://github.com/LieberInstitute/Smoking_DLPFC_Devel" target="_blank" rel="noopener">
  Code
</a>




  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/project/brainseq/">
    Project
  </a>
  









<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1038/s41380-018-0223-1" target="_blank" rel="noopener">
  DOI
</a>


  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://t.co/0RsJ1kDbOF8" target="_blank" rel="noopener">
    
    Read for free
  </a>


  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2018">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Jaffe AE</span>, <span >Straub R</span>, <span >Shin JH</span>, <span >Tao R</span>, <span >Gao Y</span>, <span >__Collado-Torres L__</span>, <span >Kam-Thong T</span>, <span >Xi HS</span>, <span >Quan J</span>, <span >Chen Q</span>, <span >Colantuoni C</span>, <span >Ulrich WS</span>, <span >Maher BJ</span>, <span >Deep-Soboslay A</span>, <span >The BrainSeq Consortium</span>, <span >Cross AJ</span>, <span >Brandon NJ</span>, <span >Leek JT</span>, <span >Hyde TM</span>, <span >Kleinman JE</span>, <span >Weinberger DR</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    Thu, Jul 26, 2018
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Nature Neuroscience
    
  </span>
  

  

  
  
  
  <span class="middot-divider"></span>
  <a href="/publication/2018-07_bsp1/#disqus_thread"></a>
  

  
  

</div>

  

  
  
  
  
  <a href="/publication/2018-07_bsp1/">
      <img src="/publication/2018-07_bsp1/featured_huff893b7c4781f4483265f7536c65ba15_202241_918x517_fill_q90_lanczos_smart1_2.png" class="article-banner" alt="Developmental and genetic regulation of the human cortex transcriptome illuminate schizophrenia pathogenesis">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/2018-07_bsp1/">Developmental and genetic regulation of the human cortex transcriptome illuminate schizophrenia pathogenesis</a>
  </h3>

  
  <a href="/publication/2018-07_bsp1/" class="summary-link">
    <div class="article-style">
      <p>Genome-wide association studies have identified 108 schizophrenia risk loci, but biological mechanisms for individual loci are largely unknown. Using developmental, genetic and illness-based RNA sequencing expression analysis in human brain, we characterized the human brain transcriptome around these loci and found enrichment for developmentally regulated genes with novel examples of shifting isoform usage across pre- and postnatal life. We found widespread expression quantitative trait loci (eQTLs), including many with transcript specificity and previously unannotated sequence that were independently replicated. We leveraged this general eQTL database to show that 48.1% of risk variants for schizophrenia associate with nearby expression. We lastly found 237 genes significantly differentially expressed between patients and controls, which replicated in an independent dataset, implicated synaptic processes, and were strongly regulated in early development. These findings together offer genetics- and diagnosis-related targets for better modeling of schizophrenia risk. This resource is publicly available at <a href="http://eqtl.brainseq.org/phase1">http://eqtl.brainseq.org/phase1</a>.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www.biorxiv.org/content/10.1101/124321v2" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www.nature.com/articles/s41593-018-0197-y.pdf" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2018-07_bsp1/cite.bib">
  Cite
</button>





  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/project/brainseq/">
    Project
  </a>
  

  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/project/derfinder/">
    Project
  </a>
  









<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1038/s41593-018-0197-y" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-0 year-2018">
          
            





  






<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span class="font-weight-bold"><a href="/authors/admin/">Leonardo Collado-Torres</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    Fri, Jun 1, 2018
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Poster presented at SAGES2018
    
  </span>
  

  

  
  
  
  <span class="middot-divider"></span>
  <a href="/publication/poster2018sages/#disqus_thread"></a>
  

  
  

</div>

  

  
  
  
  
  <a href="/publication/poster2018sages/">
      <img src="/publication/poster2018sages/featured_hu113101afe2b5a6c3452c664971211c9a_1402911_918x517_fill_q90_lanczos_smart1_2.png" class="article-banner" alt="BrainSeq Phase II: schizophrenia-associated expression differences between the hippocampus and the dorsolateral prefrontal cortex">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/poster2018sages/">BrainSeq Phase II: schizophrenia-associated expression differences between the hippocampus and the dorsolateral prefrontal cortex</a>
  </h3>

  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="http://lcolladotor.github.io/Posters/SAGES2018.pdf" target="_blank" rel="noopener">
  PDF
</a>







  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/project/brainseq/">
    Project
  </a>
  











  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2018">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Shannon SE</span>, <span >__Collado-Torres L__</span>, <span >Jaffe AE</span>, <span >Leek JT</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    Mon, Mar 5, 2018
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Nucleic Acids Research
    
  </span>
  

  

  
  
  
  <span class="middot-divider"></span>
  <a href="/publication/2018-03_rnaseq_prediction/#disqus_thread"></a>
  

  
  

</div>

  

  
  
  
  
  <a href="/publication/2018-03_rnaseq_prediction/">
      <img src="/publication/2018-03_rnaseq_prediction/featured_hua4d9fb4833bf342dfdc873622df802de_235231_918x517_fill_q90_lanczos_smart1.jpg" class="article-banner" alt="Improving the value of public RNA-seq expression data by phenotype prediction">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/2018-03_rnaseq_prediction/">Improving the value of public RNA-seq expression data by phenotype prediction</a>
  </h3>

  
  <a href="/publication/2018-03_rnaseq_prediction/" class="summary-link">
    <div class="article-style">
      <p>Background: Publicly available genomic data are a valuable resource for studying normal human variation and disease, but these data are often not well labeled or annotated. The lack of phenotype information for public genomic data severely limits their utility for addressing targeted biological questions. Results: We develop an in silico phenotyping approach for predicting critical missing annotation directly from genomic measurements using, well-annotated genomic and phenotypic data produced by consortia like TCGA and GTEx as training data. We apply in silico phenotyping to a set of 70,000 RNA-seq samples we recently processed on a common pipeline as part of the recount2 project (<a href="https://jhubiostatistics.shinyapps.io/recount/)">https://jhubiostatistics.shinyapps.io/recount/)</a>. We use gene expression data to build and evaluate predictors for both biological phenotypes (sex, tissue, sample source) and experimental conditions (sequencing strategy). We demonstrate how these predictions can be used to study cross-sample properties of public genomic data, select genomic projects with specific characteristics, and perform downstream analyses using predicted phenotypes. The methods to perform phenotype prediction are available in the phenopredict R package (<a href="https://github.com/leekgroup/phenopredict">https://github.com/leekgroup/phenopredict</a>) and the predictions for recount2 are available from the recount R package (<a href="https://bioconductor.org/packages/release/bioc/html/recount.html)">https://bioconductor.org/packages/release/bioc/html/recount.html)</a>. Conclusion: Having leveraging massive public data sets to generate a well-phenotyped set of expression data for more than 70,000 human samples, expression data is available for use on a scale that was not previously feasible.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www.biorxiv.org/content/10.1101/145656v1" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://academic.oup.com/nar/advance-article/doi/10.1093/nar/gky102/4920847" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2018-03_rnaseq_prediction/cite.bib">
  Cite
</button>


  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://github.com/ShanEllis/phenopredict_usecase" target="_blank" rel="noopener">
  Code
</a>




  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/project/recount2/">
    Project
  </a>
  









<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1093/nar/gky102" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-3 year-2018">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Fu J</span>, <span >Kammers K</span>, <span >Nellore A</span>, <span >__Collado-Torres L__</span>, <span >Leek JT</span>, <span >Taub MA</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    Fri, Jan 12, 2018
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      bioRxiv 247346
    
  </span>
  

  

  
  
  
  <span class="middot-divider"></span>
  <a href="/publication/preprint_recount_tx/#disqus_thread"></a>
  

  
  

</div>

  

  
  
  
  
  <a href="/publication/preprint_recount_tx/">
      <img src="/publication/preprint_recount_tx/featured_hua06a972d1bb66609511008f642f307a0_136615_918x517_fill_q90_lanczos_smart1_2.png" class="article-banner" alt="RNA-seq transcript quantification from reduced-representation data in recount2">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/preprint_recount_tx/">RNA-seq transcript quantification from reduced-representation data in recount2</a>
  </h3>

  
  <a href="/publication/preprint_recount_tx/" class="summary-link">
    <div class="article-style">
      <p>More than 70,000 short-read RNA-sequencing samples are publicly available through the recount2 project, a curated database of summary coverage data. However, no current methods can estimate transcript-level abundances using the reduced-representation information stored in this database. Here we present a linear model utilizing coverage of junctions and subdivided exons to generate transcript abundance estimates of comparable accuracy to those obtained from methods requiring read-level data. Our approach flexibly models bias, produces standard errors, and is easy to refresh given updated annotation. We illustrate our method on simulated and real data and release transcript abundance estimates for the samples in recount2.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www.biorxiv.org/content/10.1101/247346v2" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www.biorxiv.org/content/early/2018/01/12/247346.full.pdf" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/preprint_recount_tx/cite.bib">
  Cite
</button>





  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/project/recount2/">
    Project
  </a>
  









<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1101/247346" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-0 year-2017">
          
            





  






<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span class="font-weight-bold"><a href="/authors/admin/">Leonardo Collado-Torres</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    Fri, Oct 20, 2017
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Poster presented at IDIES2017
    
  </span>
  

  

  
  
  
  <span class="middot-divider"></span>
  <a href="/publication/poster2017idies/#disqus_thread"></a>
  

  
  

</div>

  

  
  
  
  
  <a href="/publication/poster2017idies/">
      <img src="/publication/poster2017idies/featured_hu87ecd3455c6fe7e504a939f3e07970e2_1416677_918x517_fill_q90_lanczos_top_2.png" class="article-banner" alt="Getting started with recount2 and accessing it via R">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/poster2017idies/">Getting started with recount2 and accessing it via R</a>
  </h3>

  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="http://lcolladotor.github.io/Posters/IDIES2017.pdf" target="_blank" rel="noopener">
  PDF
</a>







  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/project/recount2/">
    Project
  </a>
  











  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2017">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >[__L Collado-Torres__](/authors/admin) &amp;dagger;</span>, <span >A Nellore</span>, <span >AE Jaffe</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    Thu, Aug 24, 2017
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      F1000Research
    
  </span>
  

  

  
  
  
  <span class="middot-divider"></span>
  <a href="/publication/2017-08_recountworkflow/#disqus_thread"></a>
  

  
  

</div>

  

  
  
  
  
  <a href="/publication/2017-08_recountworkflow/">
      <img src="/publication/2017-08_recountworkflow/featured_hu3895338370ae321089968d378f88b52e_7454_918x517_fill_q90_lanczos_smart1_2.png" class="article-banner" alt="recount workflow: Accessing over 70,000 human RNA-seq samples with Bioconductor [version 1; referees: 1 approved, 2 approved with reservations]">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/2017-08_recountworkflow/">recount workflow: Accessing over 70,000 human RNA-seq samples with Bioconductor [version 1; referees: 1 approved, 2 approved with reservations]</a>
  </h3>

  
  <a href="/publication/2017-08_recountworkflow/" class="summary-link">
    <div class="article-style">
      <p>The recount2 resource is composed of over 70,000 uniformly processed human RNA-seq samples spanning TCGA and SRA, including GTEx. The processed data can be accessed via the recount2 website and the recount Bioconductor package. This workflow explains in detail how to use the recount package and how to integrate it with other Bioconductor packages for several analyses that can be carried out with the recount2 resource. In particular, we describe how the coverage count matrices were computed in recount2 as well as different ways of obtaining public metadata, which can facilitate downstream analyses. Step-by-step directions show how to do a gene-level differential expression analysis, visualize base-level genome coverage data, and perform an analyses at multiple feature levels. This workflow thus provides further information to understand the data in recount2 and a compendium of R code to use the data.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://f1000research.com/articles/6-1558/v1" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2017-08_recountworkflow/cite.bib">
  Cite
</button>


  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://github.com/LieberInstitute/recountWorkflow" target="_blank" rel="noopener">
  Code
</a>




  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/project/recount2/">
    Project
  </a>
  









<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.12688/f1000research.12223.1" target="_blank" rel="noopener">
  DOI
</a>


  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="http://bioconductor.org/packages/release/workflows/html/recountWorkflow.html" target="_blank" rel="noopener">
    
    Bioconductor
  </a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="http://LieberInstitute.github.io/recountWorkshop/" target="_blank" rel="noopener">
    
    BioC2017 Workshop
  </a>


  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2017">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >C Wright</span>, <span >JH Shin</span>, <span >A Rajpurohit</span>, <span >A Deep-Soboslay</span>, <span >[__L Collado-Torres__](/authors/admin)</span>, <span >NJ Brandon</span>, <span >TM Hyde</span>, <span >JE Kleinman</span>, <span >AE Jaffe</span>, <span >AJ Cross</span>, <span >DR Weinberger</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    Tue, May 9, 2017
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Translational Psychiatry
    
  </span>
  

  

  
  
  
  <span class="middot-divider"></span>
  <a href="/publication/2017-05_histamine/#disqus_thread"></a>
  

  
  

</div>

  

  
  
  
  
  <a href="/publication/2017-05_histamine/">
      <img src="/publication/2017-05_histamine/featured_hu6d32d65792e3bc285bcd06e6b134828f_172962_918x517_fill_q90_lanczos_smart1_2.png" class="article-banner" alt="Altered expression of histamine signaling genes in autism spectrum disorder">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/2017-05_histamine/">Altered expression of histamine signaling genes in autism spectrum disorder</a>
  </h3>

  
  <a href="/publication/2017-05_histamine/" class="summary-link">
    <div class="article-style">
      <p>Expression of the gene set of HNMT, HRH1, HRH2 and HRH3 was significantly altered between ASD and matched controls, and this finding was replicated with an independent data set.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www.nature.com/tp/journal/v7/n5/full/tp201787a.html" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2017-05_histamine/cite.bib">
  Cite
</button>


  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://github.com/LieberInstitute/Wright_2017_ASD_Histamine" target="_blank" rel="noopener">
  Code
</a>




  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/project/">
    Project
  </a>
  









<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1038/tp.2017.87" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2017">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >[__L Collado-Torres__](/authors/admin) __*__</span>, <span >A Nellore __*__</span>, <span >K Kammers</span>, <span >SE Ellis</span>, <span >MA Taub</span>, <span >KD Hansen</span>, <span >AE Jaffe</span>, <span >B Langmead</span>, <span >JT Leek</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    Tue, Apr 11, 2017
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Nature Biotechnology
    
  </span>
  

  

  
  
  
  <span class="middot-divider"></span>
  <a href="/publication/2017-04_recount/#disqus_thread"></a>
  

  
  

</div>

  

  
  
  
  
  <a href="/publication/2017-04_recount/">
      <img src="/publication/2017-04_recount/featured_hu38d48fdc126e5a16b49d21329437221e_293563_918x517_fill_q90_lanczos_bottom_2.png" class="article-banner" alt="Reproducible RNA-seq analysis using recount2">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/2017-04_recount/">Reproducible RNA-seq analysis using recount2</a>
  </h3>

  
  <a href="/publication/2017-04_recount/" class="summary-link">
    <div class="article-style">
      <p>recount2 is a resource of processed and summarized expression data spanning over 70,000 human RNA-seq samples from the Sequence Read Archive (SRA). The associated recount Bioconductor package provides a convenient API for querying, downloading, and analyzing the data. Each processed study consists of meta/phenotype data, the expression levels of genes and their underlying exons and splice junctions, and corresponding genomic annotation. We also provide data summarization types for quantifying novel transcribed sequence including base-resolution coverage and potentially unannotated splice junctions. We present workflows illustrating how to use recount to perform differential expression analysis including meta-analysis, annotation-free base-level analysis, and replication of smaller studies using data from larger studies. recount provides a valuable and user-friendly resource of processed RNA-seq datasets to draw additional biological insights from existing public data. The resource is available at <a href="https://jhubiostatistics.shinyapps.io/recount/">https://jhubiostatistics.shinyapps.io/recount/</a>.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www.biorxiv.org/content/10.1101/068478v1" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="http://www.nature.com/nbt/journal/v35/n4/full/nbt.3838.html" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2017-04_recount/cite.bib">
  Cite
</button>


  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://github.com/leekgroup/recount-website" target="_blank" rel="noopener">
  Code
</a>




  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/project/recount2/">
    Project
  </a>
  









<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1038/nbt.3838" target="_blank" rel="noopener">
  DOI
</a>


  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="http://bioconductor.org/packages/recount" target="_blank" rel="noopener">
    
    Bioconductor
  </a>


  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2016">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >A Nellore</span>, <span >AE Jaffe</span>, <span >JP Fortin</span>, <span >J Alquicira-Hernández</span>, <span >[__L Collado-Torres__](/authors/admin)</span>, <span >S Wang</span>, <span >RA Phillips</span>, <span >N Karbhari</span>, <span >KD Hansen</span>, <span >B Langmead</span>, <span >JT Leek</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    Fri, Dec 30, 2016
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Genome Biology
    
  </span>
  

  

  
  
  
  <span class="middot-divider"></span>
  <a href="/publication/2016-12_intropolis/#disqus_thread"></a>
  

  
  

</div>

  

  
  
  
  
  <a href="/publication/2016-12_intropolis/">
      <img src="/publication/2016-12_intropolis/featured_hu86bb6472eb05fa8968374278fa4ec260_236812_918x517_fill_q90_lanczos_smart1_2.png" class="article-banner" alt="Human splicing diversity and the extent of unannotated splice junctions across human RNA-seq samples on the Sequence Read Archive">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/2016-12_intropolis/">Human splicing diversity and the extent of unannotated splice junctions across human RNA-seq samples on the Sequence Read Archive</a>
  </h3>

  
  <a href="/publication/2016-12_intropolis/" class="summary-link">
    <div class="article-style">
      <p>We found 56,861 junctions (18.6%) in at least 1000 samples that were not annotated out of 21,504 samples, and their expression associated with tissue type. We compiled junction data into a resource called intropolis available at <a href="http://intropolis.rail.bio">http://intropolis.rail.bio</a>.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www.biorxiv.org/content/10.1101/038224v1" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="http://genomebiology.biomedcentral.com/track/pdf/10.1186/s13059-016-1118-6?site=genomebiology.biomedcentral.com" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2016-12_intropolis/cite.bib">
  Cite
</button>


  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://figshare.com/articles/Scripts_outputs_for_paper_Human_splicing_diversity_across_the_Sequence_Read_Archive_by_Nellore_et_al_/3811629/1" target="_blank" rel="noopener">
  Code
</a>


  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://figshare.com/articles/intropolis_exon-exon_junctions_across_publicly_available_RNA-seq_samples_on_the_Sequence_Read_Archive/3811680/1" target="_blank" rel="noopener">
  Dataset
</a>



  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/project/recount2/">
    Project
  </a>
  









<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1186/s13059-016-1118-6" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2016">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >[__L Collado-Torres__](/authors/admin)</span>, <span >A Nellore</span>, <span >AC Frazee</span>, <span >C Wilks</span>, <span >MI Love</span>, <span >B Langmead</span>, <span >RA Irizarry</span>, <span >JT Leek</span>, <span >AE Jaffe</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    Mon, Sep 26, 2016
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      NAR
    
  </span>
  

  

  
  
  
  <span class="middot-divider"></span>
  <a href="/publication/2016-09_derfinder/#disqus_thread"></a>
  

  
  

</div>

  

  
  
  
  
  <a href="/publication/2016-09_derfinder/">
      <img src="/publication/2016-09_derfinder/featured_hu80015c1f13253d2568ff2a092af02f6b_231747_918x517_fill_q90_lanczos_smart1_2.png" class="article-banner" alt="Flexible expressed region analysis for RNA-seq with derfinder">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/2016-09_derfinder/">Flexible expressed region analysis for RNA-seq with derfinder</a>
  </h3>

  
  <a href="/publication/2016-09_derfinder/" class="summary-link">
    <div class="article-style">
      <p>derfinder analysis using expressed region-level and single base-level approaches provides a compromise between full transcript reconstruction and feature-level analysis. The package is available from Bioconductor.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="http://biorxiv.org/content/early/2016/05/19/015370" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="http://nar.oxfordjournals.org/content/early/2016/09/29/nar.gkw852.full.pdf&#43;html" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2016-09_derfinder/cite.bib">
  Cite
</button>


  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://github.com/leekgroup/derSupplement" target="_blank" rel="noopener">
  Code
</a>




  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/project/derfinder/">
    Project
  </a>
  









<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1093/nar/gkw852" target="_blank" rel="noopener">
  DOI
</a>


  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="http://bioconductor.org/packages/derfinder" target="_blank" rel="noopener">
    
    Bioconductor
  </a>


  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2016">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >A Nellore</span>, <span >[__L Collado-Torres__](/authors/admin)</span>, <span >AE Jaffe</span>, <span > J Alquicira-Hernández</span>, <span >C Wilks</span>, <span >J Pritt</span>, <span >J Morton</span>, <span >JT Leek</span>, <span >B Langmead</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    Sat, Sep 3, 2016
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Bioinformatics
    
  </span>
  

  

  
  
  
  <span class="middot-divider"></span>
  <a href="/publication/2016-09_rail/#disqus_thread"></a>
  

  
  

</div>

  

  
  
  
  
  <a href="/publication/2016-09_rail/">
      <img src="/publication/2016-09_rail/featured_hub94be178e119e17f0204dfbeac3b7990_213872_918x517_fill_q90_lanczos_smart1_2.png" class="article-banner" alt="Rail-RNA: Scalable analysis of RNA-seq splicing and coverage">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/2016-09_rail/">Rail-RNA: Scalable analysis of RNA-seq splicing and coverage</a>
  </h3>

  
  <a href="/publication/2016-09_rail/" class="summary-link">
    <div class="article-style">
      <p>We describe Rail-RNA, a cloud-enabled spliced aligner that analyzes many samples at once. Rail-RNA eliminates redundant work across samples, making it more efficient as samples are added. For many samples, Rail-RNA is more accurate than annotation-assisted aligners.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www.biorxiv.org/content/10.1101/019067v2" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="http://bioinformatics.oxfordjournals.org/content/early/2016/09/02/bioinformatics.btw575.full.pdf&#43;html" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2016-09_rail/cite.bib">
  Cite
</button>





  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/project/derfinder/">
    Project
  </a>
  

  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/project/recount2/">
    Project
  </a>
  









<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1093/bioinformatics/btw575" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2016">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >[__L Collado-Torres__](/authors/admin)</span>, <span >AE Jaffe</span>, <span >JT Leek</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    Wed, Jun 29, 2016
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      F1000Research 2016, 4:105
    
  </span>
  

  

  
  
  
  <span class="middot-divider"></span>
  <a href="/publication/2016-06_regionreport/#disqus_thread"></a>
  

  
  

</div>

  

  
  
  
  
  <a href="/publication/2016-06_regionreport/">
      <img src="/publication/2016-06_regionreport/featured_hu2bab50de816c0b9ea4d83a70e2362e56_299655_918x517_fill_q90_lanczos_smart1_2.png" class="article-banner" alt="regionReport: Interactive reports for region-level and feature-level genomic analyses">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/2016-06_regionreport/">regionReport: Interactive reports for region-level and feature-level genomic analyses</a>
  </h3>

  
  <a href="/publication/2016-06_regionreport/" class="summary-link">
    <div class="article-style">
      <p>regionReport is an R package for generating detailed interactive reports from region-level genomic analyses as well as feature-level RNA-seq. The report includes quality-control checks, an overview of the results, an interactive table of the genomic regions or features of interest and reproducibility information. regionReport provides specialised reports for exploring DESeq2, edgeR, or derfinder differential expression analyses results. regionReport is also flexible and can easily be expanded with report templates for other analysis pipelines.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    





  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://www.biorxiv.org/content/10.1101/016659v1" target="_blank" rel="noopener">
  Preprint
</a>




  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://f1000research.com/articles/4-105/v2/pdf" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2016-06_regionreport/cite.bib">
  Cite
</button>


  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://github.com/leekgroup/regionReportSupp" target="_blank" rel="noopener">
  Code
</a>




  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/project/derfinder/">
    Project
  </a>
  









<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.12688/f1000research.6379.2" target="_blank" rel="noopener">
  DOI
</a>


  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="http://bioconductor.org/packages/regionReport" target="_blank" rel="noopener">
    
    Bioconductor
  </a>


  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-0 year-2015">
          
            





  






<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span class="font-weight-bold"><a href="/authors/admin/">Leonardo Collado-Torres</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    Mon, Oct 19, 2015
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Poster presented at ASHG2015 and IDIES2015
    
  </span>
  

  

  
  
  
  <span class="middot-divider"></span>
  <a href="/publication/poster2015/#disqus_thread"></a>
  

  
  

</div>

  

  
  
  
  
  <a href="/publication/poster2015/">
      <img src="/publication/poster2015/featured_hue6a40410f569f5345fa305581f98a591_993972_918x517_fill_q90_lanczos_smart1_2.png" class="article-banner" alt="Annotation-agnostic RNA-seq differential expression analysis software">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/poster2015/">Annotation-agnostic RNA-seq differential expression analysis software</a>
  </h3>

  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="http://lcolladotor.github.io/Posters/ASHG2015.pdf" target="_blank" rel="noopener">
  PDF
</a>







  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/project/derfinder/">
    Project
  </a>
  











  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2014">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Jaffe AE</span>, <span >Shin J</span>, <span >__Collado-Torres L__</span>, <span >Leek JT</span>, <span >Tao R</span>, <span >Li C</span>, <span >Gao Y</span>, <span >Jia Y</span>, <span >Maher BJ</span>, <span >Hyde TM</span>, <span >Kleinman JE</span>, <span >Weinberger DR</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    Fri, Dec 19, 2014
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Nature Neuroscience
    
  </span>
  

  

  
  
  
  <span class="middot-divider"></span>
  <a href="/publication/2014-12_brainders/#disqus_thread"></a>
  

  
  

</div>

  

  
  
  
  
  <a href="/publication/2014-12_brainders/">
      <img src="/publication/2014-12_brainders/featured_hu55d5bd73db4245a0b08a0597e0f0bedc_376739_918x517_fill_q90_lanczos_smart1_2.png" class="article-banner" alt="Developmental regulation of human cortex transcription and its clinical relevance at single base resolution">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/2014-12_brainders/">Developmental regulation of human cortex transcription and its clinical relevance at single base resolution</a>
  </h3>

  
  <a href="/publication/2014-12_brainders/" class="summary-link">
    <div class="article-style">
      <p>Transcriptome analysis of human brain provides fundamental insight into development and disease, but it largely relies on existing annotation. We sequenced transcriptomes of 72 prefrontal cortex samples across six life stages and identified 50,650 differentially expression regions (DERs) associated with developmental and aging, agnostic of annotation. While many DERs annotated to non-exonic sequence (41.1%), most were similarly regulated in cytosolic mRNA extracted from independent samples. The DERs were developmentally conserved across 16 brain regions and in the developing mouse cortex, and were expressed in diverse cell and tissue types. The DERs were further enriched for active chromatin marks and clinical risk for neurodevelopmental disorders such as schizophrenia. Lastly, we demonstrate quantitatively that these DERs associate with a changing neuronal phenotype related to differentiation and maturation. These data show conserved molecular signatures of transcriptional dynamics across brain development, have potential clinical relevance and highlight the incomplete annotation of the human brain transcriptome.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="http://www.nature.com/neuro/journal/v18/n1/full/nn.3898.html" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2014-12_brainders/cite.bib">
  Cite
</button>





  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/project/derfinder/">
    Project
  </a>
  









<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1038/nn.3898" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-6 year-2014">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >AC Frazee</span>, <span >L Collado-Torres</span>, <span >AE Jaffe</span>, <span >B Langmead</span>, <span >JT Leek</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    Tue, Jun 17, 2014
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Springer
    
  </span>
  

  

  
  
  
  <span class="middot-divider"></span>
  <a href="/publication/2014-06_rnaseqbook/#disqus_thread"></a>
  

  
  

</div>

  

  
  
  
  
  <a href="/publication/2014-06_rnaseqbook/">
      <img src="/publication/2014-06_rnaseqbook/featured_hueaa2dadef3cbf156b33855f895f55cec_80877_918x517_fill_q90_lanczos_smart1_2.png" class="article-banner" alt="Measurement, Summary, and Methodological Variation in RNA-sequencing in Statistical Analysis of Next Generation Sequencing Data">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/2014-06_rnaseqbook/">Measurement, Summary, and Methodological Variation in RNA-sequencing in Statistical Analysis of Next Generation Sequencing Data</a>
  </h3>

  
  <a href="/publication/2014-06_rnaseqbook/" class="summary-link">
    <div class="article-style">
      <p>There has been a major shift from microarrays to RNA-sequencing (RNA-seq) for measuring gene expression as the price per measurement between these technologies has become comparable. The advantages of RNA-seq are increased measurement flexibility to detect alternative transcription, allele specific transcription, or transcription outside of known coding regions. The price of this increased flexibility is: (a) an increase in raw data size and (b) more decisions that must be made by the data analyst. Here we provide a selective review and extension of our previous work in attempting to measure variability in results due to different choices about how to summarize and analyze RNA-sequencing data. We discuss a standard model for gene expression measurements that breaks variability down into variation due to technology, biology, and measurement error. Finally, wee show the importance of gene model selection, normalization, and choice for statistical model on the ultimate results of an RNA-sequencing experiment.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="http://link.springer.com/chapter/10.1007/978-3-319-07212-8_6" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2014-06_rnaseqbook/cite.bib">
  Cite
</button>





  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/project/derfinder/">
    Project
  </a>
  









<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1007/978-3-319-07212-8_6" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-0 year-2014">
          
            





  






<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span class="font-weight-bold"><a href="/authors/admin/">Leonardo Collado-Torres</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    Wed, Mar 19, 2014
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Poster presented at ENAR2014
    
  </span>
  

  

  
  
  
  <span class="middot-divider"></span>
  <a href="/publication/poster2014enar/#disqus_thread"></a>
  

  
  

</div>

  

  
  
  
  
  <a href="/publication/poster2014enar/">
      <img src="/publication/poster2014enar/featured_hu00faf59d0f9890e844071e418f06e48c_1174197_918x517_fill_q90_lanczos_smart1_2.png" class="article-banner" alt="Fast Annotation-Agnostic Differential Expression Analysis">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/poster2014enar/">Fast Annotation-Agnostic Differential Expression Analysis</a>
  </h3>

  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="http://lcolladotor.github.io/Posters/ENAR2014.pdf" target="_blank" rel="noopener">
  PDF
</a>







  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/project/derfinder/">
    Project
  </a>
  











  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-0 year-2013">
          
            





  






<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span class="font-weight-bold"><a href="/authors/admin/">Leonardo Collado-Torres</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    Sat, Oct 19, 2013
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Poster presented at Genomics and Bioinformatics Symposium 2013, Delta Omega Poster Competition
    
  </span>
  

  

  
  
  
  <span class="middot-divider"></span>
  <a href="/publication/poster2013gbs/#disqus_thread"></a>
  

  
  

</div>

  

  
  
  
  
  <a href="/publication/poster2013gbs/">
      <img src="/publication/poster2013gbs/featured_hu2a58cdc5db42ffc8f03e7853a2b9f97b_1018007_918x517_fill_q90_lanczos_smart1_2.png" class="article-banner" alt="Fast Annotation-Agnostic Differential Expression Analysis">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/poster2013gbs/">Fast Annotation-Agnostic Differential Expression Analysis</a>
  </h3>

  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="http://lcolladotor.github.io/Posters/GBS2013.pdf" target="_blank" rel="noopener">
  PDF
</a>







  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/project/derfinder/">
    Project
  </a>
  











  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-0 year-2013">
          
            





  






<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span class="font-weight-bold"><a href="/authors/admin/">Leonardo Collado-Torres</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    Fri, Apr 19, 2013
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Poster presented at JHU Biostatistics Department Retreat
    
  </span>
  

  

  
  
  
  <span class="middot-divider"></span>
  <a href="/publication/poster2013retreat/#disqus_thread"></a>
  

  
  

</div>

  

  
  
  
  
  <a href="/publication/poster2013retreat/">
      <img src="/publication/poster2013retreat/featured_huc50708a1d3886bade62f59f9e11ba8ea_1273134_918x517_fill_q90_lanczos_smart1_2.png" class="article-banner" alt="Differential expression RNA-seq analysis with a large data set from brain samples">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/poster2013retreat/">Differential expression RNA-seq analysis with a large data set from brain samples</a>
  </h3>

  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="http://lcolladotor.github.io/Posters/retreat2013.pdf" target="_blank" rel="noopener">
  PDF
</a>







  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/project/derfinder/">
    Project
  </a>
  











  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2011">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Shank EA</span>, <span >Klepac-Ceraj V</span>, <span >__Collado-Torres L__</span>, <span >Powers GE</span>, <span >Losick R</span>, <span >Kolter R</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    Thu, Nov 10, 2011
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      PNAS
    
  </span>
  

  

  
  
  
  <span class="middot-divider"></span>
  <a href="/publication/2011-11_bsubtilis/#disqus_thread"></a>
  

  
  

</div>

  

  
  
  
  
  <a href="/publication/2011-11_bsubtilis/">
      <img src="/publication/2011-11_bsubtilis/featured_hu2c325411603e74e59221b38ee16d0a7c_125553_918x517_fill_q90_lanczos_center_2.png" class="article-banner" alt="Interspecies interactions that result in Bacillus subtilis forming biofilms are mediated mainly by members of its own genus">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/2011-11_bsubtilis/">Interspecies interactions that result in Bacillus subtilis forming biofilms are mediated mainly by members of its own genus</a>
  </h3>

  
  <a href="/publication/2011-11_bsubtilis/" class="summary-link">
    <div class="article-style">
      <p>Many different systems of bacterial interactions have been described. However, relatively few studies have explored how interactions between different microorganisms might influence bacterial development. To explore such interspecies interactions, we focused on Bacillus subtilis, which characteristically develops into matrix-producing cannibals before entering sporulation. We investigated whether organisms from the natural environment of B. subtilis—the soil—were able to alter the development of B.subtilis. To test this possibility, we developed a coculture microcolony screen in which we used fluorescent reporters to identify soil bacteria able to induce matrix production in B. subtilis. Most of the bacteria that influence matrix production in B. subtilis are members of the genus Bacillus, suggesting that such interactions may be predominantly with close relatives. The interactions we observed were mediated via two different mechanisms. One resulted in increased expression of matrix genes via the activation of a sensor histidine kinase, KinD. The second was kinase independent and conceivably functions by altering the relative subpopulations of B. subtilis cell types by preferentially killing noncannibals. These two mechanisms were grouped according to the inducing strain’s relatedness to B. subtilis. Our results suggest that bacteria preferentially alter their development in response to secreted molecules from closely related bacteria and do so using mechanisms that depend on the phylogenetic relatedness of the interacting bacteria.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="http://www.pnas.org/content/108/48/E1236.full.pdf&#43;html" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2011-11_bsubtilis/cite.bib">
  Cite
</button>





  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/project/">
    Project
  </a>
  









<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1073/pnas.1103630108" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-2 year-2011">
          
            





  





  


<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span >Gama-Castro S</span>, <span >Salgado H</span>, <span >Peralta-Gil M</span>, <span >Santos-Zavaleta A</span>, <span >Muñiz-Rascado L</span>, <span >Solano-Lira H</span>, <span >Jimenez-Jacinto V</span>, <span >Weiss V</span>, <span >García-Sotelo JS</span>, <span >López-Fuentes A</span>, <span >Porrón-Sotelo L</span>, <span >Alquicira-Hernández S</span>, <span >Medina-Rivera A</span>, <span >Martínez-Flores I</span>, <span >Alquicira-Hernández K</span>, <span >Martínez-Adame R</span>, <span >Bonavides-Martínez C</span>, <span >Miranda-Ríos J</span>, <span >Huerta AM</span>, <span >Mendoza-Vargas A</span>, <span >__Collado-Torres L__</span>, <span >Taboada B</span>, <span >Vega-Alvarado L</span>, <span >Olvera M</span>, <span >Olvera L</span>, <span >Grande R</span>, <span >Morett E</span>, <span >Collado-Vides J</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    Wed, Jan 19, 2011
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      NAR
    
  </span>
  

  

  
  
  
  <span class="middot-divider"></span>
  <a href="/publication/2011-01_regulondbv7/#disqus_thread"></a>
  

  
  

</div>

  

  
  
  
  
  <a href="/publication/2011-01_regulondbv7/">
      <img src="/publication/2011-01_regulondbv7/featured_hu1353773f91cd92b58099ed7d1d0a81b4_200773_918x517_fill_q90_lanczos_smart1_2.png" class="article-banner" alt="RegulonDB version 7.0: transcriptional regulation of Escherichia coli K-12 integrated within genetic sensory response units (Gensor Units)">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/2011-01_regulondbv7/">RegulonDB version 7.0: transcriptional regulation of Escherichia coli K-12 integrated within genetic sensory response units (Gensor Units)</a>
  </h3>

  
  <a href="/publication/2011-01_regulondbv7/" class="summary-link">
    <div class="article-style">
      <p>RegulonDB (<a href="http://regulondb.ccg.unam.mx/">http://regulondb.ccg.unam.mx/</a>) is the primary reference database of the best-known regulatory network of any free-living organism, that of Escherichia coli K-12. The major conceptual change since 3 years ago is an expanded biological context so that transcriptional regulation is now part of a unit that initiates with the signal and continues with the signal transduction to the core of regulation, modifying expression of the affected target genes responsible for the response. We call these genetic sensory response units, or Gensor Units. We have initiated their high-level curation, with graphic maps and superreactions with links to other databases. Additional connectivity uses expandable submaps. RegulonDB has summaries for every transcription factor (TF) and TF-binding sites with internal symmetry. Several DNA-binding motifs and their sizes have been redefined and relocated. In addition to data from the literature, we have incorporated our own information on transcription start sites (TSSs) and transcriptional units (TUs), obtained by using high-throughput whole-genome sequencing technologies. A new portable drawing tool for genomic features is also now available, as well as new ways to download the data, including web services, files for several relational database manager systems and text files including BioPAX format.</p>
    </div>
  </a>
  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="http://nar.oxfordjournals.org/content/39/suppl_1/D98.full.pdf&#43;html" target="_blank" rel="noopener">
  PDF
</a>



<button type="button" class="btn btn-outline-primary my-1 mr-1 btn-sm js-cite-modal"
        data-filename="/publication/2011-01_regulondbv7/cite.bib">
  Cite
</button>





  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/project/">
    Project
  </a>
  









<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="https://doi.org/10.1093/nar/gkq1110" target="_blank" rel="noopener">
  DOI
</a>



  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-0 year-2010">
          
            





  






<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span class="font-weight-bold"><a href="/authors/admin/">Leonardo Collado-Torres</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    Tue, Oct 19, 2010
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Poster presented at EMBO Conference Series from Functional Genomics to Systems Biology 2010
    
  </span>
  

  

  
  
  
  <span class="middot-divider"></span>
  <a href="/publication/poster2010/#disqus_thread"></a>
  

  
  

</div>

  

  
  
  
  
  <a href="/publication/poster2010/">
      <img src="/publication/poster2010/featured_hu3512b051a4aa8904e759044e180c5020_645445_918x517_fill_q90_lanczos_smart1_2.png" class="article-banner" alt="Global Analysis of Transcription Start Sites and Transcription Units in Bacterial Genomes">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/poster2010/">Global Analysis of Transcription Start Sites and Transcription Units in Bacterial Genomes</a>
  </h3>

  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="http://lcolladotor.github.io/Posters/EMBL2010poster.pdf" target="_blank" rel="noopener">
  PDF
</a>







  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/project/">
    Project
  </a>
  











  </div>
  

</div>

          
        </div>

        

        
          
        

        <div class="grid-sizer col-lg-12 isotope-item pubtype-0 year-2010">
          
            





  






<div class="card-simple">

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  
  <span class="font-weight-bold"><a href="/authors/admin/">Leonardo Collado-Torres</a></span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    Sun, Jul 25, 2010
  </span>
  

  
  <span class="middot-divider"></span>
  <span class="pub-publication">
    
      Poster presented at BioC2010
    
  </span>
  

  

  
  
  
  <span class="middot-divider"></span>
  <a href="/publication/poster2010bioc/#disqus_thread"></a>
  

  
  

</div>

  

  
  
  
  
  <a href="/publication/poster2010bioc/">
      <img src="/publication/poster2010bioc/featured_hu9839a251d184bb07f5bb7a746a32907b_1438410_918x517_fill_q90_lanczos_smart1_2.png" class="article-banner" alt="Global Analysis of Transcription Start Sites and Transcription Units in Bacterial Genomes">
  </a>
  

  <h3 class="article-title mb-1 mt-3">
    <a href="/publication/poster2010bioc/">Global Analysis of Transcription Start Sites and Transcription Units in Bacterial Genomes</a>
  </h3>

  

  
  <div class="btn-links">
    








  
    
  



<a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="http://lcolladotor.github.io/Posters/BioC2010-v4.pdf" target="_blank" rel="noopener">
  PDF
</a>







  
  <a class="btn btn-outline-primary my-1 mr-1 btn-sm" href="/project/">
    Project
  </a>
  











  </div>
  

</div>

          
        </div>

        
      </div>

    </div>
  </div>
</div>

      

    
    
    
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js" integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.4/imagesloaded.pkgd.min.js" integrity="sha256-lqvxZrPLtfffUl2G/e7szqSvPBILGbwmsGE1MKlOi0Q=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.6/isotope.pkgd.min.js" integrity="sha256-CBrpuqrMhXwcLLUd5tvQ4euBHCdh7wGlDfNz8vbu/iI=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.5.7/jquery.fancybox.min.js" integrity="sha256-yt2kYMy0w8AbtF89WXb2P1rfjcP/HTHLT7097U8Y5b8=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/instant.page/5.1.0/instantpage.min.js" integrity="sha512-1+qUtKoh9XZW7j+6LhRMAyOrgSQKenQ4mluTR+cvxXjP1Z54RxZuzstR/H9kgPXQsVB8IW7DMDFUJpzLjvhGSQ==" crossorigin="anonymous"></script>

      

      
        
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.1.2/highlight.min.js" integrity="sha512-7t8APmYpzEsZP7CYoA7RfMPV9Bb+PJHa9x2WiUnDXZx3XHveuyWUtvNOexhkierl5flZ3tr92dP1mMS+SGlD+A==" crossorigin="anonymous"></script>
        
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.1.2/languages/r.min.js"></script>
        
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.1.2/languages/bash.min.js"></script>
        
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.1.2/languages/python.min.js"></script>
        
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.1.2/languages/stata.min.js"></script>
        
      

    

    
    
      <script src="https://cdnjs.cloudflare.com/ajax/libs/leaflet/1.5.1/leaflet.js" integrity="sha256-EErZamuLefUnbMBQbsEqu1USa+btR2oIlCpBJbyD4/g=" crossorigin="anonymous"></script>
    

    
    
    <script>const code_highlighting = true;</script>
    

    
    
    <script>const isSiteThemeDark = false;</script>
    

    
    
    
    
    
    
    <script>
      const search_config = {"indexURI":"/index.json","minLength":1,"threshold":0.3};
      const i18n = {"no_results":"No results found","placeholder":"Search...","results":"results found"};
      const content_type = {
        'post': "Posts",
        'project': "Projects",
        'publication' : "Publications",
        'talk' : "Talks",
        'slides' : "Slides"
        };
    </script>
    

    
    

    
    
    <script id="search-hit-fuse-template" type="text/x-template">
      <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
      </div>
    </script>
    

    
    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/3.2.1/fuse.min.js" integrity="sha256-VzgmKYmhsGNNN4Ph1kMW+BjoYJM2jV5i4IlFoeZA9XI=" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/jquery.mark.min.js" integrity="sha256-4HLtjeVgH0eIB3aZ9mLYF6E8oU5chNdjU6p6rrXpl9U=" crossorigin="anonymous"></script>
    

    
    

    
    

    
    
    <script id="dsq-count-scr" src="https://fellgernonbit.disqus.com/count.js" async></script>
    

    
    
    
    
    
    
    
    
    
      
    
    
    
    
    <script src="/js/academic.min.66c553246b0f279a03be6e5597f72b52.js"></script>

    






  
  
  <div class="container">
    <footer class="site-footer">
  

  <p class="powered-by">
    © 2011-2020. All thoughts and opinions here are my own. The icon was designed by <a href="https://www.linkedin.com/in/mauricio-guzman-6529b551/">Mauricio Guzmán</a> and is inspired by <a href="https://en.wikipedia.org/wiki/Huichol">Huichol culture</a>; it represents my community building interests
  </p>

  
  





  
  
  
  

  
  
    
  
  
    
  

  

  
  <p class="powered-by copyright-license-text">
    This work is licensed under <a href="https://creativecommons.org/licenses/by-nc-sa/4.0" rel="noopener noreferrer" target="_blank">CC BY NC SA 4.0</a>
  </p>
  

  <p class="powered-by footer-license-icons">
    <a href="https://creativecommons.org/licenses/by-nc-sa/4.0" rel="noopener noreferrer" target="_blank">
      <img src="https://search.creativecommons.org/static/img/cc_icon.svg" alt="CC icon">
      <img src="https://search.creativecommons.org/static/img/cc-by_icon.svg" alt="CC by icon">
      
        <img src="https://search.creativecommons.org/static/img/cc-nc_icon.svg" alt="CC NC icon">
      
      
        <img src="https://search.creativecommons.org/static/img/cc-sa_icon.svg" alt="CC SA icon">
      
    </a>
  </p>



  
  <p>
      
      
      <script type="text/javascript" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=ra-4ef23e545d41b87c"></script>
      
      

      <script type="text/javascript">
      var addthis_config = {
      	data_ga_property: 'UA-45299226-1',
      	data_ga_social : true,
      	data_track_addressbar: true,
        data_track_addressbar_paths: ['/201*/*/*/*/*']
      };
      var addthis_share = {
      	
      	url_transforms : {
      		shorten: {
      			twitter: 'bitly'
      		}
      	}, 
      	shorteners : {
      		bitly : {}
      	}
      }
      </script>
      
      <center>	 
      <div id="clustrmaps-widget"></div><script type='text/javascript' id='clustrmaps' src='//cdn.clustrmaps.com/map_v2.js?cl=ffffff&w=300&t=m&d=Zo-deincaJGsqhqjIWsujtdoNOMJsnnZLefKmxDKVv8&co=124b73&cmo=dd53ff&cmn=4ec724'></script>
      </center>
  </p>

  <p class="powered-by">
    
    Published with
    <a href="https://sourcethemes.com/academic/" target="_blank" rel="noopener">Academic Website Builder</a>
    

    
    <span class="float-right" aria-hidden="true">
      <a href="#" class="back-to-top">
        <span class="button_icon">
          <i class="fas fa-chevron-up fa-2x"></i>
        </span>
      </a>
    </span>
    
  </p>
</footer>

  </div>
  

  
<div id="modal" class="modal fade" role="dialog">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <h5 class="modal-title">Cite</h5>
        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
      </div>
      <div class="modal-body">
        <pre><code class="tex hljs"></code></pre>
      </div>
      <div class="modal-footer">
        <a class="btn btn-outline-primary my-1 js-copy-cite" href="#" target="_blank">
          <i class="fas fa-copy"></i> Copy
        </a>
        <a class="btn btn-outline-primary my-1 js-download-cite" href="#" target="_blank">
          <i class="fas fa-download"></i> Download
        </a>
        <div id="modal-error"></div>
      </div>
    </div>
  </div>
</div>

</body>
</html>
